
<p><strong>Receptor-type tyrosine-protein phosphatase kappa</strong> is an <a href="enzyme" class="uri" title="wikilink">enzyme</a> that in humans is encoded by the <em>PTPRK</em> <a href="gene" class="uri" title="wikilink">gene</a>.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a><a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a><a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a> PTPRK is also known as PTPkappa and PTP&lt;U+03BA&gt;.</p>
<h2 id="function">Function</h2>
<p>The protein encoded by this gene is a member of the <a href="protein_tyrosine_phosphatase" title="wikilink">protein tyrosine phosphatase</a> (PTP) family. Protein tyrosine phosphatases are protein enzymes that remove <a href="phosphate" class="uri" title="wikilink">phosphate</a> moieties from <a href="tyrosine" class="uri" title="wikilink">tyrosine</a> residues on other proteins. Tyrosine kinases are enzymes that add phosphates to tyrosine residues, and are the opposing enzymes to PTPs. PTPs are known to be signaling molecules that regulate a variety of cellular processes including cell growth, differentiation, mitotic cycle, and oncogenic transformation.</p>
<p>The human PTPRK gene is located on the long arm of chromosome 6, a putative tumor suppressor region of the genome. <a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></p>
<h3 id="during-development">During development</h3>
<p>The same reporter construct used by Shen and colleagues, and described above was created by Skarnes et al. during a screen to identify genes important in mouse development.<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> The transgenic mouse was created by combining a &lt;U+03B2&gt;-galactosidase (&lt;U+03B2&gt;-gal) reporter gene with a signal sequence and the transmembrane domain of the type I transmembrane protein CD4. If the transgene was incorporated into a gene with a signal sequence, &lt;U+03B2&gt;-gal activity would remain in the cytosol of the cell and therefore be active. If the reporter gene was incorporated into a gene that lacked a signal sequence, &lt;U+03B2&gt;-gal activity would be in the ER where it would lose &lt;U+03B2&gt;-gal activity. This construct inserted into the phosphatase domain of PTPkappa.<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> Mice generated from these ES cells were viable, suggesting that PTPkappa phosphatase activity is not necessary for embryonic development.<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a><a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a></p>
<p>Additional studies have suggested a function for PTPkappa during nervous system development. PTPkappa promotes neurite outgrowth from embryonic cerebellar neurons, and thus may be involved in axonal extension or guidance in vivo.<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a> Neurites are extensions from neurons that can be considered the in vitro equivalent of axons and dendrites. The extension of cerebellar neurites on purified PTPkappa fusion proteins was demonstrated to require Grb2 and MEK1 activity.<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a></p>
<h3 id="in-t-cells">In T cells</h3>
<p>PTPkappa has also been shown to regulate CD4+ positive T cell development.<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> PTPkappa and the <a href="protein_THEMIS" title="wikilink">THEMIS</a> gene are both deleted in the rat Long-Evans Cinnamon (LEC) strain, and are both required for the CD4+ T-cell deficiency observed in this strain of rats.<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a> <a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a> Deletion of PTPkappa was shown to generate T-helper immunodeficiency in the LEC strain.<a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a></p>
<p>By expressing a dominant negative form of PTPkappa or by using short-hairpin RNA for PTPkappa in bone-marrow derived stem cells, Erdenbayer and colleagues demonstrated that CD4(+) T cells development was inhibited. <a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a> PTPkappa likely regulates T-cell development by positively regulating ERK1/2 phosphorylation via the regulation of MEK1/2 and c-Raf phosphorylation. <a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a></p>
<h3 id="cadherin-catenin-signaling">Cadherin-catenin signaling</h3>
<p>PTPkappa is localized to cell-cell contact sites, where it colocalizes and co-immunoprecipitates with &lt;U+03B2&gt;-catenin and plakoglobin/&lt;U+03B3&gt;-catenin<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a> &lt;U+03B2&gt;-catenin may be a PTPkappa substrate.<a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a><a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a> The presence of full-length PTPkappa in melanoma cells decreases the level of free-cytosolic &lt;U+03B2&gt;-catenin, which consequently reduces the level of nuclear &lt;U+03B2&gt;-catenin and reduces the expression of the &lt;U+03B2&gt;-catenin-regulated genes, cyclin D1 and c-myc.<a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a> Expression of ful-length PTPkappa in melanoma cells that normally lack its expression results in reduced cell migration and cell proliferation. Because the presence of PTPkappa at the cell membrane was shown to sequester &lt;U+03B2&gt;-catenin to the plasma membrane, these data suggest that one mechanism whereby PTPkappa functions as a tumor suppressor is by regulating the intracellular localization of free-&lt;U+03B2&gt;-catenin.<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a></p>
<p>The intracellular fragments of PTPkappa, P&lt;U+0394&gt;E and PIC, are catalytically active, and can also dephosphorylate &lt;U+03B2&gt;-catenin. <a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a> Tyrosine phosphorylated &lt;U+03B2&gt;-catenin translocates to the cell nucleus and activates TCF-mediated transcription to promote cell proliferation and migration. While full-length PTPkappa antagonizes TCF-mediated transcription, the PIC fragment augments it, perhaps by regulating other proteins in TCF-mediated transcription. <a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a> This suggests that phosphatase activity of the PIC fragment opposes that of full-length PTPkappa.<a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a></p>
<p>PTPkappa interacts by co-immunoprecipitation with E-cadherin, &lt;U+03B1&gt;-catenin and &lt;U+03B2&gt;-catenin in pancreatic acinar cells prior to the dissolution of adherens junctions in a rat model of pancreatitis.<a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a> The authors suggest that the presence of PTPkappa at the plasma membrane in association with the cadherin/catenin complex is important for the maintenance of adherens junction in pancreatic acinar cells, much as it was suggested above in melanoma cells. <a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a></p>
<h3 id="egfr-signaling">EGFR signaling</h3>
<p>Use of short interfering RNA (siRNA) of PTPkappa to reduce PTPkappa protein expression in the mammary epithelial cell line, MCF10A, resulted in increased cell proliferation.<a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a> PTPkappa expression , conversely, was demonstrated to reduce cell proliferation in Chinese hamster ovary cells.<a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a> The mechanism proposed to explain the influence of PTPkappa on cell proliferation is via PTPkappa dephosphorylation of the EGFR on tyrosines 1068 and 1173 directly. The reduction of PTPkappa expression in CHO cells with PTPkappa siRNA increased EGFR phosphorylation.<a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a> Therefore, the hypothesis is that PTPkappa functions as a tumor suppressor gene by dephosphorylating and inactivating EGFR. <a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a></p>
<p>In addition, glycosylation by N-acetylglucosaminyltransferase-V (GnT-V) has been shown to reduce full-length PTPkappa expression, likely via increasing its cleavage. <a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a> This aberrant glycosylation has been shown to increase the phosphorylation of EGFR on tyrosine 1068, likely because of reduced plasma-membrane associated PTPkappa expression and hence reduced PTPkappa-mediated dephosphorylation of its membrane associated susbstrates, such as EGFR.<a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a></p>
<h2 id="structure">Structure</h2>
<p>PTPkappa possesses an extracellular region, a single transmembrane region, and two tandem catalytic domains, and thus represents a receptor-type PTP (RPTP). The extracellular region contains a meprin-A5 antigen-PTP mu (MAM) domain, an Ig-like domain and four fibronectin type III-like repeats. <a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a> PTPkappa is a member of the R2B subfamily of RPTPs, which includes <a href="PTPmu" title="wikilink">RPTPM</a>, <a href="PTPrho" title="wikilink">RPTPT</a>, and <a href="PCP-2" title="wikilink">RPTPU</a>. PTPkappa shares most sequence similarity with PTPmu and PTPrho.</p>
<p>Crystal structure analysis of the first phosphatase domain of PTPkappa demonstrates that it shares many conformational features with PTPmu, including an unhindered open conformation for the catalytically important WPD loop, and a phosphate binding loop for the active-site cysteine (Cys1083). PTPkappa exists as a monomer in solution, with the caveat that dimers of PTPkappa are observed depending on the nature of the buffer used.<a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a></p>
<h2 id="alternative-splicing">Alternative splicing</h2>
<p>Alternative splicing of exons 16, 17a, and 20a has been described for PTPRK. <a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a> Two novel forms of PTPRK were identified from mouse full-length cDNA sequences and were predicted to result in two PTPkappa splice variants: a secreted form of PTPkappa and a membrane tethered form. <a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a></p>
<h2 id="homophilic-binding">Homophilic binding</h2>
<p>PTPkappa mediates homophilic cell-cell aggregation via its extracellular domain.<a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a> PTPkappa only mediates binding between cells expressing PTPkappa (ie. homophilic), and will not mediate cell aggregation between cells expressing PTPkappa, PTPmu or PTPrho (ie. heterophilic).<a href="#fn38" class="footnote-ref" id="fnref38"><sup>38</sup></a><a href="#fn39" class="footnote-ref" id="fnref39"><sup>39</sup></a></p>
<h2 id="regulation">Regulation</h2>
<h3 id="proteolysis-and-n-glycosylation">Proteolysis and N-glycosylation</h3>
<p>Full-length PTPkappa protein is cleaved by furin to generate two cleaved fragments that remain associated at the plasma membrane, an extracellular (E) subunit and an intracellular phosphatase (P) subunit. <a href="#fn40" class="footnote-ref" id="fnref40"><sup>40</sup></a><a href="#fn41" class="footnote-ref" id="fnref41"><sup>41</sup></a> In response to high cell density or calcium influx following trifluoperazine (TFP) stimulation, PTPkappa is further cleaved by ADAM 10 to yield a shed extracellular fragment and a membrane tethered intracellular fragment, P&lt;U+0394&gt;E. <a href="#fn42" class="footnote-ref" id="fnref42"><sup>42</sup></a> The membrane tethered P&lt;U+0394&gt;E fragment is further cleaved by the gamma secretase complex to yield a membrane-released fragment, PIC, that can translocate to the cellular nucleus, where it is catalytically active. <a href="#fn43" class="footnote-ref" id="fnref43"><sup>43</sup></a></p>
<p>Glycosylation of the extracellular domain of PTPkappa was demonstrated to occur preferentially in WiDr colon cancer cells that over-express N-acetylglucosaminyl transferase V (GnT-V). <a href="#fn44" class="footnote-ref" id="fnref44"><sup>44</sup></a> Over-expression of GnT-V in these cells increased the cleavage and shedding of PTPkappa ectodomain and increased migration of WiDr cells in transwell assays. <a href="#fn45" class="footnote-ref" id="fnref45"><sup>45</sup></a> As a result of glycosylation of PTPkappa by GnT-V, EGFR was phosphorylated on tyrosine 1068 and activated, and is likely the cause of the increased cell migration observed following PTPkappa cleavage.<a href="#fn46" class="footnote-ref" id="fnref46"><sup>46</sup></a></p>
<p>Shedding of PTPkappa may also be regulated by the presence of galectin-3 binding protein, as has been shown in WiDr cells. <a href="#fn47" class="footnote-ref" id="fnref47"><sup>47</sup></a> The authors suggest that the ratio of galectin-3 binding protein to galectin 3 influences the cleavage and shedding of PTPkappa, although the exact mechanism of how these proteins regulate PTPkappa cleavage was not determined.</p>
<h3 id="by-reactive-oxygen-species-in-cancer">By reactive oxygen species in cancer</h3>
<p>One mechanism whereby PTPkappa tyrosine phosphatase activity can be perturbed in cancer is via oxidative inhibition mediated by reactive oxygen species generated by either hydrogen peroxide in vitro or UV irradiation of skin cells in vivo.<a href="#fn48" class="footnote-ref" id="fnref48"><sup>48</sup></a> In cell-free assays, the presence of hydrogen peroxide reduces PTPkappa tyrosine phophatase activity and increases EGFR tyrosine phosphorylation. <a href="#fn49" class="footnote-ref" id="fnref49"><sup>49</sup></a> UV-irradiation of primary human keratinocytes yields the same results, namely a reduction of PTPkappa tyrosine phophatase activity and an increase in EGFR tyrosine phosphorylation. EGFR phopshorylation then leads to cell proliferation, suggesting that PTPkappa may function as a tumor suppressor in skin cancer in additon to melanoma. <a href="#fn50" class="footnote-ref" id="fnref50"><sup>50</sup></a></p>
<h3 id="expression">Expression</h3>
<p>PTPkappa is expressed in human keratinocytes. TGF&lt;U+03B2&gt;1 is a growth inhibitor in human keratinocytes. Stimulation of the cultured human keratinocyte cell line, HaCaT, with TGF&lt;U+03B2&gt;1 increases the levels of PTPkappa (PTPRK) mRNA as assayed by northern blot analysis.<a href="#fn51" class="footnote-ref" id="fnref51"><sup>51</sup></a> TGF&lt;U+03B2&gt;1 also increased PTPkappa mRNA and protein in normal and tumor mammary cell lines.<a href="#fn52" class="footnote-ref" id="fnref52"><sup>52</sup></a> HER2 overexpression reduced PTPkappa mRNA and protein expression.<a href="#fn53" class="footnote-ref" id="fnref53"><sup>53</sup></a></p>
<h2 id="clinical-significance">Clinical significance</h2>
<h3 id="melanoma-and-skin-cancer">Melanoma and skin cancer</h3>
<p>Expression analysis of PTPkappa mRNA in normal melanocytes and in melanoma cells and tissues demonstrated that PTPkappa is downregulated or absent 20% of the time in melanoma, suggesting that PTPkappa is a tumor suppressor gene in melanoma.<a href="#fn54" class="footnote-ref" id="fnref54"><sup>54</sup></a> A form of PTPkappa with a point mutation in the fourth fibronectin III repeat was identified to be a melanoma specific antigen recognized by CD4+ T cells in a melanoma patient with 10 year tumor-free survival after lymph node resection.<a href="#fn55" class="footnote-ref" id="fnref55"><sup>55</sup></a> This particular mutated form of PTPkappa was not identified in 10 other melanoma cell lines, and may thus represent a unique mutation in one patient.<a href="#fn56" class="footnote-ref" id="fnref56"><sup>56</sup></a></p>
<h3 id="lymphoma">Lymphoma</h3>
<p>PTPkappa was also identified as the putative tumor suppressor gene commonly deleted in primary central nervous system lymphomas (PCNSLs).<a href="#fn57" class="footnote-ref" id="fnref57"><sup>57</sup></a></p>
<p>Downregulation of PTPkappa was found to occur following Epstein-Barr Virus (EBV) infection of Hodgkin&lt;U+2019&gt;s Lymphomas cells.<a href="#fn58" class="footnote-ref" id="fnref58"><sup>58</sup></a></p>
<h3 id="colorectal-cancer">Colorectal cancer</h3>
<p>Using a transposon-based genetic screen, researchers found that disruption of the PTPRK gene in gastrointestinal tract epithelium resulted in an intestinal lesion, classified as either an intraepithelial neoplasia, an adenocarcinoma or an adenoma. <a href="#fn59" class="footnote-ref" id="fnref59"><sup>59</sup></a></p>
<h3 id="lung-cancer">Lung cancer</h3>
<p>PTPRK mRNA was shown to be significantly reduced by RT-PCR in human lung cancer-derived cell lines.<a href="#fn60" class="footnote-ref" id="fnref60"><sup>60</sup></a></p>
<h3 id="prostate-cancer">Prostate cancer</h3>
<p>PTPRK has also been shown to be downregulated in response to androgen stimulation in human LNCaP prostate cancer cells. <a href="#fn61" class="footnote-ref" id="fnref61"><sup>61</sup></a> The mechanism whereby PTPRK is downregulated is via the expression of a microRNA, miR-133b, which is upregulated in response to androgen stimulation. <a href="#fn62" class="footnote-ref" id="fnref62"><sup>62</sup></a></p>
<h3 id="breast-cancer">Breast cancer</h3>
<p>Patients with reduced PTPRK transcript expression have shorter breast cancer survival times and are more likely to have breast cancer metastases or to die from breast cancer. <a href="#fn63" class="footnote-ref" id="fnref63"><sup>63</sup></a>In an experimental model of breast cancer, PTPRK was reduced in breast cancer cell lines with PTPRK ribozymes. <a href="#fn64" class="footnote-ref" id="fnref64"><sup>64</sup></a> In these cells, adhesion to matrigel, transwell migration, and cell growth were all increased following the reduction of PTPRK expression, again supporting a function for PTPRK as a tumor suppressor.<a href="#fn65" class="footnote-ref" id="fnref65"><sup>65</sup></a></p>
<h3 id="glioma">Glioma</h3>
<p>Assem and colleagues identified loss of heterozygosity (LOH) events in malignant glioma specimens, and identified PTPRK as a significant gene candidate in one LOH region. <a href="#fn66" class="footnote-ref" id="fnref66"><sup>66</sup></a> A significant correlation between the presence of PTPRK mutations and short patient survival time was observed. <a href="#fn67" class="footnote-ref" id="fnref67"><sup>67</sup></a> PTPRK was amplified from tumor cDNA to confirm the LOH observed. In these specimens, 6 different mutations were observed, two of which (one in each phosphatase domain) disrupted the enzymatic activity of PTPRK. <a href="#fn68" class="footnote-ref" id="fnref68"><sup>68</sup></a> Expression of wild-type PTPkappa in U87-MG and U251-MG cells resulted in a reduction in cell proliferation, migration and invasion. <a href="#fn69" class="footnote-ref" id="fnref69"><sup>69</sup></a> Expression of the variants of PTPkappa with mutations in the phosphatase domains, however, increased cell proliferation, migration and invasion, supporting a role for the involvement of the mutated variants of PTPkappa in tumorigenicity. <a href="#fn70" class="footnote-ref" id="fnref70"><sup>70</sup></a></p>
<h3 id="in-development">In development</h3>
<p>In situ hybridization localized PTPkappa mRNA to the brain, lung, skeletal muscle, heart, placenta, liver, kidney and intestines during development. <a href="#fn71" class="footnote-ref" id="fnref71"><sup>71</sup></a> PTPkappa was also found to be expressed in the developing retina, in nestin-positive radial progenitor cells and later in development, in the ganglion cell layer, inner plexiform layer and outer segments of photoreceptors.<a href="#fn72" class="footnote-ref" id="fnref72"><sup>72</sup></a> PTPkappa protein is observed in neural progenitor cells and radial glial cells of the developing mouse superior colliculus, as well.<a href="#fn73" class="footnote-ref" id="fnref73"><sup>73</sup></a></p>
<p>In the adult rat brain, PTPkappa mRNA is highly expressed in regions of the brain with cellular plasticity and growth, such as the olfactory bulb, the hippocampus and the cerebral cortex.<a href="#fn74" class="footnote-ref" id="fnref74"><sup>74</sup></a> PTPkappa mRNA is also observed in the adult mouse cerebellum. <a href="#fn75" class="footnote-ref" id="fnref75"><sup>75</sup></a></p>
<p>Using a &lt;U+03B2&gt;-galactosidase (&lt;U+03B2&gt;-gal) reporter gene inserted into the phosphatase domain of the murine PTPkappa (PTPRK) gene, Shen and colleagues determined the detailed expression pattern of endogenous PTPRK.<a href="#fn76" class="footnote-ref" id="fnref76"><sup>76</sup></a> &lt;U+03B2&gt;-gal activity was observed in many areas of the adult forebrain, including layers II and IV, and to a lesser extent in layer VI of the cortex. &lt;U+03B2&gt;-gal activity was also observed in apical dendrites of cortical pyramidal cells, the granule layer of the olfactory and accessory olfactory bulbs, the anterior hypothalamus, paraventricular nucleus, and in granule and pyramidal layers of the dentate gyrus and CA 1-3 regions of the hippocampus.<a href="#fn77" class="footnote-ref" id="fnref77"><sup>77</sup></a> In the midbrain, &lt;U+03B2&gt;-gal was observed in the subthalamic nucleus, the superior and inferior colliculi and in the red nucleus. &lt;U+03B2&gt;-gal activity was also observed in the neural retina, in the inner nuclear layer and in small ganglion cells of the ganglion cell layer.<a href="#fn78" class="footnote-ref" id="fnref78"><sup>78</sup></a></p>
<h2 id="interactions">Interactions</h2>
<p>PTPRK has been shown to interact with:</p>
<ul>
<li><a href="Beta-catenin" class="uri" title="wikilink">Beta-catenin</a><a href="#fn79" class="footnote-ref" id="fnref79"><sup>79</sup></a><a href="#fn80" class="footnote-ref" id="fnref80"><sup>80</sup></a><a href="#fn81" class="footnote-ref" id="fnref81"><sup>81</sup></a></li>
<li><a href="Plakoglobin" class="uri" title="wikilink">Plakoglobin</a><a href="#fn82" class="footnote-ref" id="fnref82"><sup>82</sup></a></li>
<li><a href="&lt;U+03B1&gt;-catenin" class="uri" title="wikilink">&lt;U+03B1&gt;-catenin</a><a href="#fn83" class="footnote-ref" id="fnref83"><sup>83</sup></a></li>
<li><a href="E-cadherin" class="uri" title="wikilink">E-cadherin</a> (<a href="CDH-1" class="uri" title="wikilink">CDH-1</a>)<a href="#fn84" class="footnote-ref" id="fnref84"><sup>84</sup></a></li>
<li><a href="Epidermal_growth_factor_receptor" title="wikilink">Epidermal growth factor receptor</a> (EGFR)<a href="#fn85" class="footnote-ref" id="fnref85"><sup>85</sup></a></li>
<li><a href="HER2" class="uri" title="wikilink">HER2</a><a href="#fn86" class="footnote-ref" id="fnref86"><sup>86</sup></a></li>
</ul>
<h2 id="references">References</h2>

<h2 id="further-reading">Further reading</h2>

<ul>
<li></li>
<li></li>
</ul>



<section class="footnotes">
<hr />
<ol>
<li id="fn1"><a href="#fnref1" class="footnote-back">↩</a></li>
<li id="fn2"><a href="#fnref2" class="footnote-back">↩</a></li>
<li id="fn3"><a href="#fnref3" class="footnote-back">↩</a></li>
<li id="fn4"><a href="#fnref4" class="footnote-back">↩</a></li>
<li id="fn5"><a href="#fnref5" class="footnote-back">↩</a></li>
<li id="fn6"><a href="#fnref6" class="footnote-back">↩</a></li>
<li id="fn7"><a href="#fnref7" class="footnote-back">↩</a></li>
<li id="fn8"><a href="#fnref8" class="footnote-back">↩</a></li>
<li id="fn9"><a href="#fnref9" class="footnote-back">↩</a></li>
<li id="fn10"><a href="#fnref10" class="footnote-back">↩</a></li>
<li id="fn11"><a href="#fnref11" class="footnote-back">↩</a></li>
<li id="fn12"><a href="#fnref12" class="footnote-back">↩</a></li>
<li id="fn13"><a href="#fnref13" class="footnote-back">↩</a></li>
<li id="fn14"><a href="#fnref14" class="footnote-back">↩</a></li>
<li id="fn15"><a href="#fnref15" class="footnote-back">↩</a></li>
<li id="fn16"></li>
<li id="fn17"><a href="#fnref17" class="footnote-back">↩</a></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19" class="footnote-back">↩</a></li>
<li id="fn20"><a href="#fnref20" class="footnote-back">↩</a></li>
<li id="fn21"></li>
<li id="fn22"><a href="#fnref22" class="footnote-back">↩</a></li>
<li id="fn23"><a href="#fnref23" class="footnote-back">↩</a></li>
<li id="fn24"><a href="#fnref24" class="footnote-back">↩</a></li>
<li id="fn25"><a href="#fnref25" class="footnote-back">↩</a></li>
<li id="fn26"><a href="#fnref26" class="footnote-back">↩</a></li>
<li id="fn27"><a href="#fnref27" class="footnote-back">↩</a></li>
<li id="fn28"><a href="#fnref28" class="footnote-back">↩</a></li>
<li id="fn29"><a href="#fnref29" class="footnote-back">↩</a></li>
<li id="fn30"><a href="#fnref30" class="footnote-back">↩</a></li>
<li id="fn31"><a href="#fnref31" class="footnote-back">↩</a></li>
<li id="fn32"><a href="#fnref32" class="footnote-back">↩</a></li>
<li id="fn33"><a href="#fnref33" class="footnote-back">↩</a></li>
<li id="fn34"><a href="#fnref34" class="footnote-back">↩</a></li>
<li id="fn35"><a href="#fnref35" class="footnote-back">↩</a></li>
<li id="fn36"><a href="#fnref36" class="footnote-back">↩</a></li>
<li id="fn37"><a href="#fnref37" class="footnote-back">↩</a></li>
<li id="fn38"><a href="#fnref38" class="footnote-back">↩</a></li>
<li id="fn39"><a href="#fnref39" class="footnote-back">↩</a></li>
<li id="fn40"><a href="#fnref40" class="footnote-back">↩</a></li>
<li id="fn41"><a href="#fnref41" class="footnote-back">↩</a></li>
<li id="fn42"><a href="#fnref42" class="footnote-back">↩</a></li>
<li id="fn43"><a href="#fnref43" class="footnote-back">↩</a></li>
<li id="fn44"><a href="#fnref44" class="footnote-back">↩</a></li>
<li id="fn45"><a href="#fnref45" class="footnote-back">↩</a></li>
<li id="fn46"><a href="#fnref46" class="footnote-back">↩</a></li>
<li id="fn47"><a href="#fnref47" class="footnote-back">↩</a></li>
<li id="fn48"><a href="#fnref48" class="footnote-back">↩</a></li>
<li id="fn49"><a href="#fnref49" class="footnote-back">↩</a></li>
<li id="fn50"><a href="#fnref50" class="footnote-back">↩</a></li>
<li id="fn51"><a href="#fnref51" class="footnote-back">↩</a></li>
<li id="fn52"><a href="#fnref52" class="footnote-back">↩</a></li>
<li id="fn53"><a href="#fnref53" class="footnote-back">↩</a></li>
<li id="fn54"><a href="#fnref54" class="footnote-back">↩</a></li>
<li id="fn55"><a href="#fnref55" class="footnote-back">↩</a></li>
<li id="fn56"><a href="#fnref56" class="footnote-back">↩</a></li>
<li id="fn57"><a href="#fnref57" class="footnote-back">↩</a></li>
<li id="fn58"><a href="#fnref58" class="footnote-back">↩</a></li>
<li id="fn59"><a href="#fnref59" class="footnote-back">↩</a></li>
<li id="fn60"><a href="#fnref60" class="footnote-back">↩</a></li>
<li id="fn61"><a href="#fnref61" class="footnote-back">↩</a></li>
<li id="fn62"><a href="#fnref62" class="footnote-back">↩</a></li>
<li id="fn63"><a href="#fnref63" class="footnote-back">↩</a></li>
<li id="fn64"><a href="#fnref64" class="footnote-back">↩</a></li>
<li id="fn65"><a href="#fnref65" class="footnote-back">↩</a></li>
<li id="fn66"><a href="#fnref66" class="footnote-back">↩</a></li>
<li id="fn67"><a href="#fnref67" class="footnote-back">↩</a></li>
<li id="fn68"><a href="#fnref68" class="footnote-back">↩</a></li>
<li id="fn69"><a href="#fnref69" class="footnote-back">↩</a></li>
<li id="fn70"><a href="#fnref70" class="footnote-back">↩</a></li>
<li id="fn71"><a href="#fnref71" class="footnote-back">↩</a></li>
<li id="fn72"><a href="#fnref72" class="footnote-back">↩</a></li>
<li id="fn73"><a href="#fnref73" class="footnote-back">↩</a></li>
<li id="fn74"><a href="#fnref74" class="footnote-back">↩</a></li>
<li id="fn75"><a href="#fnref75" class="footnote-back">↩</a></li>
<li id="fn76"><a href="#fnref76" class="footnote-back">↩</a></li>
<li id="fn77"><a href="#fnref77" class="footnote-back">↩</a></li>
<li id="fn78"><a href="#fnref78" class="footnote-back">↩</a></li>
<li id="fn79"><a href="#fnref79" class="footnote-back">↩</a></li>
<li id="fn80"><a href="#fnref80" class="footnote-back">↩</a></li>
<li id="fn81"><a href="#fnref81" class="footnote-back">↩</a></li>
<li id="fn82"><a href="#fnref82" class="footnote-back">↩</a></li>
<li id="fn83"><a href="#fnref83" class="footnote-back">↩</a></li>
<li id="fn84"><a href="#fnref84" class="footnote-back">↩</a></li>
<li id="fn85"><a href="#fnref85" class="footnote-back">↩</a></li>
<li id="fn86"><a href="#fnref86" class="footnote-back">↩</a></li>
</ol>
</section>
