<p><a href="Image:ShRNA_Lentivirus.svg" title="wikilink">thumb|upright=2.8|right|<a href="Lentiviral" class="uri" title="wikilink">Lentiviral</a> Delivery of designed shRNA's and the mechanism of RNA interference in mammalian cells.</a></p>
<p><strong>RNA interference</strong> (<strong>RNAi</strong>) is a system within living cells that helps to control which genes are active and how active they are. Two types of small <a href="RNA" class="uri" title="wikilink">RNA</a> molecules &lt;U+2013&gt; <a href="microRNA" class="uri" title="wikilink">microRNA</a> (miRNA) and <a href="small_interfering_RNA" title="wikilink">small interfering RNA</a> (siRNA) &lt;U+2013&gt; are central to RNA interference. RNAs are the direct products of genes, and these small RNAs can bind to specific other RNAs and either increase or decrease their activity, for example by preventing a <a href="messenger_RNA" title="wikilink">messenger RNA</a> from producing a protein. RNA interference has an important role in defending cells against parasitic genes &lt;U+2013&gt; <a href="virus" title="wikilink">viruses</a> and <a href="transposon" title="wikilink">transposons</a> &lt;U+2013&gt; but also in directing <a href="developmental_biology" title="wikilink">development</a> as well as <a href="gene_expression" title="wikilink">gene expression</a> in general.</p>
<p>The RNAi pathway is found in many <a href="eukaryote" title="wikilink">eukaryotes</a> including animals and is initiated by the enzyme <a href="Dicer" class="uri" title="wikilink">Dicer</a>, which cleaves long <a href="RNA#Double-stranded_RNA" title="wikilink">double-stranded RNA</a> (dsRNA) molecules into short fragments of ~20 <a href="nucleotide" title="wikilink">nucleotides</a>. The siRNA will be unwound into two ssRNA, namely the passenger strand and the guide strand. The passenger strand will be degraded, and the guide strand is incorporated into the <a href="RNA-induced_silencing_complex" title="wikilink">RNA-induced silencing complex</a> (RISC). The most well-studied outcome is <a href="post-transcriptional_gene_silencing" title="wikilink">post-transcriptional gene silencing</a>, which occurs when the guide strand base pairs with a complementary sequence of a messenger RNA molecule and induces cleavage by <a href="Argonaute" class="uri" title="wikilink">Argonaute</a>, the catalytic component of the RISC complex. This process is known to spread systemically throughout the organism despite initially limited molar concentrations of siRNA.</p>
<p>The selective and robust effect of RNAi on gene expression makes it a valuable research tool, both in <a href="cell_culture" title="wikilink">cell culture</a> and in living organisms because synthetic dsRNA introduced into cells can induce suppression of specific genes of interest. RNAi may also be used for large-scale screens that systematically shut down each gene in the cell, which can help identify the components necessary for a particular cellular process or an event such as <a href="cell_division" title="wikilink">cell division</a>. Exploitation of the pathway is also a promising tool in <a href="biotechnology" class="uri" title="wikilink">biotechnology</a> and <a href="medicine" class="uri" title="wikilink">medicine</a>.</p>
<p>Historically, RNA interference was known by other names, including cosuppression, post transcriptional gene silencing, and quelling. Only after these apparently unrelated processes were fully understood did it become clear that they all described the RNAi phenomenon. In 2006, <a href="Andrew_Fire" title="wikilink">Andrew Fire</a> and <a href="Craig_C._Mello" title="wikilink">Craig C. Mello</a> shared the <a href="Nobel_Prize_in_Physiology_or_Medicine" title="wikilink">Nobel Prize in Physiology or Medicine</a> for their work on RNA interference in the <a href="nematode" class="uri" title="wikilink">nematode</a> worm <em>C. elegans</em>,<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> which they published in 1998.<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a></p>
<h2 id="cellular-mechanism">Cellular mechanism</h2>
<figure>
<img src="2ffl-by-domain.png" title="The dicer protein from Giardia intestinalis, which catalyzes the cleavage of dsRNA to siRNAs. The RNase domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue." alt="The dicer protein from Giardia intestinalis, which catalyzes the cleavage of dsRNA to siRNAs. The RNase domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue." /><figcaption>The <a href="dicer" class="uri" title="wikilink">dicer</a> protein from <em><a href="Giardia_intestinalis" title="wikilink">Giardia intestinalis</a></em>, which catalyzes the cleavage of dsRNA to siRNAs. The <a href="RNase" class="uri" title="wikilink">RNase</a> domains are colored green, the PAZ domain yellow, the platform domain red, and the connector helix blue.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></figcaption>
</figure>
<p>RNAi is an RNA-dependent <a href="gene_silencing" title="wikilink">gene silencing</a> process that is controlled by the <a href="RNA-induced_silencing_complex" title="wikilink">RNA-induced silencing complex</a> (RISC) and is initiated by short double-stranded RNA molecules in a cell's <a href="cytoplasm" class="uri" title="wikilink">cytoplasm</a>, where they interact with the catalytic RISC component <a href="argonaute" class="uri" title="wikilink">argonaute</a>.<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a> When the dsRNA is exogenous (coming from infection by a <a href="virus" class="uri" title="wikilink">virus</a> with an RNA genome or laboratory manipulations), the RNA is imported directly into the cytoplasm and cleaved to short fragments by the enzyme <a href="Dicer" class="uri" title="wikilink">Dicer</a>. The initiating dsRNA can also be endogenous (originating in the cell), as in pre-<a href="microRNA" title="wikilink">microRNAs</a> expressed from <a href="non-coding_RNA" title="wikilink">RNA-coding genes</a> in the genome. The primary transcripts from such genes are first processed to form the characteristic <a href="stem-loop" class="uri" title="wikilink">stem-loop</a> structure of pre-miRNA in the <a href="cell_nucleus" title="wikilink">nucleus</a>, then exported to the cytoplasm to be cleaved by Dicer. Thus, the two dsRNA pathways, exogenous and endogenous, converge at the RISC complex.<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a></p>
<h3 id="dsrna-cleavage">dsRNA cleavage</h3>
<p>Endogenous dsRNA initiates RNAi by activating the <a href="ribonuclease" class="uri" title="wikilink">ribonuclease</a> protein <a href="Dicer" class="uri" title="wikilink">Dicer</a>,<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> which binds and cleaves double-stranded RNAs (dsRNAs) to produce double-stranded fragments of 20&lt;U+2013&gt;25 <a href="base_pair" title="wikilink">base pairs</a> with a 2 nucleotide overhang at 3' end.<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a><a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a><a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a><a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a> <a href="Bioinformatics" class="uri" title="wikilink">Bioinformatics</a> studies on the genomes of multiple organisms suggest this length maximizes target-gene specificity and minimizes non-specific effects.<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> These short double-stranded fragments are called small interfering RNAs (<a href="siRNA" title="wikilink">siRNAs</a>). These siRNAs are then separated into single strands and integrated into an active RISC complex. After integration into the RISC, siRNAs base-pair to their target mRNA and induce cleavage of the mRNA, thereby preventing it from being used as a <a href="transcription_(genetics)" title="wikilink">transcription</a> template.<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a></p>
<p>Exogenous dsRNA is detected and bound by an effector protein, known as RDE-4 in <em><a href="Caenorhabditis_elegans" title="wikilink">C. elegans</a></em> and R2D2 in <em><a href="Drosophila" class="uri" title="wikilink">Drosophila</a></em>, that stimulates dicer activity.<a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a> This protein only binds long dsRNAs, but the mechanism producing this length specificity is unknown.<a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a> These RNA-binding proteins then facilitate transfer of cleaved siRNAs to the RISC complex.<a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a></p>
<p>In <em><a href="Caenorhabditis_elegans" title="wikilink">C. elegans</a></em>, this initiation response is amplified by the cell by the synthesis of a population of 'secondary' siRNAs using the dicer-produced initiating or 'primary' siRNAs as templates.<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a> These siRNAs are structurally distinct from dicer-produced siRNAs and appear to be produced by an <a href="RNA-dependent_RNA_polymerase" title="wikilink">RNA-dependent RNA polymerase</a> (RdRP).<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a><a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a></p>
<h3 id="microrna">MicroRNA</h3>
<p><img src="Microrna_secondary_structure.png" title="fig:The stem-loop secondary structure of a pre-microRNA from Brassica oleracea." alt="The stem-loop secondary structure of a pre-microRNA from Brassica oleracea." />  <a href="MicroRNA" title="wikilink">MicroRNAs</a> (miRNAs) are <a href="genome" title="wikilink">genomically</a> encoded <a href="non-coding_RNA" title="wikilink">non-coding RNAs</a> that help regulate <a href="gene_expression" title="wikilink">gene expression</a>, particularly during <a href="developmental_biology" title="wikilink">development</a>.<a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a><a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a> The phenomenon of RNA interference, broadly defined, includes the endogenously induced gene silencing effects of miRNAs as well as silencing triggered by foreign dsRNA. Mature miRNAs are structurally similar to siRNAs produced from exogenous dsRNA, but before reaching maturity, miRNAs must first undergo extensive <a href="post-transcriptional_modification" title="wikilink">post-transcriptional modification</a>. An miRNA is expressed from a much longer RNA-coding gene as a <a href="primary_transcript" title="wikilink">primary transcript</a> known as a <em>pri-miRNA</em> which is processed, in the <a href="cell_nucleus" title="wikilink">cell nucleus</a>, to a 70-nucleotide <a href="stem-loop" class="uri" title="wikilink">stem-loop</a> structure called a <em>pre-miRNA</em> by the microprocessor complex. This complex consists of an <a href="RNase_III" title="wikilink">RNase III</a> enzyme called <a href="Drosha" class="uri" title="wikilink">Drosha</a> and a dsRNA-binding protein <a href="Pasha" class="uri" title="wikilink">Pasha</a>. The dsRNA portion of this pre-miRNA is bound and cleaved by Dicer to produce the mature miRNA molecule that can be integrated into the RISC complex; thus, miRNA and siRNA share the same cellular machinery downstream of their initial processing.<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a></p>
<p>The siRNAs derived from long dsRNA precursors differ from miRNAs in that miRNAs, especially those in animals, typically have incomplete base pairing to a target and inhibit the translation of many different mRNAs with similar sequences. In contrast, siRNAs typically base-pair perfectly and induce mRNA cleavage only in a single, specific target.<a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a> In <em><a href="Drosophila_melanogaster" title="wikilink">Drosophila</a></em> and <em><a href="Caenorhabditis_elegans" title="wikilink">C. elegans</a></em>, miRNA and siRNA are processed by distinct argonaute proteins and dicer enzymes.<a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a><a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a></p>
<figure>
<img src="Argonaute_1u04_1ytu_composite.png" title="Left: A full-length argonaute protein from the archaea species Pyrococcus furiosus. Right: The PIWI domain of an argonaute protein in complex with double-stranded RNA." alt="Left: A full-length argonaute protein from the archaea species Pyrococcus furiosus. Right: The PIWI domain of an argonaute protein in complex with double-stranded RNA." width="300" /><figcaption><em>Left:</em> A full-length <a href="argonaute" class="uri" title="wikilink">argonaute</a> protein from the <a href="archaea" class="uri" title="wikilink">archaea</a> species <em><a href="Pyrococcus_furiosus" title="wikilink">Pyrococcus furiosus</a></em>. <em>Right:</em> The <a href="PIWI_domain" title="wikilink">PIWI domain</a> of an <a href="argonaute" class="uri" title="wikilink">argonaute</a> protein in complex with <a href="double-stranded_RNA" title="wikilink">double-stranded RNA</a>.</figcaption>
</figure>
<h3 id="risc-activation-and-catalysis">RISC activation and catalysis</h3>
<p>The active components of an <a href="RNA-induced_silencing_complex" title="wikilink">RNA-induced silencing complex</a> (RISC) are <a href="endonuclease" title="wikilink">endonucleases</a> called <a href="argonaute" class="uri" title="wikilink">argonaute</a> proteins, which cleave the target mRNA strand <a href="complementarity_(molecular_biology)" title="wikilink">complementary</a> to their bound siRNA.<a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a> As the fragments produced by dicer are double-stranded, they could each in theory produce a functional siRNA. However, only one of the two strands, which is known as the <em>guide strand</em>, binds the argonaute protein and directs gene silencing. The other <em>anti-guide strand</em> or <em>passenger strand</em> is degraded during RISC activation.<a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a> Although it was first believed that an <a href="adenosine_triphosphate" title="wikilink">ATP</a>-dependent <a href="helicase" class="uri" title="wikilink">helicase</a> separated these two strands,<a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a> the process is actually ATP-independent and performed directly by the protein components of RISC.<a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a><a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a> The strand selected as the guide tends to be the one whose <a href="5&#39;_end" title="wikilink">5' end</a> is least paired to its complement,<a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a> but strand selection is unaffected by the direction in which dicer cleaves the dsRNA before RISC incorporation.<a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a> Instead, the R2D2 protein may serve as the differentiating factor by binding the more-stable 5' end of the passenger strand.<a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a></p>
<p>The structural basis for binding of RNA to the argonaute protein was examined by <a href="X-ray_crystallography" title="wikilink">X-ray crystallography</a> of the binding <a href="structural_domain" title="wikilink">domain</a> of an RNA-bound argonaute protein. Here, the <a href="phosphorylation" title="wikilink">phosphorylated</a> 5' end of the RNA strand enters a <a href="Conserved_sequence" title="wikilink">conserved</a> <a href="basicity" title="wikilink">basic</a> surface <a href="binding_site" title="wikilink">pocket</a> and makes contacts through a <a href="divalent" class="uri" title="wikilink">divalent</a> <a href="cation" class="uri" title="wikilink">cation</a> (an atom with two positive charges) such as <a href="magnesium" class="uri" title="wikilink">magnesium</a> and by <a href="aromaticity" title="wikilink">aromatic</a> <a href="stacking_(chemistry)" title="wikilink">stacking</a> (a process that allows more than one atom to share an electron by passing it back and forth) between the 5' nucleotide in the siRNA and a conserved <a href="tyrosine" class="uri" title="wikilink">tyrosine</a> residue. This site is thought to form a nucleation site for the binding of the siRNA to its mRNA target.<a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a></p>
<p>It is not understood how the activated RISC complex locates complementary mRNAs within the cell. Although the cleavage process has been proposed to be linked to <a href="translation_(genetics)" title="wikilink">translation</a>, translation of the mRNA target is not essential for RNAi-mediated degradation.<a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a> Indeed, RNAi may be more effective against mRNA targets that are not translated.<a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a> Argonaute proteins, the catalytic components of RISC, are localized to specific regions in the <a href="cytoplasm" class="uri" title="wikilink">cytoplasm</a> called <a href="P-bodies" class="uri" title="wikilink">P-bodies</a> (also cytoplasmic bodies or GW bodies), which are regions with high rates of mRNA decay;<a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a> miRNA activity is also clustered in P-bodies.<a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a> Disruption of P-bodies decreases the efficiency of RNA interference, suggesting that they are the site of a critical step in the RNAi process.<a href="#fn38" class="footnote-ref" id="fnref38"><sup>38</sup></a> <img src="RNAi-simplified.png" title="fig:The enzyme dicer trims double stranded RNA, to form small interfering RNA or microRNA. These processed RNAs are incorporated into the RNA-induced silencing complex (RISC), which targets messenger RNA to prevent translation." alt="The enzyme dicer trims double stranded RNA, to form small interfering RNA or microRNA. These processed RNAs are incorporated into the RNA-induced silencing complex (RISC), which targets messenger RNA to prevent translation." width="350" /></p>
<h3 id="transcriptional-silencing">Transcriptional silencing</h3>
<p>Components of the RNA interference pathway are also used in many eukaryotes in the maintenance of the organisation and structure of their <a href="genome" title="wikilink">genomes</a>. Modification of <a href="histone" title="wikilink">histones</a> and associated induction of <a href="heterochromatin" class="uri" title="wikilink">heterochromatin</a> formation serves to downregulate genes pre-<a href="transcription_(genetics)" title="wikilink">transcriptionally</a>;<a href="#fn39" class="footnote-ref" id="fnref39"><sup>39</sup></a> this process is referred to as <a href="RNA-induced_transcriptional_silencing" title="wikilink">RNA-induced transcriptional silencing</a> (RITS), and is carried out by a complex of proteins called the RITS complex. In <a href="fission_yeast" title="wikilink">fission yeast</a> this complex contains <a href="argonaute" class="uri" title="wikilink">argonaute</a>, a <a href="chromodomain" class="uri" title="wikilink">chromodomain</a> protein Chp1, and a protein called Tas3 of unknown function.<a href="#fn40" class="footnote-ref" id="fnref40"><sup>40</sup></a> As a consequence, the induction and spread of heterochromatic regions requires the argonaute and RdRP proteins.<a href="#fn41" class="footnote-ref" id="fnref41"><sup>41</sup></a> Indeed, deletion of these genes in the fission yeast <em><a href="Schizosaccharomyces_pombe" title="wikilink">S. pombe</a></em> disrupts <a href="histone_methylation" title="wikilink">histone methylation</a> and <a href="centromere" class="uri" title="wikilink">centromere</a> formation,<a href="#fn42" class="footnote-ref" id="fnref42"><sup>42</sup></a> causing slow or stalled <a href="anaphase" class="uri" title="wikilink">anaphase</a> during <a href="cell_division" title="wikilink">cell division</a>.<a href="#fn43" class="footnote-ref" id="fnref43"><sup>43</sup></a> In some cases, similar processes associated with histone modification have been observed to transcriptionally upregulate genes.<a href="#fn44" class="footnote-ref" id="fnref44"><sup>44</sup></a></p>
<p>The mechanism by which the RITS complex induces heterochromatin formation and organization is not well understood, and most studies have focused on the <a href="mating-type_region" title="wikilink">mating-type region</a> in fission yeast, which may not be representative of activities in other genomic regions or organisms. In maintenance of existing heterochromatin regions, RITS forms a complex with siRNAs <a href="complementarity_(molecular_biology)" title="wikilink">complementary</a> to the local genes and stably binds local methylated histones, acting co-transcriptionally to degrade any nascent pre-mRNA transcripts that are initiated by <a href="RNA_polymerase" title="wikilink">RNA polymerase</a>. The formation of such a heterochromatin region, though not its maintenance, is dicer-dependent, presumably because dicer is required to generate the initial complement of siRNAs that target subsequent transcripts.<a href="#fn45" class="footnote-ref" id="fnref45"><sup>45</sup></a> Heterochromatin maintenance has been suggested to function as a self-reinforcing feedback loop, as new siRNAs are formed from the occasional nascent transcripts by RdRP for incorporation into local RITS complexes.<a href="#fn46" class="footnote-ref" id="fnref46"><sup>46</sup></a> The relevance of observations from fission yeast mating-type regions and centromeres to <a href="mammal" title="wikilink">mammals</a> is not clear, as heterochromatin maintenance in mammalian cells may be independent of the components of the RNAi pathway.<a href="#fn47" class="footnote-ref" id="fnref47"><sup>47</sup></a></p>
<h3 id="crosstalk-with-rna-editing">Crosstalk with RNA editing</h3>
<p>The type of <a href="RNA_editing" title="wikilink">RNA editing</a> that is most prevalent in higher eukaryotes converts <a href="adenosine" class="uri" title="wikilink">adenosine</a> nucleotides into <a href="inosine" class="uri" title="wikilink">inosine</a> in dsRNAs via the enzyme <a href="adenosine_deaminase" title="wikilink">adenosine deaminase</a> (ADAR).<a href="#fn48" class="footnote-ref" id="fnref48"><sup>48</sup></a> It was originally proposed in 2000 that the RNAi and A&lt;U+2192&gt;I RNA editing pathways might compete for a common dsRNA substrate.<a href="#fn49" class="footnote-ref" id="fnref49"><sup>49</sup></a> Indeed, some pre-miRNAs do undergo A&lt;U+2192&gt;I RNA editing,<a href="#fn50" class="footnote-ref" id="fnref50"><sup>50</sup></a><a href="#fn51" class="footnote-ref" id="fnref51"><sup>51</sup></a> and this mechanism may regulate the processing and expression of mature miRNAs.<a href="#fn52" class="footnote-ref" id="fnref52"><sup>52</sup></a> Furthermore, at least one mammalian ADAR can sequester siRNAs from RNAi pathway components.<a href="#fn53" class="footnote-ref" id="fnref53"><sup>53</sup></a> Further support for this model comes from studies on ADAR-null <em>C. elegans</em> strains indicating that A&lt;U+2192&gt;I RNA editing may counteract RNAi silencing of endogenous genes and transgenes.<a href="#fn54" class="footnote-ref" id="fnref54"><sup>54</sup></a> <img src="Rnai_diagram_retrovirology.png" title="fig:Illustration of the major differences between plant and animal gene silencing. Natively expressed microRNA or exogenous small interfering RNA is processed by dicer and integrated into the RISC complex, which mediates gene silencing." alt="Illustration of the major differences between plant and animal gene silencing. Natively expressed microRNA or exogenous small interfering RNA is processed by dicer and integrated into the RISC complex, which mediates gene silencing." width="300" /></p>
<h3 id="variation-among-organisms">Variation among organisms</h3>
<p>Organisms vary in their ability to take up foreign dsRNA and use it in the RNAi pathway. The effects of RNA interference can be both systemic and heritable in plants and <em><a href="Caenorhabditis_elegans" title="wikilink">C. elegans</a></em>, although not in <em><a href="Drosophila" class="uri" title="wikilink">Drosophila</a></em> or <a href="mammal" title="wikilink">mammals</a>. In plants, RNAi is thought to propagate by the transfer of siRNAs between cells through <a href="plasmodesmata" class="uri" title="wikilink">plasmodesmata</a> (channels in the cell walls that enable communication and transport).<a href="#fn55" class="footnote-ref" id="fnref55"><sup>55</sup></a> The heritability comes from <a href="DNA_methylation" title="wikilink">methylation</a> of promoters targeted by RNAi; the new methylation pattern is copied in each new generation of the cell.<a href="#fn56" class="footnote-ref" id="fnref56"><sup>56</sup></a> A broad general distinction between plants and animals lies in the targeting of endogenously produced miRNAs; in plants, miRNAs are usually perfectly or nearly perfectly complementary to their target genes and induce direct mRNA cleavage by RISC, while animals' miRNAs tend to be more divergent in sequence and induce translational repression.<a href="#fn57" class="footnote-ref" id="fnref57"><sup>57</sup></a> This translational effect may be produced by inhibiting the interactions of translation <a href="eukaryotic_initiation_factor" title="wikilink">initiation factors</a> with the messenger RNA's <a href="polyadenylation" title="wikilink">polyadenine tail</a>.<a href="#fn58" class="footnote-ref" id="fnref58"><sup>58</sup></a></p>
<p>Some eukaryotic protozoa such as <em><a href="Leishmania_major" title="wikilink">Leishmania major</a></em> and <em><a href="Trypanosoma_cruzi" title="wikilink">Trypanosoma cruzi</a></em> lack the RNAi pathway entirely.<a href="#fn59" class="footnote-ref" id="fnref59"><sup>59</sup></a><a href="#fn60" class="footnote-ref" id="fnref60"><sup>60</sup></a> Most or all of the components are also missing in some <a href="fungi" class="uri" title="wikilink">fungi</a>, most notably the <a href="model_organism" title="wikilink">model organism</a> <em><a href="Saccharomyces_cerevisiae" title="wikilink">Saccharomyces cerevisiae</a></em>.<a href="#fn61" class="footnote-ref" id="fnref61"><sup>61</sup></a> A recent study however reveals the presence of RNAi in other budding yeast species such as <em><a href="Saccharomyces_castellii" title="wikilink">Saccharomyces castellii</a></em> and <em><a href="Candida_albicans" title="wikilink">Candida albicans</a></em>, further demonstrating that inducing two RNAi-related proteins from <em>S. castellii</em> facilitates RNAi in <em>S. cerevisiae</em><a href="#fn62" class="footnote-ref" id="fnref62"><sup>62</sup></a>. That certain <a href="ascomycete" title="wikilink">ascomycetes</a> and <a href="basidiomycete" title="wikilink">basidiomycetes</a> are missing RNA interference pathways indicates that proteins required for RNA silencing have been lost independently from many fungal <a href="Lineage_(evolution)" title="wikilink">lineages</a>, possibly due to the evolution of a novel pathway with similar function, or to the lack of selective advantage in certain <a href="Ecological_niche" title="wikilink">niches</a>.<a href="#fn63" class="footnote-ref" id="fnref63"><sup>63</sup></a></p>
<h3 id="related-prokaryotic-systems">Related prokaryotic systems</h3>
<p>Gene expression in prokaryotes is influenced by an RNA-based system similar in some respects to RNAi. Here, RNA-encoding genes control mRNA abundance or translation by producing a complementary RNA that binds to an mRNA by base pairing. However these regulatory RNAs are not generally considered to be analogous to miRNAs because the dicer enzyme is not involved.<a href="#fn64" class="footnote-ref" id="fnref64"><sup>64</sup></a> It has been suggested that <a href="CRISPR_interference" title="wikilink">CRISPR interference</a> systems in prokaryotes are analogous to eukaryotic RNA interference systems, although none of the protein components are <a href="homology_(biology)" title="wikilink">orthologous</a>.<a href="#fn65" class="footnote-ref" id="fnref65"><sup>65</sup></a></p>
<h2 id="biological-functions">Biological functions</h2>
<h3 id="immunity">Immunity</h3>
<p>RNA interference is a vital part of the <a href="immune_response" title="wikilink">immune response</a> to <a href="virus" title="wikilink">viruses</a> and other foreign <a href="genes" title="wikilink">genetic material</a>, especially in plants where it may also prevent self-propagation by <a href="transposon" title="wikilink">transposons</a>.<a href="#fn66" class="footnote-ref" id="fnref66"><sup>66</sup></a> Plants such as <em><a href="Arabidopsis_thaliana" title="wikilink">Arabidopsis thaliana</a></em> express multiple dicer <a href="homology_(biology)" title="wikilink">homologs</a> that are specialized to react differently when the plant is exposed to different types of viruses.<a href="#fn67" class="footnote-ref" id="fnref67"><sup>67</sup></a> Even before the RNAi pathway was fully understood, it was known that induced gene silencing in plants could spread throughout the plant in a systemic effect, and could be transferred from stock to scion plants via <a href="grafting" class="uri" title="wikilink">grafting</a>.<a href="#fn68" class="footnote-ref" id="fnref68"><sup>68</sup></a> This phenomenon has since been recognized as a feature of the plant adaptive immune system, and allows the entire plant to respond to a virus after an initial localized encounter.<a href="#fn69" class="footnote-ref" id="fnref69"><sup>69</sup></a> In response, many plant viruses have evolved elaborate mechanisms that suppress the RNAi response in plant cells.<a href="#fn70" class="footnote-ref" id="fnref70"><sup>70</sup></a> These include viral proteins that bind short double-stranded RNA fragments with single-stranded overhang ends, such as those produced by the action of dicer.<a href="#fn71" class="footnote-ref" id="fnref71"><sup>71</sup></a> Some plant genomes also express endogenous siRNAs in response to infection by specific types of <a href="bacteria" class="uri" title="wikilink">bacteria</a>.<a href="#fn72" class="footnote-ref" id="fnref72"><sup>72</sup></a> These effects may be part of a generalized response to pathogens that downregulates any metabolic processes in the host that aid the infection process.<a href="#fn73" class="footnote-ref" id="fnref73"><sup>73</sup></a></p>
<p>Although animals generally express fewer variants of the dicer enzyme than plants, RNAi in some animals has also been shown to produce an antiviral response. In both juvenile and adult <em>Drosophila</em>, RNA interference is important in antiviral <a href="innate_immunity" title="wikilink">innate immunity</a> and is active against pathogens such as <a href="Drosophila_X_virus" title="wikilink">Drosophila X virus</a>.<a href="#fn74" class="footnote-ref" id="fnref74"><sup>74</sup></a><a href="#fn75" class="footnote-ref" id="fnref75"><sup>75</sup></a> A similar role in immunity may operate in <em>C. elegans</em>, as argonaute proteins are upregulated in response to viruses and worms that overexpress components of the RNAi pathway are resistant to viral infection.<a href="#fn76" class="footnote-ref" id="fnref76"><sup>76</sup></a><a href="#fn77" class="footnote-ref" id="fnref77"><sup>77</sup></a></p>
<p>The role of RNA interference in mammalian innate immunity is poorly understood, and relatively little data is available. However, the existence of viruses that encode genes able to suppress the RNAi response in mammalian cells may be evidence in favour of an RNAi-dependent mammalian immune response.<a href="#fn78" class="footnote-ref" id="fnref78"><sup>78</sup></a><a href="#fn79" class="footnote-ref" id="fnref79"><sup>79</sup></a> However, this hypothesis of RNAi-mediated immunity in mammals has been challenged as poorly substantiated.<a href="#fn80" class="footnote-ref" id="fnref80"><sup>80</sup></a> Alternative functions for RNAi in mammalian viruses also exist, such as miRNAs expressed by the <a href="herpes_virus" title="wikilink">herpes virus</a> that may act as <a href="heterochromatin" class="uri" title="wikilink">heterochromatin</a> organization triggers to mediate viral latency.<a href="#fn81" class="footnote-ref" id="fnref81"><sup>81</sup></a></p>
<h3 id="downregulation-of-genes">Downregulation of genes</h3>
<p>Endogenously expressed <a href="miRNA" title="wikilink">miRNAs</a>, including both <a href="intron" title="wikilink">intronic</a> and <a href="intergenic_region" title="wikilink">intergenic</a> miRNAs, are most important in <a href="translation_(genetics)" title="wikilink">translational repression</a><a href="#fn82" class="footnote-ref" id="fnref82"><sup>82</sup></a> and in the regulation of <a href="developmental_biology" title="wikilink">development</a>, especially on the timing of <a href="morphogenesis" class="uri" title="wikilink">morphogenesis</a> and the maintenance of <a href="cell_differentiation" title="wikilink">undifferentiated</a> or incompletely differentiated cell types such as <a href="stem_cell" title="wikilink">stem cells</a>.<a href="#fn83" class="footnote-ref" id="fnref83"><sup>83</sup></a> The role of endogenously expressed miRNA in downregulating <a href="gene_expression" title="wikilink">gene expression</a> was first described in <em>C. elegans</em> in 1993.<a href="#fn84" class="footnote-ref" id="fnref84"><sup>84</sup></a> In plants this function was discovered when the &quot;JAW microRNA&quot; of <em><a href="Arabidopsis_thaliana" title="wikilink">Arabidopsis</a></em> was shown to be involved in the regulation of several genes that control plant shape.<a href="#fn85" class="footnote-ref" id="fnref85"><sup>85</sup></a> In plants, the majority of genes regulated by miRNAs are <a href="transcription_factor" title="wikilink">transcription factors</a>;<a href="#fn86" class="footnote-ref" id="fnref86"><sup>86</sup></a> thus miRNA activity is particularly wide-ranging and regulates entire <a href="gene_network" title="wikilink">gene networks</a> during development by modulating the expression of key regulatory genes, including transcription factors as well as <a href="F-box_protein" title="wikilink">F-box proteins</a>.<a href="#fn87" class="footnote-ref" id="fnref87"><sup>87</sup></a> In many organisms, including humans, miRNAs have also been linked to the formation of <a href="tumor" title="wikilink">tumors</a> and dysregulation of the <a href="cell_cycle" title="wikilink">cell cycle</a>. Here, miRNAs can function as both <a href="oncogene" title="wikilink">oncogenes</a> and <a href="tumor_suppressor" title="wikilink">tumor suppressors</a>.<a href="#fn88" class="footnote-ref" id="fnref88"><sup>88</sup></a></p>
<h3 id="upregulation-of-genes">Upregulation of genes</h3>
<p>RNA sequences (siRNA and miRNA) that are complementary to parts of a promoter can increase gene transcription, a phenomenon dubbed <a href="RNAa" title="wikilink">RNA activation</a>. Part of the mechanism for how these RNA upregulate genes is known: <a href="dicer" class="uri" title="wikilink">dicer</a> and <a href="argonaute" class="uri" title="wikilink">argonaute</a> are involved, and there is <a href="Histone_methylation" title="wikilink">histone demethylation</a>.<a href="#fn89" class="footnote-ref" id="fnref89"><sup>89</sup></a><a href="#fn90" class="footnote-ref" id="fnref90"><sup>90</sup></a></p>
<h2 id="evolution">Evolution</h2>
<p>Based on <a href="computational_phylogenetics#Maximum_parsimony" title="wikilink">parsimony-based</a> <a href="phylogenetic" class="uri" title="wikilink">phylogenetic</a> analysis, the <a href="most_recent_common_ancestor" title="wikilink">most recent common ancestor</a> of all <a href="eukaryote" title="wikilink">eukaryotes</a> most likely already possessed an early RNA interference pathway; the absence of the pathway in certain eukaryotes is thought to be a derived characteristic.<a href="#fn91" class="footnote-ref" id="fnref91"><sup>91</sup></a> This ancestral RNAi system probably contained at least one <a href="dicer" class="uri" title="wikilink">dicer</a>-like protein, one <a href="argonaute" class="uri" title="wikilink">argonaute</a>, one <a href="PIWI_protein" title="wikilink">PIWI protein</a>, and an <a href="RNA-dependent_RNA_polymerase" title="wikilink">RNA-dependent RNA polymerase</a> that may have also played other cellular roles. A large-scale <a href="comparative_genomics" title="wikilink">comparative genomics</a> study likewise indicates that the eukaryotic <a href="crown_group" title="wikilink">crown group</a> already possessed these components, which may then have had closer functional associations with generalized RNA degradation systems such as the <a href="Exosome_complex" title="wikilink">exosome</a>.<a href="#fn92" class="footnote-ref" id="fnref92"><sup>92</sup></a> This study also suggests that the RNA-binding argonaute protein family, which is shared among eukaryotes, most archaea, and at least some bacteria (such as <em><a href="Aquifex_aeolicus" title="wikilink">Aquifex aeolicus</a></em>), is homologous to and originally evolved from components of the <a href="translation_(genetics)" title="wikilink">translation initiation</a> system.</p>
<p>The ancestral function of the RNAi system is generally agreed to have been immune defense against exogenous genetic elements such as <a href="transposon" title="wikilink">transposons</a> and <a href="virus" title="wikilink">viral</a> genomes.<a href="#fn93" class="footnote-ref" id="fnref93"><sup>93</sup></a><a href="#fn94" class="footnote-ref" id="fnref94"><sup>94</sup></a> Related functions such as <a href="histone" class="uri" title="wikilink">histone</a> modification may have already been present in the ancestor of modern eukaryotes, although other functions such as regulation of development by miRNA are thought to have evolved later.<a href="#fn95" class="footnote-ref" id="fnref95"><sup>95</sup></a></p>
<p>RNA interference genes, as components of the antiviral innate immune system in many eukaryotes, are involved in an <a href="evolutionary_arms_race" title="wikilink">evolutionary arms race</a> with viral genes. Some viruses have evolved mechanisms for suppressing the RNAi response in their host cells, an effect that has been noted particularly for plant viruses.<a href="#fn96" class="footnote-ref" id="fnref96"><sup>96</sup></a> Studies of evolutionary rates in <em>Drosophila</em> have shown that genes in the RNAi pathway are subject to strong <a href="directional_selection" title="wikilink">directional selection</a> and are among the fastest-<a href="evolution" title="wikilink">evolving</a> genes in the <em>Drosophila</em> <a href="genome" class="uri" title="wikilink">genome</a>.<a href="#fn97" class="footnote-ref" id="fnref97"><sup>97</sup></a></p>
<h2 id="technological-applications">Technological applications</h2>
<h3 id="gene-knockdown">Gene knockdown</h3>
<p>The RNA interference pathway is often exploited in experimental biology to study the function of genes in <a href="cell_culture" title="wikilink">cell culture</a> and <em>in vivo</em> in <a href="model_organism" title="wikilink">model organisms</a>.<a href="#fn98" class="footnote-ref" id="fnref98"><sup>98</sup></a> Double-stranded RNA is synthesized with a sequence complementary to a gene of interest and introduced into a cell or organism, where it is recognized as exogenous genetic material and activates the RNAi pathway. Using this mechanism, researchers can cause a drastic decrease in the expression of a targeted gene. Studying the effects of this decrease can show the physiological role of the gene product. Since RNAi may not totally abolish expression of the gene, this technique is sometimes referred as a &quot;<a href="Gene_knockdown" title="wikilink">knockdown</a>&quot;, to distinguish it from &quot;<a href="Gene_knockout" title="wikilink">knockout</a>&quot; procedures in which expression of a gene is entirely eliminated.<a href="#fn99" class="footnote-ref" id="fnref99"><sup>99</sup></a></p>
<p>Extensive efforts in <a href="computational_biology" title="wikilink">computational biology</a> have been directed toward the design of successful dsRNA reagents that maximize gene knockdown but minimize &quot;off-target&quot; effects. Off-target effects arise when an introduced RNA has a base sequence that can pair with and thus reduce the expression of multiple genes at a time. Such problems occur more frequently when the dsRNA contains repetitive sequences. It has been estimated from studying the genomes of <em><a href="H._sapiens" title="wikilink">H. sapiens</a></em>, <em><a href="Caenorhabditis_elegans" title="wikilink">C. elegans</a></em>, and <em><a href="S._pombe" title="wikilink">S. pombe</a></em> that about 10% of possible siRNAs will have substantial off-target effects.<a href="#fn100" class="footnote-ref" id="fnref100"><sup>100</sup></a> A multitude of software tools have been developed implementing <a href="algorithm" title="wikilink">algorithms</a> for the design of general,<a href="#fn101" class="footnote-ref" id="fnref101"><sup>101</sup></a><a href="#fn102" class="footnote-ref" id="fnref102"><sup>102</sup></a> mammal-specific,<a href="#fn103" class="footnote-ref" id="fnref103"><sup>103</sup></a> and virus-specific<a href="#fn104" class="footnote-ref" id="fnref104"><sup>104</sup></a> siRNAs that are automatically checked for possible cross-reactivity.</p>
<p>Depending on the organism and experimental system, the exogenous RNA may be a long strand designed to be cleaved by dicer, or short RNAs designed to serve as siRNA substrates. In most <a href="mammal" title="wikilink">mammalian</a> cells, shorter RNAs are used because long double-stranded RNA molecules induce the mammalian <a href="interferon" class="uri" title="wikilink">interferon</a> response, a form of <a href="innate_immunity" title="wikilink">innate immunity</a> that reacts nonspecifically to foreign genetic material.<a href="#fn105" class="footnote-ref" id="fnref105"><sup>105</sup></a> Mouse <a href="oocyte" title="wikilink">oocytes</a> and cells from early mouse <a href="embryo" title="wikilink">embryos</a> lack this reaction to exogenous dsRNA and are therefore a common model system for studying gene-knockdown effects in mammals.<a href="#fn106" class="footnote-ref" id="fnref106"><sup>106</sup></a> Specialized laboratory techniques have also been developed to improve the utility of RNAi in mammalian systems by avoiding the direct introduction of siRNA, for example, by stable <a href="transfection" class="uri" title="wikilink">transfection</a> with a <a href="plasmid" class="uri" title="wikilink">plasmid</a> encoding the appropriate sequence from which siRNAs can be transcribed,<a href="#fn107" class="footnote-ref" id="fnref107"><sup>107</sup></a> or by more elaborate <a href="lentivirus" title="wikilink">lentiviral</a> vector systems allowing the inducible activation or deactivation of transcription, known as <em>conditional RNAi</em>.<a href="#fn108" class="footnote-ref" id="fnref108"><sup>108</sup></a><a href="#fn109" class="footnote-ref" id="fnref109"><sup>109</sup></a></p>
<h3 id="functional-genomics">Functional genomics</h3>
<p><img src="Drosophila_melanogaster_-_side_(aka).jpg" title="fig:A normal adult Drosophila fly, a common model organism used in RNAi experiments." alt="A normal adult Drosophila fly, a common model organism used in RNAi experiments." width="250" /> <img src="Celegans_wt_nhr80rnai.png" title="fig:An adult C. elegans worm, grown under RNAi suppression of a nuclear hormone receptor involved in desaturase regulation. These worms have abnormal fatty acid metabolism but are viable and fertile." alt="An adult C. elegans worm, grown under RNAi suppression of a nuclear hormone receptor involved in desaturase regulation. These worms have abnormal fatty acid metabolism but are viable and fertile." width="250" /> Most <a href="functional_genomics" title="wikilink">functional genomics</a> applications of RNAi in animals have used <em>C. elegans</em><a href="#fn110" class="footnote-ref" id="fnref110"><sup>110</sup></a> and <em>Drosophila</em>,<a href="#fn111" class="footnote-ref" id="fnref111"><sup>111</sup></a> as these are the common <a href="model_organism" title="wikilink">model organisms</a> in which RNAi is most effective. <em>C. elegans</em> is particularly useful for RNAi research for two reasons: firstly, the effects of the gene silencing are generally heritable, and secondly because delivery of the dsRNA is extremely simple. Through a mechanism whose details are poorly understood, bacteria such as <em><a href="Escherichia_coli" title="wikilink">E. coli</a></em> that carry the desired dsRNA can be fed to the worms and will transfer their RNA payload to the worm via the intestinal tract. This &quot;delivery by feeding&quot; is just as effective at inducing gene silencing as more costly and time-consuming delivery methods, such as soaking the worms in dsRNA solution and injecting dsRNA into the gonads.<a href="#fn112" class="footnote-ref" id="fnref112"><sup>112</sup></a> Although delivery is more difficult in most other organisms, efforts are also underway to undertake large-scale genomic screening applications in cell culture with mammalian cells.<a href="#fn113" class="footnote-ref" id="fnref113"><sup>113</sup></a></p>
<p>Approaches to the design of genome-wide RNAi libraries can require more sophistication than the design of a single siRNA for a defined set of experimental conditions. <a href="Artificial_neural_network" title="wikilink">Artificial neural networks</a> are frequently used to design siRNA libraries<a href="#fn114" class="footnote-ref" id="fnref114"><sup>114</sup></a> and to predict their likely efficiency at gene knockdown.<a href="#fn115" class="footnote-ref" id="fnref115"><sup>115</sup></a> Mass genomic screening is widely seen as a promising method for <a href="genome_annotation" title="wikilink">genome annotation</a> and has triggered the development of high-throughput screening methods based on <a href="microarray" title="wikilink">microarrays</a>.<a href="#fn116" class="footnote-ref" id="fnref116"><sup>116</sup></a><a href="#fn117" class="footnote-ref" id="fnref117"><sup>117</sup></a> However, the utility of these screens and the ability of techniques developed on model organisms to generalize to even closely related species has been questioned, for example from <em>C. elegans</em> to related parasitic nematodes.<a href="#fn118" class="footnote-ref" id="fnref118"><sup>118</sup></a><a href="#fn119" class="footnote-ref" id="fnref119"><sup>119</sup></a></p>
<p>Functional genomics using RNAi is a particularly attractive technique for genomic mapping and annotation in plants because many plants are <a href="polyploid" class="uri" title="wikilink">polyploid</a>, which presents substantial challenges for more traditional genetic engineering methods. For example, RNAi has been successfully used for functional genomics studies in <a href="Triticum_aestivum" title="wikilink">bread wheat</a> (which is hexaploid)<a href="#fn120" class="footnote-ref" id="fnref120"><sup>120</sup></a> as well as more common plant model systems <em><a href="Arabidopsis_thaliana" title="wikilink">Arabidopsis</a></em> and <a href="maize" class="uri" title="wikilink">maize</a>.<a href="#fn121" class="footnote-ref" id="fnref121"><sup>121</sup></a></p>
<h3 id="medicine">Medicine</h3>
<p>It may be possible to exploit RNA interference in therapy. Although it is difficult to introduce long dsRNA strands into mammalian cells due to the <a href="interferon" class="uri" title="wikilink">interferon</a> response, the use of <a href="siRNA" title="wikilink">short interfering RNA</a> mimics has been more successful.<a href="#fn122" class="footnote-ref" id="fnref122"><sup>122</sup></a> Among the first applications to reach <a href="clinical_trial" title="wikilink">clinical trials</a> were in the treatment of <a href="macular_degeneration" title="wikilink">macular degeneration</a> and <a href="respiratory_syncytial_virus" title="wikilink">respiratory syncytial virus</a>,<a href="#fn123" class="footnote-ref" id="fnref123"><sup>123</sup></a> RNAi has also been shown to be effective in the reversal of induced liver failure in mouse models.<a href="#fn124" class="footnote-ref" id="fnref124"><sup>124</sup></a></p>
<p>Other proposed clinical uses center on antiviral therapies, including topical microbicide treatments that use RNAi to treat infection (at <a href="Harvard_University_Medical_School" title="wikilink">Harvard University Medical School</a>; in mice, so far) by <a href="herpes_simplex_virus" title="wikilink">herpes simplex virus</a> type 2 and the inhibition of viral gene expression in cancerous cells,<a href="#fn125" class="footnote-ref" id="fnref125"><sup>125</sup></a> knockdown of host receptors and coreceptors for <a href="HIV" class="uri" title="wikilink">HIV</a>,<a href="#fn126" class="footnote-ref" id="fnref126"><sup>126</sup></a> the silencing of <a href="hepatitis_A" title="wikilink">hepatitis A</a><a href="#fn127" class="footnote-ref" id="fnref127"><sup>127</sup></a> and <a href="hepatitis_B" title="wikilink">hepatitis B</a> genes,<a href="#fn128" class="footnote-ref" id="fnref128"><sup>128</sup></a> silencing of <a href="influenza" class="uri" title="wikilink">influenza</a> gene expression,<a href="#fn129" class="footnote-ref" id="fnref129"><sup>129</sup></a> and inhibition of <a href="measles" class="uri" title="wikilink">measles</a> viral replication.<a href="#fn130" class="footnote-ref" id="fnref130"><sup>130</sup></a> Potential treatments for <a href="neurodegenerative_disease" title="wikilink">neurodegenerative diseases</a> have also been proposed, with particular attention being paid to the polyglutamine diseases such as <a href="Huntington&#39;s_disease" title="wikilink">Huntington's disease</a>.<a href="#fn131" class="footnote-ref" id="fnref131"><sup>131</sup></a> RNA interference is also often seen as a promising way to treat <a href="cancer" class="uri" title="wikilink">cancer</a> by silencing genes differentially upregulated in <a href="tumor" class="uri" title="wikilink">tumor</a> cells or genes involved in <a href="cell_division" title="wikilink">cell division</a>.<a href="#fn132" class="footnote-ref" id="fnref132"><sup>132</sup></a><a href="#fn133" class="footnote-ref" id="fnref133"><sup>133</sup></a> A key area of research in the use of RNAi for clinical applications is the development of a safe delivery method, which to date has involved mainly <a href="viral_vector" title="wikilink">viral vector</a> systems similar to those suggested for <a href="gene_therapy" title="wikilink">gene therapy</a>.<a href="#fn134" class="footnote-ref" id="fnref134"><sup>134</sup></a><a href="#fn135" class="footnote-ref" id="fnref135"><sup>135</sup></a></p>
<p>Despite the proliferation of promising cell culture studies for RNAi-based drugs, some concern has been raised regarding the safety of RNA interference, especially the potential for &quot;off-target&quot; effects in which a gene with a coincidentally similar sequence to the targeted gene is also repressed.<a href="#fn136" class="footnote-ref" id="fnref136"><sup>136</sup></a> A computational genomics study estimated that the error rate of off-target interactions is about 10%.<a href="#fn137" class="footnote-ref" id="fnref137"><sup>137</sup></a> One major study of liver disease in mice led to high death rates in the experimental animals, suggested by researchers to be the result of &quot;oversaturation&quot; of the dsRNA pathway,<a href="#fn138" class="footnote-ref" id="fnref138"><sup>138</sup></a> due to the use of shRNAs that have to be processed in the nucleus and exported to the cytoplasm using an active mechanism. All these are considerations that are under active investigation, to reduce their impact in the potential therapeutic applications for RNAi.</p>
<p>RNA interference-based applications are being developed to target persistent <a href="HIV-1" class="uri" title="wikilink">HIV-1</a> infection. Viruses like HIV-1 are particularly difficult targets for RNAi-attack because they are escape-prone, which requires combinatorial RNAi strategies to prevent viral escape. The future of antiviral RNAi therapeutics is very promising, but it remains of critical importance to include many controls in pre-clinical test models to unequivocally demonstrate sequence-specific action of the RNAi inducers.<a href="#fn139" class="footnote-ref" id="fnref139"><sup>139</sup></a></p>
<h3 id="biotechnology">Biotechnology</h3>
<p>RNA interference has been used for applications in <a href="biotechnology" class="uri" title="wikilink">biotechnology</a>, particularly in the engineering of food plants that produce lower levels of natural plant toxins. Such techniques take advantage of the stable and heritable RNAi phenotype in plant stocks. For example, <a href="cotton" class="uri" title="wikilink">cotton</a> seeds are rich in <a href="protein_in_nutrition" title="wikilink">dietary protein</a> but naturally contain the toxic <a href="terpenoid" class="uri" title="wikilink">terpenoid</a> product <a href="gossypol" class="uri" title="wikilink">gossypol</a>, making them unsuitable for human consumption. RNAi has been used to produce cotton stocks whose seeds contain reduced levels of <a href="delta-cadinene_synthase" title="wikilink">delta-cadinene synthase</a>, a key enzyme in gossypol production, without affecting the enzyme's production in other parts of the plant, where gossypol is important in preventing damage from plant pests.<a href="#fn140" class="footnote-ref" id="fnref140"><sup>140</sup></a> Similar efforts have been directed toward the reduction of the <a href="cyanide" title="wikilink">cyanogenic</a> natural product <a href="linamarin" class="uri" title="wikilink">linamarin</a> in <a href="cassava" class="uri" title="wikilink">cassava</a> plants.<a href="#fn141" class="footnote-ref" id="fnref141"><sup>141</sup></a></p>
<p>Although no plant products that use RNAi-based <a href="genetic_engineering" title="wikilink">genetic engineering</a> have yet passed the experimental stage, development efforts have successfully reduced the levels of <a href="allergen" title="wikilink">allergens</a> in <a href="tomato" class="uri" title="wikilink">tomato</a> plants<a href="#fn142" class="footnote-ref" id="fnref142"><sup>142</sup></a> and decreased the precursors of likely <a href="carcinogen" title="wikilink">carcinogens</a> in <a href="tobacco" class="uri" title="wikilink">tobacco</a> plants.<a href="#fn143" class="footnote-ref" id="fnref143"><sup>143</sup></a> Other plant traits that have been engineered in the laboratory include the production of non-<a href="narcotic" class="uri" title="wikilink">narcotic</a> natural products by the <a href="opium_poppy" title="wikilink">opium poppy</a>,<a href="#fn144" class="footnote-ref" id="fnref144"><sup>144</sup></a> resistance to common plant viruses,<a href="#fn145" class="footnote-ref" id="fnref145"><sup>145</sup></a> and fortification of plants such as tomatoes with dietary <a href="antioxidant" title="wikilink">antioxidants</a>.<a href="#fn146" class="footnote-ref" id="fnref146"><sup>146</sup></a> Previous commercial products, including the <a href="Flavr_Savr" title="wikilink">Flavr Savr</a> tomato and two <a href="cultivar" title="wikilink">cultivars</a> of <a href="Papaya_ringspot_virus" title="wikilink">ringspot</a>-resistant <a href="papaya" class="uri" title="wikilink">papaya</a>, were originally developed using <a href="antisense" class="uri" title="wikilink">antisense</a> technology but likely exploited the RNAi pathway.<a href="#fn147" class="footnote-ref" id="fnref147"><sup>147</sup></a><a href="#fn148" class="footnote-ref" id="fnref148"><sup>148</sup></a></p>
<h2 id="history-and-discovery">History and discovery</h2>
<p><img src="Rnai_phenotype_petunia_crop.png" title="fig:Example petunia plants in which genes for pigmentation are silenced by RNAi. The left plant is wild-type; the right plants contain transgenes that induce suppression of both transgene and endogenous gene expression, giving rise to the unpigmented white areas of the flower." alt="Example petunia plants in which genes for pigmentation are silenced by RNAi. The left plant is wild-type; the right plants contain transgenes that induce suppression of both transgene and endogenous gene expression, giving rise to the unpigmented white areas of the flower." width="250" /> The discovery of RNAi was preceded first by observations of transcriptional inhibition by <a href="antisense" class="uri" title="wikilink">antisense</a> RNA expressed in <a href="transgenic" class="uri" title="wikilink">transgenic</a> plants,<a href="#fn149" class="footnote-ref" id="fnref149"><sup>149</sup></a> and more directly by reports of unexpected outcomes in experiments performed by plant scientists in the <a href="United_States" title="wikilink">United States</a> and the <a href="Netherlands" class="uri" title="wikilink">Netherlands</a> in the early 1990s.<a href="#fn150" class="footnote-ref" id="fnref150"><sup>150</sup></a> In an attempt to alter <a href="flower" class="uri" title="wikilink">flower</a> colors in <a href="petunia" title="wikilink">petunias</a>, researchers introduced additional copies of a gene encoding <a href="chalcone_synthase" title="wikilink">chalcone synthase</a>, a key enzyme for flower <a href="pigmentation" class="uri" title="wikilink">pigmentation</a> into petunia plants of normally pink or violet flower color. The overexpressed gene was expected to result in darker flowers, but instead produced less pigmented, fully or partially white flowers, indicating that the activity of chalcone synthase had been substantially decreased; in fact, both the endogenous genes and the transgenes were downregulated in the white flowers. Soon after, a related event termed <em>quelling</em> was noted in the <a href="fungus" class="uri" title="wikilink">fungus</a> <em><a href="Neurospora_crassa" title="wikilink">Neurospora crassa</a></em>,<a href="#fn151" class="footnote-ref" id="fnref151"><sup>151</sup></a> although it was not immediately recognized as related. Further investigation of the phenomenon in plants indicated that the downregulation was due to post-transcriptional inhibition of gene expression via an increased rate of mRNA degradation.<a href="#fn152" class="footnote-ref" id="fnref152"><sup>152</sup></a> This phenomenon was called <em>co-suppression of gene expression</em>, but the molecular mechanism remained unknown.<a href="#fn153" class="footnote-ref" id="fnref153"><sup>153</sup></a> <img src="Craig_mello.jpg" title="fig:Craig Mello at the 2006 Nobel Prize lecture." alt="Craig Mello at the 2006 Nobel Prize lecture." /> Not long after, plant <a href="virologist" title="wikilink">virologists</a> working on improving plant resistance to viral diseases observed a similar unexpected phenomenon. While it was known that plants expressing virus-specific proteins showed enhanced tolerance or resistance to viral infection, it was not expected that plants carrying only short, non-coding regions of viral RNA sequences would show similar levels of protection. Researchers believed that viral RNA produced by transgenes could also inhibit viral replication.<a href="#fn154" class="footnote-ref" id="fnref154"><sup>154</sup></a> The reverse experiment, in which short sequences of plant genes were introduced into viruses, showed that the targeted gene was suppressed in an infected plant. This phenomenon was labeled &quot;virus-induced gene silencing&quot; (VIGS), and the set of such phenomena were collectively called post transcriptional gene silencing.<a href="#fn155" class="footnote-ref" id="fnref155"><sup>155</sup></a></p>
<p>After these initial observations in plants, many laboratories around the world searched for the occurrence of this phenomenon in other organisms.<a href="#fn156" class="footnote-ref" id="fnref156"><sup>156</sup></a><a href="#fn157" class="footnote-ref" id="fnref157"><sup>157</sup></a> <a href="Craig_C._Mello" title="wikilink">Craig C. Mello</a> and <a href="Andrew_Fire" title="wikilink">Andrew Fire</a>'s 1998 <em>Nature</em> paper reported a potent gene silencing effect after injecting double stranded RNA into <em><a href="Caenorhabditis_elegans" title="wikilink">C. elegans</a></em>.<a href="#fn158" class="footnote-ref" id="fnref158"><sup>158</sup></a> In investigating the regulation of muscle protein production, they observed that neither mRNA nor <a href="antisense_RNA" title="wikilink">antisense RNA</a> injections had an effect on protein production, but double-stranded RNA successfully silenced the targeted gene. As a result of this work, they coined the term <em>RNAi</em>. Fire and Mello's discovery was particularly notable because it represented the first identification of the causative agent for the phenomenon. Fire and Mello were awarded the <a href="Nobel_Prize_in_Physiology_or_Medicine" title="wikilink">Nobel Prize in Physiology or Medicine</a> in 2006 for their work.<a href="#fn159" class="footnote-ref" id="fnref159"><sup>159</sup></a></p>
<h2 id="references">References</h2>

<h2 id="external-links">External links</h2>

<ul>
<li><a href="http://www.thenakedscientists.com/HTML/articles/article/rna-interference-explained/">Overview of the RNAi process</a>, from <em><a href="The_Naked_Scientists" title="wikilink">Cambridge University's The Naked Scientists</a></em></li>
<li><a href="http://www.nature.com/focus/rnai/animations/animation/animation.htm">Animation of the RNAi process</a>, from <em><a href="Nature_(journal)" title="wikilink">Nature</a></em></li>
<li><a href="http://www.pbs.org/wgbh/nova/sciencenow/3210/02.html">NOVA scienceNOW explains RNAi</a> &lt;U+2013&gt; A 15 minute video of the <em><a href="Nova_(TV_series)" title="wikilink">Nova</a></em> broadcast that aired on <a href="PBS" class="uri" title="wikilink">PBS</a>, July 26, 2005</li>
<li></li>
<li><a href="http://www.silencinggenomes.org">Silencing Genomes</a> RNA interference (RNAi) experiments and bioinformatics in C. elegans for education. From the Dolan DNA Learning Center of Cold Spring Harbor Laboratory.</li>
<li><a href="http://www.natureprotocols.com/2006/07/19/rnai_screens_in_c_elegans_in_a.php">RNAi screens in C. elegans in a 96-well liquid format and their application to the systematic identification of genetic interactions (a protocol)</a></li>
<li><a href="http://www.nytimes.com/2006/10/03/science/03nobel.html?ex=1180065600&amp;en=e26ec371cb398ea0&amp;ei=5070">2 American &lt;U+2018&gt;Worm People&lt;U+2019&gt; Win Nobel for RNA Work</a>, from <a href="The_New_York_Times" title="wikilink">NY Times</a>''</li>
<li><a href="http://www.nature.com/mt/webfocus/rnai/index.html#2008"><em>Molecular Therapy</em> web focus: &quot;The development of RNAi as a therapeutic strategy&quot;</a>, a collection of free articles about RNAi as a therapeutic strategy.</li>
</ul>

<p></p>
<p><a href="an:Interferencia_d&#39;ARN" title="wikilink">an:Interferencia d'ARN</a> <a href="zh-min-nan:RNA_kan-g&lt;U+0101&gt;i" title="wikilink">zh-min-nan:RNA kan-g&lt;U+0101&gt;i</a> <a href="cs:RNA_interference" title="wikilink">cs:RNA interference</a> <a href="da:RNA-interferens" class="uri" title="wikilink">da:RNA-interferens</a> <a href="de:RNA-Interferenz" class="uri" title="wikilink">de:RNA-Interferenz</a> <a href="el:&lt;U+0395&gt;&lt;U+03C0&gt;&lt;U+03B5&gt;&lt;U+03BC&gt;&lt;U+03B2&gt;&lt;U+03B1&gt;&lt;U+03C4&gt;&lt;U+03B9&gt;&lt;U+03BA&gt;&lt;U+03CC&gt;_RNA" title="wikilink">el:&lt;U+0395&gt;&lt;U+03C0&gt;&lt;U+03B5&gt;&lt;U+03BC&gt;&lt;U+03B2&gt;&lt;U+03B1&gt;&lt;U+03C4&gt;&lt;U+03B9&gt;&lt;U+03BA&gt;&lt;U+03CC&gt; RNA</a> <a href="es:ARN_interferente" title="wikilink">es:ARN interferente</a> <a href="fa:&lt;U+0622&gt;&lt;U+0631&gt;&lt;U+0627&gt;&lt;U+0646&gt;&lt;U+200C&gt;&lt;U+0627&gt;&lt;U+06CC&gt;_&lt;U+0645&gt;&lt;U+062F&gt;&lt;U+0627&gt;&lt;U+062E&gt;&lt;U+0644&gt;&lt;U+0647&gt;&lt;U+200C&gt;&lt;U+06AF&gt;&lt;U+0631&gt;" title="wikilink">fa:&lt;U+0622&gt;&lt;U+0631&gt;&lt;U+0627&gt;&lt;U+0646&gt;&lt;U+200C&gt;&lt;U+0627&gt;&lt;U+06CC&gt; &lt;U+0645&gt;&lt;U+062F&gt;&lt;U+0627&gt;&lt;U+062E&gt;&lt;U+0644&gt;&lt;U+0647&gt;&lt;U+200C&gt;&lt;U+06AF&gt;&lt;U+0631&gt;</a> <a href="fr:ARN_interf&lt;U+00E9&gt;rent" title="wikilink">fr:ARN interf&lt;U+00E9&gt;rent</a> <a href="ko:RNA_&lt;U+AC04&gt;&lt;U+C12D&gt;" title="wikilink">ko:RNA &lt;U+AC04&gt;&lt;U+C12D&gt;</a> <a href="hi:&lt;U+0906&gt;&lt;U+0930&gt;&lt;U+090F&gt;&lt;U+0928&gt;&lt;U+090F&gt;_&lt;U+0939&gt;&lt;U+0938&gt;&lt;U+094D&gt;&lt;U+0924&gt;&lt;U+0915&gt;&lt;U+094D&gt;&lt;U+0937&gt;&lt;U+0947&gt;&lt;U+092A&gt;" title="wikilink">hi:&lt;U+0906&gt;&lt;U+0930&gt;&lt;U+090F&gt;&lt;U+0928&gt;&lt;U+090F&gt; &lt;U+0939&gt;&lt;U+0938&gt;&lt;U+094D&gt;&lt;U+0924&gt;&lt;U+0915&gt;&lt;U+094D&gt;&lt;U+0937&gt;&lt;U+0947&gt;&lt;U+092A&gt;</a> <a href="id:Interferensi_RNA" title="wikilink">id:Interferensi RNA</a> <a href="it:RNA_interference" title="wikilink">it:RNA interference</a> <a href="he:RNAi" class="uri" title="wikilink">he:RNAi</a> <a href="kn:&lt;U+0C86&gt;&lt;U+0CB0&gt;&lt;U+0CCD&gt;&lt;U+200C&gt;&lt;U+0C8E&gt;&lt;U+0CA8&gt;&lt;U+0CCD&gt;&lt;U+200C&gt;&lt;U+0C8E&gt;_&lt;U+0CB9&gt;&lt;U+0CB8&gt;&lt;U+0CCD&gt;&lt;U+0CA4&gt;&lt;U+0C95&gt;&lt;U+0CCD&gt;&lt;U+0CB7&gt;&lt;U+0CC7&gt;&lt;U+0CAA&gt;" title="wikilink">kn:&lt;U+0C86&gt;&lt;U+0CB0&gt;&lt;U+0CCD&gt;&lt;U+200C&gt;&lt;U+0C8E&gt;&lt;U+0CA8&gt;&lt;U+0CCD&gt;&lt;U+200C&gt;&lt;U+0C8E&gt; &lt;U+0CB9&gt;&lt;U+0CB8&gt;&lt;U+0CCD&gt;&lt;U+0CA4&gt;&lt;U+0C95&gt;&lt;U+0CCD&gt;&lt;U+0CB7&gt;&lt;U+0CC7&gt;&lt;U+0CAA&gt;</a> <a href="lv:RNS_interference" title="wikilink">lv:RNS interference</a> <a href="nl:RNA-interferentie" class="uri" title="wikilink">nl:RNA-interferentie</a> <a href="ja:RNAi" class="uri" title="wikilink">ja:RNAi</a> <a href="no:RNA-interferens" class="uri" title="wikilink">no:RNA-interferens</a> <a href="nn:RNA-interferens" class="uri" title="wikilink">nn:RNA-interferens</a> <a href="pl:Interferencja_RNA" title="wikilink">pl:Interferencja RNA</a> <a href="pt:RNA_interferente" title="wikilink">pt:RNA interferente</a> <a href="ru:&lt;U+0420&gt;&lt;U+041D&gt;&lt;U+041A&gt;-&lt;U+0438&gt;&lt;U+043D&gt;&lt;U+0442&gt;&lt;U+0435&gt;&lt;U+0440&gt;&lt;U+0444&gt;&lt;U+0435&gt;&lt;U+0440&gt;&lt;U+0435&gt;&lt;U+043D&gt;&lt;U+0446&gt;&lt;U+0438&gt;&lt;U+044F&gt;" class="uri" title="wikilink">ru:&lt;U+0420&gt;&lt;U+041D&gt;&lt;U+041A&gt;-&lt;U+0438&gt;&lt;U+043D&gt;&lt;U+0442&gt;&lt;U+0435&gt;&lt;U+0440&gt;&lt;U+0444&gt;&lt;U+0435&gt;&lt;U+0440&gt;&lt;U+0435&gt;&lt;U+043D&gt;&lt;U+0446&gt;&lt;U+0438&gt;&lt;U+044F&gt;</a> <a href="fi:RNA-interferenssi" class="uri" title="wikilink">fi:RNA-interferenssi</a> <a href="sv:RNA-interferens" class="uri" title="wikilink">sv:RNA-interferens</a> <a href="ta:&lt;U+0B86&gt;&lt;U+0BB0&gt;&lt;U+0BCD&gt;.&lt;U+0B8E&gt;&lt;U+0BA9&gt;&lt;U+0BCD&gt;.&lt;U+0B8F&gt;_&lt;U+0B95&gt;&lt;U+0BC1&gt;&lt;U+0BB1&gt;&lt;U+0BC1&gt;&lt;U+0B95&gt;&lt;U+0BCD&gt;&lt;U+0B95&gt;&lt;U+0BC0&gt;&lt;U+0B9F&gt;&lt;U+0BC1&gt;" title="wikilink">ta:&lt;U+0B86&gt;&lt;U+0BB0&gt;&lt;U+0BCD&gt;.&lt;U+0B8E&gt;&lt;U+0BA9&gt;&lt;U+0BCD&gt;.&lt;U+0B8F&gt; &lt;U+0B95&gt;&lt;U+0BC1&gt;&lt;U+0BB1&gt;&lt;U+0BC1&gt;&lt;U+0B95&gt;&lt;U+0BCD&gt;&lt;U+0B95&gt;&lt;U+0BC0&gt;&lt;U+0B9F&gt;&lt;U+0BC1&gt;</a> <a href="th:RNA_interference" title="wikilink">th:RNA interference</a> <a href="ur:&lt;U+0627&gt;&lt;U+0631&gt;&lt;U+0646&gt;&lt;U+0627&gt;_&lt;U+062A&gt;&lt;U+062F&gt;&lt;U+0627&gt;&lt;U+062E&gt;&lt;U+0644&gt;" title="wikilink">ur:&lt;U+0627&gt;&lt;U+0631&gt;&lt;U+0646&gt;&lt;U+0627&gt; &lt;U+062A&gt;&lt;U+062F&gt;&lt;U+0627&gt;&lt;U+062E&gt;&lt;U+0644&gt;</a> <a href="zh:RNA&lt;U+5E72&gt;&lt;U+6270&gt;" class="uri" title="wikilink">zh:RNA&lt;U+5E72&gt;&lt;U+6270&gt;</a></p>
<p><a href="Category:RNA" class="uri" title="wikilink">Category:RNA</a> <a href="Category:RNA_interference" title="wikilink">Category:RNA interference</a> <a href="Category:Gene_expression" title="wikilink">Category:Gene expression</a> <a href="Category:Molecular_genetics" title="wikilink">Category:Molecular genetics</a></p>
<section class="footnotes">
<hr />
<ol>
<li id="fn1"><a href="#fnref1" class="footnote-back">↩</a></li>
<li id="fn2"><a href="#fnref2" class="footnote-back">↩</a></li>
<li id="fn3"><a href="#fnref3" class="footnote-back">↩</a></li>
<li id="fn4"></li>
<li id="fn5"><a href="#fnref5" class="footnote-back">↩</a></li>
<li id="fn6"><a href="#fnref6" class="footnote-back">↩</a></li>
<li id="fn7"><a href="#fnref7" class="footnote-back">↩</a></li>
<li id="fn8"><a href="#fnref8" class="footnote-back">↩</a></li>
<li id="fn9"><a href="#fnref9" class="footnote-back">↩</a></li>
<li id="fn10"><a href="#fnref10" class="footnote-back">↩</a></li>
<li id="fn11"><a href="#fnref11" class="footnote-back">↩</a></li>
<li id="fn12"><a href="#fnref12" class="footnote-back">↩</a></li>
<li id="fn13"><a href="#fnref13" class="footnote-back">↩</a></li>
<li id="fn14"></li>
<li id="fn15"><a href="#fnref15" class="footnote-back">↩</a></li>
<li id="fn16"><a href="#fnref16" class="footnote-back">↩</a></li>
<li id="fn17"><a href="#fnref17" class="footnote-back">↩</a></li>
<li id="fn18"><a href="#fnref18" class="footnote-back">↩</a></li>
<li id="fn19"><a href="#fnref19" class="footnote-back">↩</a></li>
<li id="fn20"><a href="#fnref20" class="footnote-back">↩</a></li>
<li id="fn21"><a href="#fnref21" class="footnote-back">↩</a></li>
<li id="fn22"><a href="#fnref22" class="footnote-back">↩</a></li>
<li id="fn23"><a href="#fnref23" class="footnote-back">↩</a></li>
<li id="fn24"><a href="#fnref24" class="footnote-back">↩</a></li>
<li id="fn25"></li>
<li id="fn26"><a href="#fnref26" class="footnote-back">↩</a></li>
<li id="fn27"><a href="#fnref27" class="footnote-back">↩</a></li>
<li id="fn28"><a href="#fnref28" class="footnote-back">↩</a></li>
<li id="fn29"><a href="#fnref29" class="footnote-back">↩</a></li>
<li id="fn30"><a href="#fnref30" class="footnote-back">↩</a></li>
<li id="fn31"><a href="#fnref31" class="footnote-back">↩</a></li>
<li id="fn32"><a href="#fnref32" class="footnote-back">↩</a></li>
<li id="fn33"><a href="#fnref33" class="footnote-back">↩</a></li>
<li id="fn34"><a href="#fnref34" class="footnote-back">↩</a></li>
<li id="fn35"><a href="#fnref35" class="footnote-back">↩</a></li>
<li id="fn36"><a href="#fnref36" class="footnote-back">↩</a></li>
<li id="fn37"><a href="#fnref37" class="footnote-back">↩</a></li>
<li id="fn38"><a href="#fnref38" class="footnote-back">↩</a></li>
<li id="fn39"><a href="#fnref39" class="footnote-back">↩</a></li>
<li id="fn40"><a href="#fnref40" class="footnote-back">↩</a></li>
<li id="fn41"><a href="#fnref41" class="footnote-back">↩</a></li>
<li id="fn42"><a href="#fnref42" class="footnote-back">↩</a></li>
<li id="fn43"><a href="#fnref43" class="footnote-back">↩</a></li>
<li id="fn44">Li LC, Okino ST, Zhao H, Pookot D, Place RF, Urakami S, Enokida H, Dahiya R. (2006). Small dsRNAs induce transcriptional activation in human cells. <em>Proc Natl Acad Sci USA</em> 103(46):17337&lt;U+2013&gt;42. PMID 17085592<a href="#fnref44" class="footnote-back">↩</a></li>
<li id="fn45"><a href="#fnref45" class="footnote-back">↩</a></li>
<li id="fn46"><a href="#fnref46" class="footnote-back">↩</a></li>
<li id="fn47"><a href="#fnref47" class="footnote-back">↩</a></li>
<li id="fn48"><a href="#fnref48" class="footnote-back">↩</a></li>
<li id="fn49"><a href="#fnref49" class="footnote-back">↩</a></li>
<li id="fn50"><a href="#fnref50" class="footnote-back">↩</a></li>
<li id="fn51"><a href="#fnref51" class="footnote-back">↩</a></li>
<li id="fn52"></li>
<li id="fn53"><a href="#fnref53" class="footnote-back">↩</a></li>
<li id="fn54"><a href="#fnref54" class="footnote-back">↩</a></li>
<li id="fn55"></li>
<li id="fn56"><a href="#fnref56" class="footnote-back">↩</a></li>
<li id="fn57"></li>
<li id="fn58"><a href="#fnref58" class="footnote-back">↩</a></li>
<li id="fn59"><a href="#fnref59" class="footnote-back">↩</a></li>
<li id="fn60"><a href="#fnref60" class="footnote-back">↩</a></li>
<li id="fn61"><a href="#fnref61" class="footnote-back">↩</a></li>
<li id="fn62"><a href="#fnref62" class="footnote-back">↩</a></li>
<li id="fn63"><a href="#fnref63" class="footnote-back">↩</a></li>
<li id="fn64"><a href="#fnref64" class="footnote-back">↩</a></li>
<li id="fn65"><a href="#fnref65" class="footnote-back">↩</a></li>
<li id="fn66"><a href="#fnref66" class="footnote-back">↩</a></li>
<li id="fn67"><a href="#fnref67" class="footnote-back">↩</a></li>
<li id="fn68"><a href="#fnref68" class="footnote-back">↩</a></li>
<li id="fn69"><a href="#fnref69" class="footnote-back">↩</a></li>
<li id="fn70"><a href="#fnref70" class="footnote-back">↩</a></li>
<li id="fn71"><a href="#fnref71" class="footnote-back">↩</a></li>
<li id="fn72"><a href="#fnref72" class="footnote-back">↩</a></li>
<li id="fn73"><a href="#fnref73" class="footnote-back">↩</a></li>
<li id="fn74"><a href="#fnref74" class="footnote-back">↩</a></li>
<li id="fn75"><a href="#fnref75" class="footnote-back">↩</a></li>
<li id="fn76"><a href="#fnref76" class="footnote-back">↩</a></li>
<li id="fn77"><a href="#fnref77" class="footnote-back">↩</a></li>
<li id="fn78"><a href="#fnref78" class="footnote-back">↩</a></li>
<li id="fn79"><a href="#fnref79" class="footnote-back">↩</a></li>
<li id="fn80"><a href="#fnref80" class="footnote-back">↩</a></li>
<li id="fn81"><a href="#fnref81" class="footnote-back">↩</a></li>
<li id="fn82"></li>
<li id="fn83"><a href="#fnref83" class="footnote-back">↩</a></li>
<li id="fn84"><a href="#fnref84" class="footnote-back">↩</a></li>
<li id="fn85"><a href="#fnref85" class="footnote-back">↩</a></li>
<li id="fn86"><a href="#fnref86" class="footnote-back">↩</a></li>
<li id="fn87"><a href="#fnref87" class="footnote-back">↩</a></li>
<li id="fn88"><a href="#fnref88" class="footnote-back">↩</a></li>
<li id="fn89"><a href="#fnref89" class="footnote-back">↩</a></li>
<li id="fn90"><a href="#fnref90" class="footnote-back">↩</a></li>
<li id="fn91"><a href="#fnref91" class="footnote-back">↩</a></li>
<li id="fn92"><a href="#fnref92" class="footnote-back">↩</a></li>
<li id="fn93"></li>
<li id="fn94"><a href="#fnref94" class="footnote-back">↩</a></li>
<li id="fn95"></li>
<li id="fn96"></li>
<li id="fn97"><a href="#fnref97" class="footnote-back">↩</a></li>
<li id="fn98"></li>
<li id="fn99"><a href="#fnref99" class="footnote-back">↩</a></li>
<li id="fn100"></li>
<li id="fn101"><a href="#fnref101" class="footnote-back">↩</a></li>
<li id="fn102"><a href="#fnref102" class="footnote-back">↩</a></li>
<li id="fn103"><a href="#fnref103" class="footnote-back">↩</a></li>
<li id="fn104"><a href="#fnref104" class="footnote-back">↩</a></li>
<li id="fn105"><a href="#fnref105" class="footnote-back">↩</a></li>
<li id="fn106"><a href="#fnref106" class="footnote-back">↩</a></li>
<li id="fn107"><a href="#fnref107" class="footnote-back">↩</a></li>
<li id="fn108"><a href="#fnref108" class="footnote-back">↩</a></li>
<li id="fn109"><a href="#fnref109" class="footnote-back">↩</a></li>
<li id="fn110"><a href="#fnref110" class="footnote-back">↩</a></li>
<li id="fn111"><a href="#fnref111" class="footnote-back">↩</a></li>
<li id="fn112"><a href="#fnref112" class="footnote-back">↩</a></li>
<li id="fn113"><a href="#fnref113" class="footnote-back">↩</a></li>
<li id="fn114"><a href="#fnref114" class="footnote-back">↩</a></li>
<li id="fn115"><a href="#fnref115" class="footnote-back">↩</a></li>
<li id="fn116"><a href="#fnref116" class="footnote-back">↩</a></li>
<li id="fn117"><a href="#fnref117" class="footnote-back">↩</a></li>
<li id="fn118"><a href="#fnref118" class="footnote-back">↩</a></li>
<li id="fn119"><a href="#fnref119" class="footnote-back">↩</a></li>
<li id="fn120"><a href="#fnref120" class="footnote-back">↩</a></li>
<li id="fn121"><a href="#fnref121" class="footnote-back">↩</a></li>
<li id="fn122"><a href="#fnref122" class="footnote-back">↩</a></li>
<li id="fn123"><a href="#fnref123" class="footnote-back">↩</a></li>
<li id="fn124"><a href="#fnref124" class="footnote-back">↩</a></li>
<li id="fn125"><a href="#fnref125" class="footnote-back">↩</a></li>
<li id="fn126"><a href="#fnref126" class="footnote-back">↩</a></li>
<li id="fn127"><a href="#fnref127" class="footnote-back">↩</a></li>
<li id="fn128"><a href="#fnref128" class="footnote-back">↩</a></li>
<li id="fn129"><a href="#fnref129" class="footnote-back">↩</a></li>
<li id="fn130"><a href="#fnref130" class="footnote-back">↩</a></li>
<li id="fn131"><a href="#fnref131" class="footnote-back">↩</a></li>
<li id="fn132"><a href="#fnref132" class="footnote-back">↩</a></li>
<li id="fn133"><a href="#fnref133" class="footnote-back">↩</a></li>
<li id="fn134"><a href="#fnref134" class="footnote-back">↩</a></li>
<li id="fn135"><a href="#fnref135" class="footnote-back">↩</a></li>
<li id="fn136"><a href="#fnref136" class="footnote-back">↩</a></li>
<li id="fn137"></li>
<li id="fn138"><a href="#fnref138" class="footnote-back">↩</a></li>
<li id="fn139"><a href="#fnref139" class="footnote-back">↩</a></li>
<li id="fn140"><a href="#fnref140" class="footnote-back">↩</a></li>
<li id="fn141"><a href="#fnref141" class="footnote-back">↩</a></li>
<li id="fn142"><a href="#fnref142" class="footnote-back">↩</a></li>
<li id="fn143"><a href="#fnref143" class="footnote-back">↩</a></li>
<li id="fn144"><a href="#fnref144" class="footnote-back">↩</a></li>
<li id="fn145"><a href="#fnref145" class="footnote-back">↩</a></li>
<li id="fn146"><a href="#fnref146" class="footnote-back">↩</a></li>
<li id="fn147"><a href="#fnref147" class="footnote-back">↩</a></li>
<li id="fn148"><a href="#fnref148" class="footnote-back">↩</a></li>
<li id="fn149"><a href="#fnref149" class="footnote-back">↩</a></li>
<li id="fn150"><a href="#fnref150" class="footnote-back">↩</a></li>
<li id="fn151"><a href="#fnref151" class="footnote-back">↩</a></li>
<li id="fn152"><a href="#fnref152" class="footnote-back">↩</a></li>
<li id="fn153"><a href="#fnref153" class="footnote-back">↩</a></li>
<li id="fn154"><a href="#fnref154" class="footnote-back">↩</a></li>
<li id="fn155"><a href="#fnref155" class="footnote-back">↩</a></li>
<li id="fn156"><a href="#fnref156" class="footnote-back">↩</a></li>
<li id="fn157"><a href="#fnref157" class="footnote-back">↩</a></li>
<li id="fn158"></li>
<li id="fn159"></li>
</ol>
</section>
