
<p><img src="DNA_Overview.png" title="fig:The structure of part of a DNA double helix" alt="The structure of part of a DNA double helix" width="220" /> <strong>Deoxyribonucleic acid</strong>, or <strong>DNA</strong> is a <a href="nucleic_acid" title="wikilink">nucleic acid</a> molecule that contains the <a href="genetics" title="wikilink">genetic</a> instructions used in the <a href="developmental_biology" title="wikilink">development</a> and functioning of all <a href="life" title="wikilink">living organisms</a>. The main role of DNA is the long-term storage of information and it is often compared to a set of blueprints, since DNA contains the instructions needed to construct other components of <a href="cell_(biology)" title="wikilink">cells</a>, such as <a href="protein" title="wikilink">proteins</a> and <a href="RNA" class="uri" title="wikilink">RNA</a> <a href="molecule" title="wikilink">molecules</a>. The DNA segments that carry this genetic information are called <a href="gene" title="wikilink">genes</a>, but other DNA sequences have structural purposes, or are involved in regulating the use of this genetic information.</p>
<p>Chemically, DNA is a long <a href="polymer" class="uri" title="wikilink">polymer</a> of simple units called <a href="nucleotide" title="wikilink">nucleotides</a>, which are held together by a backbone made of alternating <a href="carbohydrate" title="wikilink">sugars</a> and <a href="phosphate" class="uri" title="wikilink">phosphate</a> groups. Attached to each sugar is one of four types of molecules called <a href="nucleobase" title="wikilink">bases</a>. It is the sequence of these four bases along the backbone that encodes information. This information is read using the <a href="genetic_code" title="wikilink">genetic code</a>, which specifies the sequence of the <a href="amino_acid" title="wikilink">amino acids</a> within proteins. The code is read by copying stretches of DNA into the related nucleic acid <a href="RNA" class="uri" title="wikilink">RNA</a>, in a process called <a href="transcription_(genetics)" title="wikilink">transcription</a>. Most of these RNA molecules are used to synthesize proteins, but others are used directly in structures such as <a href="ribosome" title="wikilink">ribosomes</a> and <a href="spliceosome" title="wikilink">spliceosomes</a>.</p>
<p>Within cells, DNA is organized into structures called <a href="chromosome" title="wikilink">chromosomes</a> and the set of chromosomes within a cell make up a <a href="genome" class="uri" title="wikilink">genome</a>. These chromosomes are duplicated before cells <a href="cell_division" title="wikilink">divide</a>, in a process called <a href="DNA_replication" title="wikilink">DNA replication</a>. <a href="Eukaryote" title="wikilink">Eukaryotic organisms</a> such as <a href="animal" title="wikilink">animals</a>, <a href="plant" title="wikilink">plants</a>, and <a href="fungi" class="uri" title="wikilink">fungi</a> store their DNA inside the <a href="cell_nucleus" title="wikilink">cell nucleus</a>, while in <a href="prokaryote" title="wikilink">prokaryotes</a> such as <a href="bacteria" class="uri" title="wikilink">bacteria</a> it is found in the cell's <a href="cytoplasm" class="uri" title="wikilink">cytoplasm</a>. Within the chromosomes, <a href="chromatin" class="uri" title="wikilink">chromatin</a> proteins such as <a href="histone" title="wikilink">histones</a> compact and organize DNA, which helps control its interactions with other proteins and thereby control which <a href="genes" class="uri" title="wikilink">genes</a> are transcribed.</p>
<h2 id="physical-and-chemical-properties">Physical and chemical properties</h2>
<figure>
<img src="DNA_chemical_structure.svg" title="The chemical structure of DNA." alt="The chemical structure of DNA." width="350" /><figcaption>The chemical structure of DNA.</figcaption>
</figure>
<p>DNA is a long <a href="polymer" class="uri" title="wikilink">polymer</a> made from repeating units called <a href="nucleotide" title="wikilink">nucleotides</a>.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a><a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> The DNA chain is 22 to 24 <a href="&lt;U+00C5&gt;ngstr&lt;U+00F6&gt;m" title="wikilink">&lt;U+00C5&gt;ngstr&lt;U+00F6&gt;ms</a> wide (2.2 to 2.4 <a href="nanometre" title="wikilink">nanometres</a>), and one nucleotide unit is 3.3 &lt;U+00C5&gt;ngstroms (0.33 nanometres) long.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a> Although each individual repeating unit is very small, DNA polymers can be enormous molecules containing millions of nucleotides. For instance, the largest human <a href="chromosome" class="uri" title="wikilink">chromosome</a>, chromosome number 1, is 220 million <a href="base_pair" title="wikilink">base pairs</a> long.<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a></p>
<p>In living organisms, DNA does not usually exist as a single molecule, but instead as a tightly-associated pair of molecules.<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a><a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> These two long strands entwine like vines, in the shape of a <a href="helix" title="wikilink">double helix</a>. The nucleotide repeats contain both the backbone of the molecule, which holds the chain together, and a base, which interacts with the other DNA strand in the helix. In general, a base linked to a sugar is called a <a href="nucleoside" class="uri" title="wikilink">nucleoside</a> and a base linked to a sugar and one or more phosphate groups is called a <a href="nucleotide" class="uri" title="wikilink">nucleotide</a>. If multiple nucleotides are linked together, as in DNA, this polymer is referred to as a <a href="polynucleotide" class="uri" title="wikilink">polynucleotide</a>.<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a></p>
<p>The backbone of the DNA strand is made from alternating <a href="phosphate" class="uri" title="wikilink">phosphate</a> and <a href="carbohydrate" title="wikilink">sugar</a> residues.<a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a> The sugar in DNA is 2-deoxyribose, which is a <a href="pentose" class="uri" title="wikilink">pentose</a> (five <a href="carbon" class="uri" title="wikilink">carbon</a>) sugar. The sugars are joined together by phosphate groups that form <a href="phosphodiester_bond" title="wikilink">phosphodiester bonds</a> between the third and fifth carbon <a href="atom" title="wikilink">atoms</a> of adjacent sugar rings. These asymmetric <a href="covalent_bond" title="wikilink">bonds</a> mean a strand of DNA has a direction. In a double helix the direction of the nucleotides in one strand is opposite to their direction in the other strand. This arrangement of DNA strands is called antiparallel. The asymmetric ends of a strand of DNA bases are referred to as the <a href="5&#39;_end" title="wikilink">5'</a> (<em>five prime</em>) and <a href="3&#39;_end" title="wikilink">3'</a> (<em>three prime</em>) ends. One of the major differences between DNA and RNA is the sugar, with 2-deoxyribose being replaced by the alternative pentose sugar <a href="ribose" class="uri" title="wikilink">ribose</a> in RNA.<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a></p>
<p>The DNA double helix is stabilized by <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> between the bases attached to the two strands. The four bases found in DNA are <a href="adenine" class="uri" title="wikilink">adenine</a> (abbreviated A), <a href="cytosine" class="uri" title="wikilink">cytosine</a> (C), <a href="guanine" class="uri" title="wikilink">guanine</a> (G) and <a href="thymine" class="uri" title="wikilink">thymine</a> (T). These four bases are shown below and are attached to the sugar/phosphate to form the complete nucleotide, as shown for adenosine monophosphate.</p>
<div class="thumb tright" style="background-color: #f9f9f9; border: 1px solid #CCCCCC; margin:0.5em;">
<table>
<tbody>
<tr class="odd">
<td><figure>
<img src="Adenine_chemical_structure.png" title="Adenine_chemical_structure.png" alt="Adenine_chemical_structure.png" width="80" /><figcaption>Adenine_chemical_structure.png</figcaption>
</figure></td>
<td><figure>
<img src="Guanine_chemical_structure.png" title="Guanine_chemical_structure.png" alt="Guanine_chemical_structure.png" width="118" /><figcaption>Guanine_chemical_structure.png</figcaption>
</figure></td>
<td><figure>
<img src="Thymine_chemical_structure.png" title="Thymine_chemical_structure.png" alt="Thymine_chemical_structure.png" width="97" /><figcaption>Thymine_chemical_structure.png</figcaption>
</figure></td>
<td><figure>
<img src="Cytosine_chemical_structure.png" title="Cytosine_chemical_structure.png" alt="Cytosine_chemical_structure.png" width="75" /><figcaption>Cytosine_chemical_structure.png</figcaption>
</figure></td>
<td><figure>
<img src="AMP_chemical_structure.png" title="AMP_chemical_structure.png" alt="AMP_chemical_structure.png" width="130" /><figcaption>AMP_chemical_structure.png</figcaption>
</figure></td>
</tr>
<tr class="even">
<td><p><a href="Adenine" class="uri" title="wikilink">Adenine</a></p></td>
<td><p><a href="Guanine" class="uri" title="wikilink">Guanine</a></p></td>
<td><p><a href="Thymine" class="uri" title="wikilink">Thymine</a></p></td>
<td><p><a href="Cytosine" class="uri" title="wikilink">Cytosine</a></p></td>
<td><p><a href="Adenosine_monophosphate" title="wikilink">Adenosine monophosphate</a></p></td>
</tr>
</tbody>
</table>
<div style="border: none; width:500px;">
<div class="thumbcaption">
<p>Structures of the four bases found in DNA and the nucleotide adenosine monophosphate</p>
</div>
</div>
</div>
<p>These bases are classified into two types; adenine and guanine are fused five- and six-membered <a href="heterocyclic_compound" title="wikilink">heterocyclic compounds</a> called <a href="purine" title="wikilink">purines</a>, while cytosine and thymine are six-membered rings called <a href="pyrimidine" title="wikilink">pyrimidines</a>.<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a> A fifth pyrimidine base, called <a href="uracil" class="uri" title="wikilink">uracil</a> (U), usually replaces thymine in RNA and differs from thymine by lacking a <a href="methyl_group" title="wikilink">methyl group</a> on its ring. Uracil is normally only found in DNA as a breakdown product of cytosine, but a very rare exception to this rule is a <a href="phage" title="wikilink">bacterial virus</a> called PBS1 that contains uracil in its DNA.<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> In contrast, following RNA synthesis a significant number of the uracils are converted to thymines by the enzymatic addition of the missing methyl group. This occurs mostly on structural and enzymatic RNAs like <a href="transfer_RNA" title="wikilink">transfer RNAs</a> and <a href="ribosomal_RNA" title="wikilink">ribosomal RNA</a>.<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a></p>
<figure>
<img src="DNA_orbit_animated_small.gif" title="Animation of the structure of a section of DNA. The bases lie horizontally between the two spiraling strands. Large version" alt="Animation of the structure of a section of DNA. The bases lie horizontally between the two spiraling strands. Large version" /><figcaption>Animation of the structure of a section of DNA. The bases lie horizontally between the two spiraling strands. <a href=":Image:DNA_orbit_animated.gif" title="wikilink">Large version</a><a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a></figcaption>
</figure>
<p>The double helix is a right-handed spiral. As the DNA strands wind around each other, they leave gaps between each set of phosphate backbones, revealing the sides of the bases inside (see animation). There are two of these grooves twisting around the surface of the double helix: one groove is 22 &lt;U+00C5&gt; wide and the other is 12 &lt;U+00C5&gt; wide.<a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a> The larger groove is called the major groove, while the smaller, narrower groove is called the minor groove. The narrowness of the minor groove means that the edges of the bases are more accessible in the major groove. As a result, proteins like <a href="transcription_factor" title="wikilink">transcription factors</a> that can bind to specific sequences in double-stranded DNA usually read the sequence by making contacts to the sides of the bases exposed in the major groove.<a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a></p>
<div class="thumb tleft" style="background-color: #f9f9f9; border: 1px solid #CCCCCC; margin:0.5em;">
<table>
<tbody>
<tr class="odd">
<td><figure>
<img src="GC_Watson_Crick_basepair.png" title="GC_Watson_Crick_basepair.png" alt="GC_Watson_Crick_basepair.png" width="230" /><figcaption>GC_Watson_Crick_basepair.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>
<table>
<tbody>
<tr class="odd">
<td><figure>
<img src="AT_Watson_Crick_basepair.png" title="AT_Watson_Crick_basepair.png" alt="AT_Watson_Crick_basepair.png" width="230" /><figcaption>AT_Watson_Crick_basepair.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>
<div style="border: none; width:230px;">
<div class="thumbcaption">
<p>At top, a <strong>GC</strong> base pair with three <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a>. At the bottom, <strong>AT</strong> base pair with two hydrogen bonds. Hydrogen bonds are shown as dashed lines.</p>
</div>
</div>
</div>
<h3 id="base-pairing">Base pairing</h3>

<p>Each type of base on one strand forms a bond with just one type of base on the other strand. This is called complementary <a href="base_pair" title="wikilink">base pairing</a>. Here, purines form <a href="hydrogen_bond" title="wikilink">hydrogen bonds</a> to pyrimidines, with A bonding only to T, and C bonding only to G. This arrangement of two nucleotides joined together across the double helix is called a base pair. In a double helix, the two strands are also held together by <a href="force" title="wikilink">forces</a> generated by the <a href="hydrophobic_effect" title="wikilink">hydrophobic effect</a> and <a href="pi_stacking" title="wikilink">pi stacking</a>, but these forces are not affected by the sequence of the DNA.<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a> As hydrogen bonds are not <a href="covalent_bond" title="wikilink">covalent</a>, they can be broken and rejoined relatively easily. The two strands of DNA in a double helix can therefore be pulled apart like a zipper, either by a mechanical force or high <a href="temperature" class="uri" title="wikilink">temperature</a>.<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a> As a result of this complementarity, all the information in the double-stranded sequence of a DNA helix is duplicated on each strand, which is vital in DNA replication. Indeed, this reversible and specific interaction between complementary base pairs is critical for all the functions of DNA in living organisms.<a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a></p>
<p>The two types of base pairs form different numbers of hydrogen bonds, AT forming two hydrogen bonds, and GC forming three hydrogen bonds (see figures, left). The GC base pair is therefore stronger than the AT base pair. As a result, it is both the percentage of GC base pairs and the overall length of a DNA double helix that determine the strength of the association between the two strands of DNA. Long DNA helices with a high GC content have strongly interacting strands, while short helices with high AT content have weakly interacting strands.<a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a> Parts of the DNA double helix that need to separate easily, such as the TATAAT <a href="Pribnow_box" title="wikilink">Pribnow box</a> in bacterial <a href="promoter" title="wikilink">promoters</a>, tend to have sequences with a high AT content, making the strands easier to pull apart.<a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a> In the laboratory, the strength of this interaction can be measured by finding the temperature required to break the hydrogen bonds, their <a href="melting_temperature" title="wikilink">melting temperature</a> (also called <em>T<sub>m</sub></em> value). When all the base pairs in a DNA double helix melt, the strands separate and exist in solution as two entirely independent molecules. These single-stranded DNA molecules have no single shape, but some conformations are more stable than others.<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a></p>
<h3 id="sense-and-antisense">Sense and antisense</h3>

<p>A DNA sequence is called &quot;sense&quot; if its sequence is the same as that of a <a href="messenger_RNA" title="wikilink">messenger RNA</a> (mRNA) copy that is translated into protein. The sequence on the opposite strand is complementary to the sense sequence and is therefore called the &quot;antisense&quot; sequence. Since <a href="RNA_polymerase" title="wikilink">RNA polymerases</a> work by making a complementary copy of their templates, it is this antisense strand that is the template for producing the sense mRNA. Both sense and antisense sequences can exist on different parts of the same strand of DNA. In both prokaryotes and eukaryotes, antisense sequences are transcribed, but the functions of these RNAs are not entirely clear.<a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a> One proposal is that antisense RNAs are involved in regulating <a href="gene_expression" title="wikilink">gene expression</a> through RNA-RNA base pairing.<a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a></p>
<p>A few DNA sequences in prokaryotes and eukaryotes, and more in <a href="plasmid" title="wikilink">plasmids</a> and <a href="virus" title="wikilink">viruses</a>, blur the distinction made above between sense and antisense strands by having overlapping genes.<a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a> In these cases, some DNA sequences do double duty, encoding one protein when read 5' to 3' along one strand, and a second protein when read in the opposite direction (still 5' to 3') along the other strand. In <a href="bacteria" class="uri" title="wikilink">bacteria</a>, this overlap may be involved in the regulation of gene transcription,<a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a> while in viruses, overlapping genes increase the amount of information that can be encoded within the small viral genome.<a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a> Another way of reducing genome size is seen in some viruses that contain linear or circular single-stranded DNA as their genetic material.<a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a><a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a></p>
<h3 id="supercoiling">Supercoiling</h3>

<p>DNA can be twisted like a rope in a process called <a href="DNA_supercoil" title="wikilink">DNA supercoiling</a>. Normally, with DNA in its &quot;relaxed&quot; state, a strand circles the axis of the double helix once every 10.4 base pairs, but if the DNA is twisted the strands become more tightly or more loosely wound.<a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a> If the DNA is twisted in the direction of the helix, this is positive supercoiling, and the bases are held more tightly together. If they are twisted in the opposite direction, this is negative supercoiling, and the bases come apart more easily. In nature, most DNA has slight negative supercoiling that is introduced by enzymes called <a href="topoisomerase" title="wikilink">topoisomerases</a>.<a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a> These enzymes are also needed to relieve the twisting stresses introduced into DNA strands during processes such as <a href="transcription_(genetics)" title="wikilink">transcription</a> and <a href="DNA_replication" title="wikilink">DNA replication</a>.<a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a></p>
<figure>
<img src="A-DNA,_B-DNA_and_Z-DNA.png" title="From left to right, the structures of A, B and Z DNA" alt="From left to right, the structures of A, B and Z DNA" width="290" /><figcaption>From left to right, the structures of A, B and Z DNA</figcaption>
</figure>
<h3 id="alternative-double-helical-structures">Alternative double-helical structures</h3>

<p>DNA exists in several possible <a href="Conformational_isomerism" title="wikilink">conformations</a>. The conformations so far identified are: <a href="A-DNA" class="uri" title="wikilink">A-DNA</a>, B-DNA, C-DNA, D-DNA,<a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a> E-DNA,<a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a> H-DNA,<a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a> L-DNA,<a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a>, P-DNA<a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a>, and <a href="Z-DNA" class="uri" title="wikilink">Z-DNA</a>.<a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a><a href="#fn38" class="footnote-ref" id="fnref38"><sup>38</sup></a> However, only A-DNA, B-DNA, and Z-DNA have been observed in naturally occurring biological systems. Which conformation DNA adopts depends on the sequence of the DNA, the amount and direction of supercoiling, chemical modifications of the bases and also solution conditions, such as the concentration of <a href="metal" class="uri" title="wikilink">metal</a> <a href="ion" title="wikilink">ions</a> and <a href="polyamine" title="wikilink">polyamines</a>.<a href="#fn39" class="footnote-ref" id="fnref39"><sup>39</sup></a> Of these three conformations, the &quot;B&quot; form described above is most common under the conditions found in cells. The two alternative double-helical forms of DNA differ in their geometry and dimensions.</p>
<p>The A form is a wider right-handed spiral, with a shallow and wide minor groove and a narrower and deeper major groove. The A form occurs under non-physiological conditions in dehydrated samples of DNA, while in the cell it may be produced in hybrid pairings of DNA and RNA strands.<a href="#fn40" class="footnote-ref" id="fnref40"><sup>40</sup></a> Segments of DNA where the bases have been chemically-modified by <a href="methylation" class="uri" title="wikilink">methylation</a> may undergo a larger change in conformation and adopt the <a href="Z-DNA" title="wikilink">Z form</a>. Here, the strands turn about the helical axis in a left-handed spiral, the opposite of the more common B form.<a href="#fn41" class="footnote-ref" id="fnref41"><sup>41</sup></a> These unusual structures can be recognised by specific Z-DNA binding proteins and may be involved in the regulation of transcription.<a href="#fn42" class="footnote-ref" id="fnref42"><sup>42</sup></a></p>
<figure>
<img src="Telomere_quadruplex.jpg" title="Structure of a DNA quadruplex formed by telomere repeats." alt="Structure of a DNA quadruplex formed by telomere repeats." width="300" /><figcaption>Structure of a DNA quadruplex formed by <a href="telomere" class="uri" title="wikilink">telomere</a> repeats.<a href="#fn43" class="footnote-ref" id="fnref43"><sup>43</sup></a></figcaption>
</figure>
<h3 id="quadruplex-structures">Quadruplex structures</h3>
<p>At the ends of the linear <a href="chromosome" title="wikilink">chromosomes</a> are specialized regions of DNA called <a href="telomere" title="wikilink">telomeres</a>. The main function of these regions is to allow the cell to replicate chromosome ends using the enzyme <a href="telomerase" class="uri" title="wikilink">telomerase</a>, as the enzymes that normally replicate DNA cannot copy the extreme 3' ends of chromosomes.<a href="#fn44" class="footnote-ref" id="fnref44"><sup>44</sup></a> As a result, if a chromosome lacked telomeres it would become shorter each time it was replicated. These specialized chromosome caps also help protect the DNA ends from <a href="exonuclease" title="wikilink">exonucleases</a> and stop the <a href="DNA_repair" title="wikilink">DNA repair</a> systems in the cell from treating them as damage to be corrected.<a href="#fn45" class="footnote-ref" id="fnref45"><sup>45</sup></a> In human cells, telomeres are usually lengths of single-stranded DNA containing several thousand repeats of a simple TTAGGG sequence.<a href="#fn46" class="footnote-ref" id="fnref46"><sup>46</sup></a></p>
<p>These guanine-rich sequences may stabilize chromosome ends by forming very unusual structures of stacked sets of four-base units, rather than the usual base pairs found in other DNA molecules. Here, four guanine bases form a flat plate and these flat four-base units then stack on top of each other, to form a stable <em>quadruplex</em> structure.<a href="#fn47" class="footnote-ref" id="fnref47"><sup>47</sup></a> These structures are stabilized by hydrogen bonding between the edges of the bases and <a href="chelation" class="uri" title="wikilink">chelation</a> of a metal ion in the centre of each four-base unit. The structure shown to the left is a top view of the quadruplex formed by a DNA sequence found in human telomere repeats. The single DNA strand forms a loop, with the sets of four bases stacking in a central quadruplex three plates deep. In the space at the centre of the stacked bases are three chelated <a href="potassium" class="uri" title="wikilink">potassium</a> ions.<a href="#fn48" class="footnote-ref" id="fnref48"><sup>48</sup></a> Other structures can also be formed, with the central set of four bases coming from either a single strand folded around the bases, or several different parallel strands, each contributing one base to the central structure.</p>
<p>In addition to these stacked structures, telomeres also form large loop structures called telomere loops, or T-loops. Here, the single-stranded DNA curls around in a long circle stabilized by telomere-binding proteins.<a href="#fn49" class="footnote-ref" id="fnref49"><sup>49</sup></a> At the very end of the T-loop, the single-stranded telomere DNA is held onto a region of double-stranded DNA by the telomere strand disrupting the double-helical DNA and base pairing to one of the two strands. This triple-stranded structure is called a displacement loop or D-loop.<a href="#fn50" class="footnote-ref" id="fnref50"><sup>50</sup></a></p>
<h2 id="chemical-modifications">Chemical modifications</h2>
<div class="thumb tright" style="background-color: #f9f9f9; border: 1px solid #CCCCCC; margin:0.5em;">
<table>
<tbody>
<tr class="odd">
<td><figure>
<img src="Cytosine_chemical_structure.png" title="Cytosine_chemical_structure.png" alt="Cytosine_chemical_structure.png" width="75" /><figcaption>Cytosine_chemical_structure.png</figcaption>
</figure></td>
<td><figure>
<img src="5-methylcytosine.png" title="5-methylcytosine.png" alt="5-methylcytosine.png" width="95" /><figcaption>5-methylcytosine.png</figcaption>
</figure></td>
<td><figure>
<img src="Thymine_chemical_structure.png" title="Thymine_chemical_structure.png" alt="Thymine_chemical_structure.png" width="97" /><figcaption>Thymine_chemical_structure.png</figcaption>
</figure></td>
</tr>
<tr class="even">
<td><p><a href="cytosine" class="uri" title="wikilink">cytosine</a></p></td>
<td><p><a href="5-Methylcytosine" title="wikilink">5-methylcytosine</a></p></td>
<td><p><a href="thymine" class="uri" title="wikilink">thymine</a></p></td>
</tr>
</tbody>
</table>
<div style="border: none; width:300px;font-size: 85%;">
<div class="thumbcaption">
<p>Structure of cytosine with and without the 5-methyl group. After deamination the 5-methylcytosine has the same structure as thymine</p>
</div>
</div>
</div>
<h3 id="base-modifications">Base modifications</h3>

<p>The expression of genes is influenced by the <a href="chromatin" class="uri" title="wikilink">chromatin</a> structure of a chromosome and regions of <a href="heterochromatin" class="uri" title="wikilink">heterochromatin</a> (low or no gene expression) correlate with the <a href="methylation" class="uri" title="wikilink">methylation</a> of <a href="cytosine" class="uri" title="wikilink">cytosine</a>. For example, cytosine methylation, to produce <a href="5-Methylcytosine" title="wikilink">5-methylcytosine</a>, is important for <a href="X-inactivation" title="wikilink">X-chromosome inactivation</a>.<a href="#fn51" class="footnote-ref" id="fnref51"><sup>51</sup></a> The average level of methylation varies between organisms, with <em><a href="Caenorhabditis_elegans" title="wikilink">Caenorhabditis elegans</a></em> lacking cytosine methylation, while <a href="vertebrate" title="wikilink">vertebrates</a> show higher levels, with up to 1% of their DNA containing 5-methylcytosine.<a href="#fn52" class="footnote-ref" id="fnref52"><sup>52</sup></a> Despite the biological role of 5-methylcytosine it is susceptible to spontaneous <a href="deamination" class="uri" title="wikilink">deamination</a> to leave the thymine base, and methylated cytosines are therefore <a href="mutation" class="uri" title="wikilink">mutation</a> hotspots.<a href="#fn53" class="footnote-ref" id="fnref53"><sup>53</sup></a> Other base modifications include adenine methylation in bacteria and the <a href="glycosylation" class="uri" title="wikilink">glycosylation</a> of uracil to produce the &quot;J-base&quot; in <a href="kinetoplastid" title="wikilink">kinetoplastids</a>.<a href="#fn54" class="footnote-ref" id="fnref54"><sup>54</sup></a><a href="#fn55" class="footnote-ref" id="fnref55"><sup>55</sup></a></p>
<h3 id="dna-damage">DNA damage</h3>

<p><img src="Benzopyrene_DNA_adduct_1JDG.png" title="fig:Benzopyrene, the major mutagen in tobacco smoke, in an adduct to DNA." alt="Benzopyrene, the major mutagen in tobacco smoke, in an adduct to DNA." width="250" /> DNA can be damaged by many different sorts of <a href="mutagen" title="wikilink">mutagens</a>. These include <a href="oxidizing_agent" title="wikilink">oxidizing agents</a>, <a href="alkylating_agent" title="wikilink">alkylating agents</a> and also high-energy <a href="electromagnetic_radiation" title="wikilink">electromagnetic radiation</a> such as <a href="ultraviolet" class="uri" title="wikilink">ultraviolet</a> light and <a href="x-ray" title="wikilink">x-rays</a>. The type of DNA damage produced depends on the type of mutagen. For example, UV light mostly damages DNA by producing <a href="thymine_dimer" title="wikilink">thymine dimers</a>, which are cross-links between adjacent pyrimidine bases in a DNA strand.<a href="#fn56" class="footnote-ref" id="fnref56"><sup>56</sup></a> On the other hand, oxidants such as <a href="free_radical" title="wikilink">free radicals</a> or <a href="hydrogen_peroxide" title="wikilink">hydrogen peroxide</a> produce multiple forms of damage, including base modifications, particularly of guanosine, as well as double-strand breaks.<a href="#fn57" class="footnote-ref" id="fnref57"><sup>57</sup></a> It has been estimated that in each human cell, about 500 bases suffer oxidative damage per day.<a href="#fn58" class="footnote-ref" id="fnref58"><sup>58</sup></a><a href="#fn59" class="footnote-ref" id="fnref59"><sup>59</sup></a> Of these oxidative lesions, the most dangerous are double-strand breaks, as these lesions are difficult to repair and can produce <a href="point_mutation" title="wikilink">point mutations</a>, insertions and deletions from the DNA sequence, as well as <a href="chromosomal_translocation" title="wikilink">chromosomal translocations</a>.<a href="#fn60" class="footnote-ref" id="fnref60"><sup>60</sup></a></p>
<p>Many mutagens <a href="intercalation_(chemistry)" title="wikilink">intercalate</a> into the space between two adjacent base pairs. Intercalators are mostly <a href="aromaticity" title="wikilink">aromatic</a> and planar molecules, and include <a href="ethidium" class="uri" title="wikilink">ethidium</a>, <a href="daunomycin" class="uri" title="wikilink">daunomycin</a>, <a href="doxorubicin" class="uri" title="wikilink">doxorubicin</a> and <a href="thalidomide" class="uri" title="wikilink">thalidomide</a>. In order for an intercalator to fit between base pairs, the bases must separate, distorting the DNA strands by unwinding of the double helix. These structural changes inhibit both transcription and DNA replication, causing toxicity and mutations. As a result, DNA intercalators are often <a href="carcinogen" title="wikilink">carcinogens</a>, with <a href="benzopyrene" title="wikilink">benzopyrene diol epoxide</a>, <a href="acridine" title="wikilink">acridines</a>, <a href="aflatoxin" class="uri" title="wikilink">aflatoxin</a> and <a href="ethidium_bromide" title="wikilink">ethidium bromide</a> being well-known examples.<a href="#fn61" class="footnote-ref" id="fnref61"><sup>61</sup></a><a href="#fn62" class="footnote-ref" id="fnref62"><sup>62</sup></a><a href="#fn63" class="footnote-ref" id="fnref63"><sup>63</sup></a>Nevertheless, due to their properties of inhibiting DNA transcription and replication, they are also used in <a href="chemotherapy" class="uri" title="wikilink">chemotherapy</a> to inhibit rapidly-growing <a href="cancer" class="uri" title="wikilink">cancer</a> cells.<a href="#fn64" class="footnote-ref" id="fnref64"><sup>64</sup></a></p>
<h2 id="overview-of-biological-functions">Overview of biological functions</h2>
<p>The DNA is usually in linear <a href="chromosome" title="wikilink">chromosomes</a> in eukaryotes, and circular chromosomes in prokaryotes. The set of chromosomes in a cell makes up its <a href="genome" class="uri" title="wikilink">genome</a>; the <a href="human_genome" title="wikilink">human genome</a> has approximately 3 billion base pairs of DNA arranged into 46 chromosomes.<a href="#fn65" class="footnote-ref" id="fnref65"><sup>65</sup></a> The information carried by DNA is held in the <a href="DNA_sequence" title="wikilink">sequence</a> of pieces of DNA called <a href="gene" title="wikilink">genes</a>. Genetic information in genes is transmitted through complementary base pairing. For example, when a cell uses the information in a gene, the DNA sequence is copied into a complementary RNA sequence in a process called transcription. Usually, this RNA copy is then used to make a matching protein sequence in a process called translation. Alternatively, a cell may simply copy its genetic information in a process called DNA replication. The details of these functions are covered in other articles; here we focus on the interactions between DNA and other molecules that mediate the function of the genome.</p>
<h3 id="genome-structure">Genome structure</h3>

<p>Genomic DNA is located in the <a href="cell_nucleus" title="wikilink">cell nucleus</a> of eukaryotes, as well as small amounts in <a href="mitochondrion" title="wikilink">mitochondria</a> and <a href="chloroplast" title="wikilink">chloroplasts</a>. In prokaryotes, the DNA is held within an irregularly shaped body in the cytoplasm called the <a href="nucleoid" class="uri" title="wikilink">nucleoid</a>.<a href="#fn66" class="footnote-ref" id="fnref66"><sup>66</sup></a> The genetic information in a genome is held within genes. A gene is a unit of <a href="heredity" class="uri" title="wikilink">heredity</a> and is a region of DNA that influences a particular characteristic in an organism. Genes contain an <a href="open_reading_frame" title="wikilink">open reading frame</a> that can be transcribed, as well as <a href="regulatory_sequence" title="wikilink">regulatory sequences</a> such as <a href="promoter" title="wikilink">promoters</a> and <a href="enhancer_(genetics)" title="wikilink">enhancers</a>, which control the expression of the open reading frame.</p>
<p>In many <a href="species" class="uri" title="wikilink">species</a>, only a small fraction of the total sequence of the <a href="genome" class="uri" title="wikilink">genome</a> encodes protein. For example, only about 1.5% of the human genome consists of protein-coding <a href="exon" title="wikilink">exons</a>, with over 50% of human DNA consisting of non-coding <a href="repeated_sequence_(DNA)" title="wikilink">repetitive sequences</a>.<a href="#fn67" class="footnote-ref" id="fnref67"><sup>67</sup></a> The reasons for the presence of so much <a href="noncoding_DNA" title="wikilink">non-coding DNA</a> in eukaryotic genomes and the extraordinary differences in <a href="genome_size" title="wikilink">genome size</a>, or <em><a href="C-value" class="uri" title="wikilink">C-value</a></em>, among species represent a long-standing puzzle known as the &quot;<a href="C-value_enigma" title="wikilink">C-value enigma</a>.&quot;<a href="#fn68" class="footnote-ref" id="fnref68"><sup>68</sup></a> <img src="RNA_pol.jpg" title="fig:T7 RNA polymerase producing a mRNA (green) from a DNA template (red and blue). The protein is shown as a purple ribbon." alt="T7 RNA polymerase producing a mRNA (green) from a DNA template (red and blue). The protein is shown as a purple ribbon." width="300" /> Some non-coding DNA sequences play structural roles in chromosomes. <a href="Telomere" title="wikilink">Telomeres</a> and <a href="centromere" title="wikilink">centromeres</a> typically contain few genes, but are important for the function and stability of chromosomes.<a href="#fn69" class="footnote-ref" id="fnref69"><sup>69</sup></a><a href="#fn70" class="footnote-ref" id="fnref70"><sup>70</sup></a> An abundant form of non-coding DNA in humans are <a href="pseudogene" title="wikilink">pseudogenes</a>, which are copies of genes that have been disabled by mutation.<a href="#fn71" class="footnote-ref" id="fnref71"><sup>71</sup></a> These sequences are usually just molecular <a href="fossil" title="wikilink">fossils</a>, although they can occasionally serve as raw genetic material for the creation of new genes through the process of <a href="gene_duplication" title="wikilink">gene duplication</a> and <a href="divergent_evolution" title="wikilink">divergence</a>.<a href="#fn72" class="footnote-ref" id="fnref72"><sup>72</sup></a></p>
<h3 id="transcription-and-translation">Transcription and translation</h3>

<p>A gene is a sequence of DNA that contains genetic information and can influence the <a href="phenotype" class="uri" title="wikilink">phenotype</a> of an organism. Within a gene, the sequence of bases along a DNA strand defines a <a href="messenger_RNA" title="wikilink">messenger RNA</a> sequence, which then defines a protein sequence. The relationship between the nucleotide sequences of genes and the <a href="amino_acid" title="wikilink">amino-acid</a> sequences of proteins is determined by the rules of <a href="translation_(genetics)" title="wikilink">translation</a>, known collectively as the <a href="genetic_code" title="wikilink">genetic code</a>. The genetic code consists of three-letter 'words' called <em>codons</em> formed from a sequence of three nucleotides (e.g. ACT, CAG, TTT).</p>
<p>In transcription, the codons of a gene are copied into messenger RNA by <a href="RNA_polymerase" title="wikilink">RNA polymerase</a>. This RNA copy is then decoded by a <a href="ribosome" class="uri" title="wikilink">ribosome</a> that reads the RNA sequence by base-pairing the messenger RNA to <a href="transfer_RNA" title="wikilink">transfer RNA</a>, which carries amino acids. Since there are 4 bases in 3-letter combinations, there are 64 possible codons (<span class="math inline">4<sup>3</sup></span> combinations). These encode the twenty <a href="list_of_standard_amino_acids" title="wikilink">standard amino acids</a>. Most amino acids, therefore, have more than one possible codon. There are also three 'stop' or 'nonsense' codons signifying the end of the coding region; these are the TAA, TGA and TAG codons.</p>
<figure>
<img src="DNA_replication.svg" title="DNA replication. The double helix is unwound by a helicase and topoisomerase. Next, one DNA polymerase produces the leading strand copy. Another DNA polymerase binds to the lagging strand. This enzyme makes discontinuous segments (called Okazaki fragments) before DNA ligase joins them together." alt="DNA replication. The double helix is unwound by a helicase and topoisomerase. Next, one DNA polymerase produces the leading strand copy. Another DNA polymerase binds to the lagging strand. This enzyme makes discontinuous segments (called Okazaki fragments) before DNA ligase joins them together." width="450" /><figcaption>DNA replication. The double helix is unwound by a <a href="helicase" class="uri" title="wikilink">helicase</a> and <a href="topoisomerase" class="uri" title="wikilink">topoisomerase</a>. Next, one <a href="DNA_polymerase" title="wikilink">DNA polymerase</a> produces the <a href="leading_strand" title="wikilink">leading strand</a> copy. Another DNA polymerase binds to the <a href="lagging_strand" title="wikilink">lagging strand</a>. This enzyme makes discontinuous segments (called <a href="Okazaki_fragment" title="wikilink">Okazaki fragments</a>) before <a href="DNA_ligase" title="wikilink">DNA ligase</a> joins them together.</figcaption>
</figure>
<h3 id="replication">Replication</h3>

<p><a href="Cell_division" title="wikilink">Cell division</a> is essential for an organism to grow, but when a cell divides it must replicate the DNA in its genome so that the two daughter cells have the same genetic information as their parent. The double-stranded structure of DNA provides a simple mechanism for <a href="DNA_replication" title="wikilink">DNA replication</a>. Here, the two strands are separated and then each strand's complementary DNA sequence is recreated by an enzyme called <a href="DNA_polymerase" title="wikilink">DNA polymerase</a>. This enzyme makes the complementary strand by finding the correct base through complementary base pairing, and bonding it onto the original strand. As DNA polymerases can only extend a DNA strand in a 5' to 3' direction, different mechanisms are used to copy the antiparallel strands of the double helix.<a href="#fn73" class="footnote-ref" id="fnref73"><sup>73</sup></a> In this way, the base on the old strand dictates which base appears on the new strand, and the cell ends up with a perfect copy of its DNA.</p>
<h2 id="interactions-with-proteins">Interactions with proteins</h2>
<p>All the functions of DNA depend on interactions with proteins. These protein interactions can either be non-specific or specific, where the protein can only bind to a particular DNA sequence. Enzymes can also bind to DNA and of these, the polymerases that copy the DNA base sequence in transcription and DNA replication are particularly important.</p>
<h3 id="dna-binding-proteins">DNA-binding proteins</h3>
<div class="thumb tleft" style="background-color: #f9f9f9; border: 1px solid #CCCCCC; margin:0.5em;">
<table>
<tbody>
<tr class="odd">
<td><figure>
<img src="Nucleosome_2.jpg" title="Nucleosome_2.jpg" alt="Nucleosome_2.jpg" width="260" /><figcaption>Nucleosome_2.jpg</figcaption>
</figure></td>
</tr>
<tr class="even">
<td><figure>
<img src="Nucleosome_(opposites_attracts).JPG" title="Nucleosome_(opposites_attracts).JPG" alt="Nucleosome_(opposites_attracts).JPG" width="260" /><figcaption>Nucleosome_(opposites_attracts).JPG</figcaption>
</figure></td>
</tr>
</tbody>
</table>
<div style="border: none; width:260px;">
<div class="thumbcaption">
<p>Interaction of DNA with <a href="histone" title="wikilink">histones</a> (shown in white, top). These proteins' basic amino acids (below left, blue) bind to the acidic phosphate groups on DNA (below right, red).</p>
</div>
</div>
</div>
<p>Structural proteins that bind DNA are well-understood examples of non-specific DNA-protein interactions. Within chromosomes, DNA is held in complexes between DNA and structural proteins. These proteins organize the DNA into a compact structure called <a href="chromatin" class="uri" title="wikilink">chromatin</a>. In eukaryotes this structure involves DNA binding to a complex of small basic proteins called <a href="histone" title="wikilink">histones</a>, while in prokaryotes multiple types of proteins are involved.<a href="#fn74" class="footnote-ref" id="fnref74"><sup>74</sup></a> The histones form a disk-shaped complex called a <a href="nucleosome" class="uri" title="wikilink">nucleosome</a>, which contains two complete turns of double-stranded DNA wrapped around its surface. These non-specific interactions are formed through basic residues in the histones making <a href="ionic_bond" title="wikilink">ionic bonds</a> to the acidic sugar-phosphate backbone of the DNA, and are therefore largely independent of the base sequence.<a href="#fn75" class="footnote-ref" id="fnref75"><sup>75</sup></a> Chemical modifications of these basic amino acid residues include <a href="methylation" class="uri" title="wikilink">methylation</a>, <a href="phosphorylation" class="uri" title="wikilink">phosphorylation</a> and <a href="acetylation" class="uri" title="wikilink">acetylation</a>.<a href="#fn76" class="footnote-ref" id="fnref76"><sup>76</sup></a> These chemical changes alter the strength of the interaction between the DNA and the histones, making the DNA more or less accessible to <a href="transcription_factor" title="wikilink">transcription factors</a> and changing the rate of transcription.<a href="#fn77" class="footnote-ref" id="fnref77"><sup>77</sup></a> Other non-specific DNA-binding proteins found in chromatin include the high-mobility group proteins, which bind preferentially to bent or distorted DNA.<a href="#fn78" class="footnote-ref" id="fnref78"><sup>78</sup></a> These proteins are important in bending arrays of nucleosomes and arranging them into more complex chromatin structures.<a href="#fn79" class="footnote-ref" id="fnref79"><sup>79</sup></a></p>
<p>A distinct group of DNA-binding proteins are the single-stranded DNA-binding proteins that specifically bind single-stranded DNA. In humans, replication protein A is the best-characterised member of this family and is essential for most processes where the double helix is separated, including DNA replication, recombination and DNA repair.<a href="#fn80" class="footnote-ref" id="fnref80"><sup>80</sup></a> These binding proteins seem to stabilize single-stranded DNA and protect it from forming <a href="stem_loop" title="wikilink">stem loops</a> or being degraded by <a href="nuclease" title="wikilink">nucleases</a>.</p>
<p><img src="Lambda_repressor_1LMB.png" title="fig:The lambda repressor helix-turn-helix transcription factor bound to its DNA target" alt="The lambda repressor helix-turn-helix transcription factor bound to its DNA target" width="185" /> In contrast, other proteins have evolved to specifically bind particular DNA sequences. The most intensively studied of these are the various classes of <a href="transcription_factor" title="wikilink">transcription factors</a>. These proteins control gene transcription. Each one of these proteins bind to one particular set of DNA sequences and thereby activates or inhibits the transcription of genes with these sequences close to their <a href="promoter" title="wikilink">promoters</a>. The transcription factors do this in two ways. Firstly, they can bind the RNA polymerase responsible for transcription, either directly or through other mediator proteins; this locates the polymerase at the promoter and allows it to begin transcription.<a href="#fn81" class="footnote-ref" id="fnref81"><sup>81</sup></a> Alternatively, transcription factors can bind <a href="enzyme" title="wikilink">enzymes</a> that modify the histones at the promoter; this will change the accessibility of the DNA template to the polymerase.<a href="#fn82" class="footnote-ref" id="fnref82"><sup>82</sup></a></p>
<p>As these DNA targets can occur throughout an organism's genome, changes in the activity of one type of transcription factor can affect thousands of genes.<a href="#fn83" class="footnote-ref" id="fnref83"><sup>83</sup></a> Consequently, these proteins are often the targets of the <a href="signal_transduction" title="wikilink">signal transduction</a> processes that mediate responses to environmental changes or cellular differentiation and development. The specificity of these transcription factors' interactions with DNA come from the proteins making multiple contacts to the edges of the DNA bases, allowing them to &quot;read&quot; the DNA sequence. Most of these base interactions are made in the major groove, where the bases are most accessible.<a href="#fn84" class="footnote-ref" id="fnref84"><sup>84</sup></a></p>
<figure>
<img src="EcoRV_1RVA.png" title="The restriction enzyme EcoRV (green) in a complex with its substrate DNA" alt="The restriction enzyme EcoRV (green) in a complex with its substrate DNA" width="250" /><figcaption>The <a href="restriction_enzyme" title="wikilink">restriction enzyme</a> <a href="EcoRV" class="uri" title="wikilink">EcoRV</a> (green) in a complex with its substrate DNA<a href="#fn85" class="footnote-ref" id="fnref85"><sup>85</sup></a></figcaption>
</figure>
<h3 id="dna-modifying-enzymes">DNA-modifying enzymes</h3>
<h4 id="nucleases-and-ligases">Nucleases and ligases</h4>
<p>Nucleases are <a href="enzyme" title="wikilink">enzymes</a> that cut DNA strands by catalyzing the <a href="hydrolysis" class="uri" title="wikilink">hydrolysis</a> of the <a href="phosphodiester_bond" title="wikilink">phosphodiester bonds</a>. Nucleases that hydrolyse nucleotides from the ends of DNA strands are called <a href="exonuclease" title="wikilink">exonucleases</a>, while <a href="endonuclease" title="wikilink">endonucleases</a> cut within strands. The most frequently-used nucleases in <a href="molecular_biology" title="wikilink">molecular biology</a> are the <a href="restriction_enzyme" title="wikilink">restriction endonucleases</a>, which cut DNA at specific sequences. For instance, the EcoRV enzyme shown to the left recognizes the 6-base sequence 5'-GAT|ATC-3' and makes a cut at the vertical line. In nature, these enzymes protect <a href="bacteria" class="uri" title="wikilink">bacteria</a> against <a href="phage" class="uri" title="wikilink">phage</a> infection by digesting the phage DNA when it enters the bacterial cell, acting as part of the <a href="restriction_modification_system" title="wikilink">restriction modification system</a>.<a href="#fn86" class="footnote-ref" id="fnref86"><sup>86</sup></a> In technology, these sequence-specific nucleases are used in <a href="clone_(genetics)" title="wikilink">molecular cloning</a> and <a href="DNA_fingerprinting" title="wikilink">DNA fingerprinting</a>.</p>
<p>Enzymes called <a href="DNA_ligase" title="wikilink">DNA ligases</a> can rejoin cut or broken DNA strands, using the energy from either <a href="adenosine_triphosphate" title="wikilink">adenosine triphosphate</a> or <a href="nicotinamide_adenine_dinucleotide" title="wikilink">nicotinamide adenine dinucleotide</a>.<a href="#fn87" class="footnote-ref" id="fnref87"><sup>87</sup></a> Ligases are particularly important in <a href="lagging_strand" title="wikilink">lagging strand</a> DNA replication, as they join together the short segments of DNA produced at the <a href="replication_fork" title="wikilink">replication fork</a> into a complete copy of the DNA template. They are also used in <a href="DNA_repair" title="wikilink">DNA repair</a> and <a href="genetic_recombination" title="wikilink">genetic recombination</a>.<a href="#fn88" class="footnote-ref" id="fnref88"><sup>88</sup></a></p>
<h4 id="topoisomerases-and-helicases">Topoisomerases and helicases</h4>
<p><a href="Topoisomerase" title="wikilink">Topoisomerases</a> are enzymes with both nuclease and ligase activity. These proteins change the amount of <a href="DNA_supercoil" title="wikilink">supercoiling</a> in DNA. Some of these enzyme work by cutting the DNA helix and allowing one section to rotate, thereby reducing its level of supercoiling; the enzyme then seals the DNA break.<a href="#fn89" class="footnote-ref" id="fnref89"><sup>89</sup></a> Other types of these enzymes are capable of cutting one DNA helix and then passing a second strand of DNA through this break, before rejoining the helix.<a href="#fn90" class="footnote-ref" id="fnref90"><sup>90</sup></a> Topoisomerases are required for many processes involving DNA, such as DNA replication and transcription.<a href="#fn91" class="footnote-ref" id="fnref91"><sup>91</sup></a></p>
<p><a href="Helicase" title="wikilink">Helicases</a> are proteins that are a type of <a href="molecular_motor" title="wikilink">molecular motor</a>. They use the chemical energy in <a href="nucleoside_triphosphate" title="wikilink">nucleoside triphosphates</a>, predominantly <a href="Adenosine_triphosphate" title="wikilink">ATP</a>, to break hydrogen bonds between bases and unwind the DNA double helix into single strands.<a href="#fn92" class="footnote-ref" id="fnref92"><sup>92</sup></a> These enzymes are essential for most processes where enzymes need to access the DNA bases.</p>
<h4 id="polymerases">Polymerases</h4>
<p>Polymerases are enzymes that synthesise polynucleotide chains from <a href="nucleoside_triphosphate" title="wikilink">nucleoside triphosphates</a>. They function by adding nucleotides onto the 3&lt;U+02C8&gt; <a href="hydroxyl" title="wikilink">hydroxyl group</a> of the previous nucleotide in the DNA strand. As a consequence, all polymerases work in a 5' to 3' direction.<a href="#fn93" class="footnote-ref" id="fnref93"><sup>93</sup></a> In the <a href="active_site" title="wikilink">active site</a> of these enzymes, the nucleoside triphosphate substrate base-pairs to a single-stranded polynucleotide template: this allows polymerases to accurately synthesise the complementary strand of this template. Polymerases are classified depending of the type of template they use.</p>
<p>In <a href="DNA_replication" title="wikilink">DNA replication</a>, a DNA-dependent <a href="DNA_polymerase" title="wikilink">DNA polymerase</a> makes a DNA copy of a DNA sequence. Accuracy is vital in this process, so many of these polymerases have a <a href="Proofreading#Proofreading_in_biology" title="wikilink">proofreading</a> activity. Here, the polymerase recognizes the occasional mistakes in the synthesis reaction by the lack of base pairing between the mismatched nucleotides. If a mismatch is detected, a 3' to 5' <a href="exonuclease" class="uri" title="wikilink">exonuclease</a> activity is activated and the incorrect base removed.<a href="#fn94" class="footnote-ref" id="fnref94"><sup>94</sup></a> In most organisms DNA polymerases function in a large complex called the <a href="replisome" class="uri" title="wikilink">replisome</a> that contains multiple accessory subunits, such as the <a href="DNA_clamp" title="wikilink">DNA clamp</a> or <a href="helicase" title="wikilink">helicases</a>.<a href="#fn95" class="footnote-ref" id="fnref95"><sup>95</sup></a></p>
<p>RNA-dependent DNA polymerases are a specialised class of polymerases that copy the sequence of a RNA strand into DNA. They include <a href="reverse_transcriptase" title="wikilink">reverse transcriptase</a>, which is a <a href="virus" title="wikilink">viral</a> enzyme involved in the infection of cells by <a href="retrovirus" title="wikilink">retroviruses</a>, and <a href="telomerase" class="uri" title="wikilink">telomerase</a>, which is required for the replication of <a href="telomere" title="wikilink">telomeres</a>.<a href="#fn96" class="footnote-ref" id="fnref96"><sup>96</sup></a><a href="#fn97" class="footnote-ref" id="fnref97"><sup>97</sup></a> Telomerase is an unusual polymerase because it contains its own RNA template as part of its structure.<a href="#fn98" class="footnote-ref" id="fnref98"><sup>98</sup></a></p>
<p>Transcription is carried out by a DNA-dependent <a href="RNA_polymerase" title="wikilink">RNA polymerase</a> that copies the sequence of a DNA strand into RNA. To begin transcribing a gene, the RNA polymerase binds to a sequence of DNA called a <a href="promoter" class="uri" title="wikilink">promoter</a> and separates the DNA strands. It then copies the gene sequence into a <a href="messenger_RNA" title="wikilink">messenger RNA</a> transcript until it reaches a region of DNA called the <a href="terminator_(genetics)" title="wikilink">terminator</a>, where it halts and detaches from the DNA. As with human DNA-dependent DNA polymerases, RNA polymerase II, the enzyme that transcribes most of the genes in the human genome, operates as part of a large protein complex with multiple regulatory and accessory subunits.<a href="#fn99" class="footnote-ref" id="fnref99"><sup>99</sup></a></p>
<h2 id="genetic-recombination">Genetic recombination</h2>
<div class="thumb tright" style="background-color: #f9f9f9; border: 1px solid #CCCCCC; margin:0.5em;">
<table>
<tbody>
<tr class="odd">
<td><figure>
<img src="Holliday_Junction_cropped.png" title="Holliday_Junction_cropped.png" alt="Holliday_Junction_cropped.png" width="250" /><figcaption>Holliday_Junction_cropped.png</figcaption>
</figure></td>
</tr>
<tr class="even">
<td><figure>
<img src="Holliday_junction_coloured.png" title="Holliday_junction_coloured.png" alt="Holliday_junction_coloured.png" width="250" /><figcaption>Holliday_junction_coloured.png</figcaption>
</figure></td>
</tr>
</tbody>
</table>
<div style="border: none; width:250px;">
<div class="thumbcaption">
<p>Structure of the <a href="Holliday_junction" title="wikilink">Holliday junction</a> intermediate in <a href="genetic_recombination" title="wikilink">genetic recombination</a>. The four separate DNA strands are coloured red, blue, green and yellow.<a href="#fn100" class="footnote-ref" id="fnref100"><sup>100</sup></a></p>
</div>
</div>
</div>

<figure>
<img src="Chromosomal_Recombination.svg" title="Recombination involves the breakage and rejoining of two chromosomes (M and F) to produce two re-arranged chromosomes (C1 and C2)." alt="Recombination involves the breakage and rejoining of two chromosomes (M and F) to produce two re-arranged chromosomes (C1 and C2)." width="250" /><figcaption>Recombination involves the breakage and rejoining of two chromosomes (M and F) to produce two re-arranged chromosomes (C1 and C2).</figcaption>
</figure>
<p>A DNA helix does not usually interact with other segments of DNA, and in human cells the different chromosomes even occupy separate areas in the nucleus called &quot;chromosome territories&quot;.<a href="#fn101" class="footnote-ref" id="fnref101"><sup>101</sup></a> This physical separation of different chromosomes is important for the ability of DNA to function as a stable repository for information, as one of the few times chromosomes interact is when they <a href="genetic_recombination" title="wikilink">recombine</a>. Recombination is when two DNA helices break, swap a section and then rejoin.</p>
<p>Recombination allows chromosomes to exchange genetic information and produces new combinations of genes, which increases the efficiency of <a href="natural_selection" title="wikilink">natural selection</a> and can be important in the rapid evolution of new proteins.<a href="#fn102" class="footnote-ref" id="fnref102"><sup>102</sup></a> Genetic recombination can also be involved in DNA repair, particularly in the cell's response to double-strand breaks.<a href="#fn103" class="footnote-ref" id="fnref103"><sup>103</sup></a></p>
<p>The most common form of recombination is <a href="chromosomal_crossover" title="wikilink">homologous recombination</a>, where the two chromosomes involved share very similar sequences. Non-homologous recombination can be damaging to cells, as it can produce <a href="chromosomal_translocation" title="wikilink">chromosomal translocations</a> and genetic abnormalities. The recombination reaction is catalyzed by enzymes known as <em>recombinases</em>, such as <a href="Cre_recombinase" title="wikilink">Cre recombinase</a>.<a href="#fn104" class="footnote-ref" id="fnref104"><sup>104</sup></a> In the first step, the recombinase creates a nick in one strand of a DNA double helix, allowing the nicked strand to pull apart from its <a href="complementarity_(molecular_biology)" title="wikilink">complementary</a> strand and <a href="annealing_(biology)" title="wikilink">anneal</a> to one strand of the double helix on the opposite <a href="chromatid" class="uri" title="wikilink">chromatid</a>. A second nick allows the strand in the second chromatid to pull apart and anneal to the remaining strand in the first helix, forming a structure known as a <em>cross-strand exchange</em> or a <a href="Holliday_junction" title="wikilink">Holliday junction</a>. The Holliday junction is a tetrahedral junction structure that can be moved along the pair of chromosomes, swapping one strand for another. The recombination reaction is then halted by cleavage of the junction and re-ligation of the released DNA.<a href="#fn105" class="footnote-ref" id="fnref105"><sup>105</sup></a></p>
<h2 id="evolution-of-dna-based-metabolism">Evolution of DNA-based metabolism</h2>
<p>DNA contains the genetic information that allows all modern living things to function, grow and reproduce. However, it is unclear how long in the 4-billion-year <a href="Timeline_of_evolution" title="wikilink">history of life</a> DNA has performed this function, as it has been proposed that the earliest forms of life may have used RNA as their genetic material.<a href="#fn106" class="footnote-ref" id="fnref106"><sup>106</sup></a><a href="#fn107" class="footnote-ref" id="fnref107"><sup>107</sup></a> RNA may have acted as the central part of early cell metabolism as it can both transmit genetic information and carry out <a href="catalysis" class="uri" title="wikilink">catalysis</a> as part of <a href="ribozyme" title="wikilink">ribozymes</a>.<a href="#fn108" class="footnote-ref" id="fnref108"><sup>108</sup></a> This ancient <a href="RNA_world_hypothesis" title="wikilink">RNA world</a> where nucleic acid would have been used for both catalysis and genetics may have influenced the evolution of the current genetic code based on four nucleotide bases. This would occur since the number of unique bases in such an organism is a trade-off between a small number of bases increasing replication accuracy and a large number of bases increasing the catalytic efficiency of ribozymes.<a href="#fn109" class="footnote-ref" id="fnref109"><sup>109</sup></a></p>
<p>Unfortunately, there is no direct evidence of ancient genetic systems, as recovery of DNA from most fossils is impossible. This is because DNA will survive in the environment for less than one million years and slowly degrades into short fragments in solution.<a href="#fn110" class="footnote-ref" id="fnref110"><sup>110</sup></a> Although claims for older DNA have been made, most notably a report of the isolation of a viable bacterium from a salt crystal 250-million years old,<a href="#fn111" class="footnote-ref" id="fnref111"><sup>111</sup></a> these claims are controversial and have been disputed.<a href="#fn112" class="footnote-ref" id="fnref112"><sup>112</sup></a><a href="#fn113" class="footnote-ref" id="fnref113"><sup>113</sup></a></p>
<h2 id="uses-in-technology">Uses in technology</h2>

<p>Modern <a href="biology" class="uri" title="wikilink">biology</a> and <a href="biochemistry" class="uri" title="wikilink">biochemistry</a> make intensive use of recombinant DNA technology. <a href="Recombinant_DNA" title="wikilink">Recombinant DNA</a> is a man-made DNA sequence assembled from other DNA sequences in a <a href="plasmid" class="uri" title="wikilink">plasmid</a>. These plasmids can be <a href="transformation_(genetics)" title="wikilink">transformed</a> into organisms. The <a href="genetic_engineering" title="wikilink">genetically modified</a> organisms produced can be used to produce products such as recombinant <a href="protein" title="wikilink">proteins</a>, used in medical research,<a href="#fn114" class="footnote-ref" id="fnref114"><sup>114</sup></a> or be grown in <a href="agriculture" class="uri" title="wikilink">agriculture</a>.<a href="#fn115" class="footnote-ref" id="fnref115"><sup>115</sup></a><a href="#fn116" class="footnote-ref" id="fnref116"><sup>116</sup></a></p>
<h3 id="forensics">Forensics</h3>

<p><a href="Forensic_science" title="wikilink">Forensic scientists</a> can use DNA in <a href="blood" class="uri" title="wikilink">blood</a>, <a href="semen" class="uri" title="wikilink">semen</a>, <a href="skin" class="uri" title="wikilink">skin</a>, <a href="saliva" class="uri" title="wikilink">saliva</a> or <a href="hair" class="uri" title="wikilink">hair</a> at a crime scene to identify a perpetrator. This process is called <a href="genetic_fingerprinting" title="wikilink">genetic fingerprinting</a>, or more accurately, DNA profiling. In DNA profiling, the lengths of variable sections of repetitive DNA, such as <a href="short_tandem_repeat" title="wikilink">short tandem repeats</a> and <a href="minisatellite" title="wikilink">minisatellites</a>, are compared between people. This method is usually an extremely reliable technique for identifying a criminal.<a href="#fn117" class="footnote-ref" id="fnref117"><sup>117</sup></a> However, identification can be complicated if the scene is contaminated with DNA from several people.<a href="#fn118" class="footnote-ref" id="fnref118"><sup>118</sup></a> DNA profiling was developed in 1984 by British geneticist Sir <a href="Alec_Jeffreys" title="wikilink">Alec Jeffreys</a>,<a href="#fn119" class="footnote-ref" id="fnref119"><sup>119</sup></a> and first used in forensic science to convict Colin Pitchfork in the 1988 <a href="Enderby_murders" title="wikilink">Enderby murders</a> case.<a href="#fn120" class="footnote-ref" id="fnref120"><sup>120</sup></a> People convicted of certain types of crimes may be required to provide a sample of DNA for a database. This has helped investigators solve old cases where only a DNA sample was obtained from the scene. DNA profiling can also be used to identify victims of mass casualty incidents.<a href="#fn121" class="footnote-ref" id="fnref121"><sup>121</sup></a></p>
<h3 id="bioinformatics">Bioinformatics</h3>

<p><a href="Bioinformatics" class="uri" title="wikilink">Bioinformatics</a> involves the manipulation, searching, and <a href="data_mining" title="wikilink">data mining</a> of DNA sequence data. The development of techniques to store and search DNA sequences have led to widely-applied advances in <a href="computer_science" title="wikilink">computer science</a>, especially <a href="string_searching_algorithm" title="wikilink">string searching algorithms</a>, <a href="machine_learning" title="wikilink">machine learning</a> and <a href="database_theory" title="wikilink">database theory</a>.<a href="#fn122" class="footnote-ref" id="fnref122"><sup>122</sup></a> String searching or matching algorithms, which find an occurrence of a sequence of letters inside a larger sequence of letters, were developed to search for specific sequences of nucleotides.<a href="#fn123" class="footnote-ref" id="fnref123"><sup>123</sup></a> In other applications such as <a href="text_editor" title="wikilink">text editors</a>, even simple algorithms for this problem usually suffice, but DNA sequences cause these algorithms to exhibit near-worst-case behaviour due to their small number of distinct characters. The related problem of <a href="sequence_alignment" title="wikilink">sequence alignment</a> aims to identify <a href="homology_(biology)" title="wikilink">homologous</a> sequences and locate the specific <a href="mutation" title="wikilink">mutations</a> that make them distinct. These techniques, especially <a href="multiple_sequence_alignment" title="wikilink">multiple sequence alignment</a>, are used in studying <a href="phylogenetics" title="wikilink">phylogenetic</a> relationships and protein function.<a href="#fn124" class="footnote-ref" id="fnref124"><sup>124</sup></a> Data sets representing entire genomes' worth of DNA sequences, such as those produced by the <a href="Human_Genome_Project" title="wikilink">Human Genome Project</a>, are difficult to use without annotations, which label the locations of genes and regulatory elements on each chromosome. Regions of DNA sequence that have the characteristic patterns associated with protein- or RNA-coding genes can be identified by <a href="gene_finding" title="wikilink">gene finding</a> algorithms, which allow researchers to predict the presence of particular <a href="gene_product" title="wikilink">gene products</a> in an organism even before they have been isolated experimentally.<a href="#fn125" class="footnote-ref" id="fnref125"><sup>125</sup></a></p>
<h3 id="dna-and-computation">DNA and computation</h3>

<p>DNA was first used in computing to solve a small version of the directed <a href="Hamiltonian_path_problem" title="wikilink">Hamiltonian path problem</a>, an <a href="NP-complete" class="uri" title="wikilink">NP-complete</a> problem.<a href="#fn126" class="footnote-ref" id="fnref126"><sup>126</sup></a> <a href="DNA_computing" title="wikilink">DNA computing</a> is advantageous over electronic computers in power use, space use, and efficiency, due to its ability to compute in a highly parallel fashion (see <a href="parallel_computing" title="wikilink">parallel computing</a>). A number of other problems, including simulation of various <a href="abstract_machine" title="wikilink">abstract machines</a>, the <a href="boolean_satisfiability_problem" title="wikilink">boolean satisfiability problem</a>, and the bounded version of the <a href="travelling_salesman_problem" title="wikilink">travelling salesman problem</a>, have since been analysed using DNA computing.<a href="#fn127" class="footnote-ref" id="fnref127"><sup>127</sup></a> Due to its compactness, DNA also has a theoretical role in <a href="cryptography" class="uri" title="wikilink">cryptography</a>, where in particular it allows unbreakable <a href="one-time_pad" title="wikilink">one-time pads</a> to be efficiently constructed and used.<a href="#fn128" class="footnote-ref" id="fnref128"><sup>128</sup></a></p>
<h3 id="history-and-anthropology">History and anthropology</h3>

<p>Because DNA collects mutations over time, which are then inherited, it contains historical information and by comparing DNA sequences, geneticists can infer the evolutionary history of organisms, their <a href="phylogeny" class="uri" title="wikilink">phylogeny</a>.<a href="#fn129" class="footnote-ref" id="fnref129"><sup>129</sup></a> This field of <a href="phylogenetics" class="uri" title="wikilink">phylogenetics</a> is a powerful tool in <a href="evolutionary_biology" title="wikilink">evolutionary biology</a>. If DNA sequences within a species are compared, <a href="population_genetics" title="wikilink">population geneticists</a> can learn the history of particular populations. This can be used in studies ranging from <a href="ecological_genetics" title="wikilink">ecological genetics</a> to <a href="anthropology" class="uri" title="wikilink">anthropology</a>; for example, DNA evidence is being used to try to identify the <a href="Ten_Lost_Tribes_of_Israel" title="wikilink">Ten Lost Tribes of Israel</a>.<a href="#fn130" class="footnote-ref" id="fnref130"><sup>130</sup></a><a href="#fn131" class="footnote-ref" id="fnref131"><sup>131</sup></a></p>
<p>DNA has also been used to look at modern family relationships, such as establishing family relationships between the descendants of <a href="Sally_Hemings" title="wikilink">Sally Hemings</a> and <a href="Thomas_Jefferson" title="wikilink">Thomas Jefferson</a>. This usage is closely related to the use of DNA in criminal investigations detailed above. Indeed, some criminal investigations have been solved when DNA from crime scenes has matched relatives of the guilty individual.<a href="#fn132" class="footnote-ref" id="fnref132"><sup>132</sup></a></p>
<h2 id="history">History</h2>
<p><img src="Francis_Crick.png" title="fig:Francis Crick" alt="Francis Crick" width="125" /> <img src="JamesDWatson.jpg" title="fig:James Watson" alt="James Watson" width="125" /></p>

<p>DNA was first isolated by <a href="Friedrich_Miescher" title="wikilink">Friedrich Miescher</a> who, in 1869, discovered a microscopic substance in the <a href="pus" class="uri" title="wikilink">pus</a> of discarded surgical bandages. As it resided in the nuclei of cells, he called it &quot;nuclein&quot;.<a href="#fn133" class="footnote-ref" id="fnref133"><sup>133</sup></a> In 1929 this discovery was followed by <a href="Phoebus_Levene" title="wikilink">Phoebus Levene</a>'s identification of the base, sugar and phosphate nucleotide unit.<a href="#fn134" class="footnote-ref" id="fnref134"><sup>134</sup></a> Levene suggested that DNA consisted of a string of nucleotide units linked together through the phosphate groups. However, Levene thought the chain was short and the bases repeated in a fixed order. In 1937 <a href="William_Astbury" title="wikilink">William Astbury</a> produced the first <a href="X-ray_diffraction" title="wikilink">X-ray diffraction</a> patterns that showed that DNA had a regular structure.<a href="#fn135" class="footnote-ref" id="fnref135"><sup>135</sup></a></p>
<p>In 1943, <a href="Oswald_Theodore_Avery" title="wikilink">Oswald Theodore Avery</a> discovered that <a href="trait_(biology)" title="wikilink">traits</a> of the &quot;smooth&quot; form of the <em>Pneumococcus</em> could be transferred to the &quot;rough&quot; form of the same bacteria by mixing killed &quot;smooth&quot; bacteria with the live &quot;rough&quot; form. Avery identified DNA as this <a href="transforming_principle" title="wikilink">transforming principle</a>.<a href="#fn136" class="footnote-ref" id="fnref136"><sup>136</sup></a> DNA's role in <a href="heredity" class="uri" title="wikilink">heredity</a> was confirmed in 1953, when <a href="Alfred_Hershey" title="wikilink">Alfred Hershey</a> and <a href="Martha_Chase" title="wikilink">Martha Chase</a> in the <a href="Hershey-Chase_experiment" title="wikilink">Hershey-Chase experiment</a> showed that DNA is the <a href="genetic_material" title="wikilink">genetic material</a> of the <a href="T2_phage" title="wikilink">T2 phage</a>.<a href="#fn137" class="footnote-ref" id="fnref137"><sup>137</sup></a></p>
<p>In 1953, based on <a href="Photo_51" title="wikilink">X-ray diffraction images</a><a href="#fn138" class="footnote-ref" id="fnref138"><sup>138</sup></a> taken by <a href="Rosalind_Franklin" title="wikilink">Rosalind Franklin</a> and the information that the bases were paired, <a href="James_D._Watson" title="wikilink">James D. Watson</a> and <a href="Francis_Crick" title="wikilink">Francis Crick</a> suggested<a href="#fn139" class="footnote-ref" id="fnref139"><sup>139</sup></a> what is now accepted as the first accurate model of <a href="Molecular_structure_of_Nucleic_Acids" title="wikilink">DNA structure</a> in the journal <a href="Nature_(journal)" title="wikilink"><em>Nature</em></a>.<a href="#fn140" class="footnote-ref" id="fnref140"><sup>140</sup></a> Experimental evidence for Watson and Crick's model were published in a series of five articles in the same issue of <em>Nature</em>.<a href="#fn141" class="footnote-ref" id="fnref141"><sup>141</sup></a> Of these, <a href="Rosalind_Franklin" title="wikilink">Franklin</a> and <a href="Raymond_Gosling" title="wikilink">Raymond Gosling</a>'s paper<a href="#fn142" class="footnote-ref" id="fnref142"><sup>142</sup></a> saw the publication of the <a href="http://osulibrary.oregonstate.edu/specialcollections/coll/pauling/dna/pictures/franklin-typeBphoto.html">X-ray diffraction image</a>, which was key in Watson and Crick interpretation, as well as another article, co-authored by <a href="Maurice_Wilkins" title="wikilink">Maurice Wilkins</a> and his colleagues.<a href="#fn143" class="footnote-ref" id="fnref143"><sup>143</sup></a> Franklin and Gosling's subsequent paper identified the distinctions between the A and B structures of the double helix in DNA.<a href="#fn144" class="footnote-ref" id="fnref144"><sup>144</sup></a> In 1962 Watson, Crick, and <a href="Maurice_Wilkins" title="wikilink">Maurice Wilkins</a> jointly received the <a href="Nobel_Prize" title="wikilink">Nobel Prize</a> in <a href="Nobel_Prize_in_Physiology_or_Medicine" title="wikilink">Physiology or Medicine</a> (Franklin didn't share the prize with them since she had died earlier).<a href="#fn145" class="footnote-ref" id="fnref145"><sup>145</sup></a></p>
<p>In an influential presentation in 1957, Crick laid out the <a href="central_dogma_of_molecular_biology" title="wikilink">&quot;Central Dogma&quot; of molecular biology</a>, which foretold the relationship between DNA, RNA, and proteins, and articulated the &quot;adaptor hypothesis&quot;.<a href="#fn146" class="footnote-ref" id="fnref146"><sup>146</sup></a> Final confirmation of the replication mechanism that was implied by the double-helical structure followed in 1958 through the <a href="Meselson-Stahl_experiment" title="wikilink">Meselson-Stahl experiment</a>.<a href="#fn147" class="footnote-ref" id="fnref147"><sup>147</sup></a> Further work by Crick and coworkers showed that the genetic code was based on non-overlapping triplets of bases, called codons, allowing <a href="Har_Gobind_Khorana" title="wikilink">Har Gobind Khorana</a>, <a href="Robert_W._Holley" title="wikilink">Robert W. Holley</a> and <a href="Marshall_Warren_Nirenberg" title="wikilink">Marshall Warren Nirenberg</a> to decipher the <a href="genetic_code" title="wikilink">genetic code</a>.<a href="#fn148" class="footnote-ref" id="fnref148"><sup>148</sup></a> These findings represent the birth of <a href="molecular_biology" title="wikilink">molecular biology</a>.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Genetic_disorder" title="wikilink">Genetic disorder</a></li>
<li><a href="Plasmid" class="uri" title="wikilink">Plasmid</a></li>
<li><a href="DNA_sequencing" title="wikilink">DNA sequencing</a></li>
<li><a href="Southern_blot" title="wikilink">Southern blot</a></li>
<li><a href="DNA_microarray" title="wikilink">DNA microarray</a></li>
<li><a href="Polymerase_chain_reaction" title="wikilink">Polymerase chain reaction</a></li>
<li><a href="Protein-DNA_interaction_site_predictor" title="wikilink">Protein-DNA interaction site predictor</a></li>
<li><a href="Phosphoramidite" class="uri" title="wikilink">Phosphoramidite</a></li>
<li><a href="Quantification_of_nucleic_acids" title="wikilink">Quantification of nucleic acids</a></li>
</ul>
<h2 id="references">References</h2>
<div class="references-small" style="-moz-column-count:2; column-count:2;">
<references/>
</div>
<h2 id="further-reading">Further reading</h2>
<ul>
<li>Clayton, Julie. (Ed.). <em>50 Years of DNA</em>, Palgrave MacMillan Press, 2003. ISBN 978-1-40-391479-8</li>
<li>Judson, Horace Freeland. <em>The Eighth Day of Creation: Makers of the Revolution in Biology</em>, Cold Spring Harbor Laboratory Press, 1996. ISBN 978-0-87-969478-4</li>
<li><a href="Robert_Olby" title="wikilink">Olby, Robert</a>. <em>The Path to The Double Helix: Discovery of DNA</em>, first published in October 1974 by MacMillan, with foreword by Francis Crick; ISBN 978-0-48-668117-7; the definitive DNA textbook, revised in 1994, with a 9 page postscript.</li>
<li><a href="Matt_Ridley" title="wikilink">Ridley, Matt</a>. <em>Francis Crick: Discoverer of the Genetic Code (Eminent Lives)</em> HarperCollins Publishers; 192 pp, ISBN 978-0-06-082333-7 <a href="2006" class="uri" title="wikilink">2006</a></li>
<li>Rose, Steven. <em>The Chemistry of Life</em>, Penguin, ISBN 978-0-14-027273-4.</li>
<li>Watson, James D. and Francis H.C. Crick. <a href="http://www.nature.com/nature/dna50/watsoncrick.pdf">A structure for Deoxyribose Nucleic Acid</a> (PDF). <em><a href="Nature_(journal)" title="wikilink">Nature</a></em> 171, 737 &lt;U+2013&gt; 738, <a href="25_April" title="wikilink">25 April</a> <a href="1953" class="uri" title="wikilink">1953</a>.</li>
<li>Watson, James D. <em>DNA: The Secret of Life</em> ISBN 978-0-375-41546-3.</li>
<li>Watson, James D. <em><a href="The_Double_Helix" title="wikilink">The Double Helix: A Personal Account of the Discovery of the Structure of DNA (Norton Critical Editions)</a></em>. ISBN 978-0-393-95075-5</li>
<li>Watson, James D. &quot;Avoid boring people and other lessons from a life in science&quot; New York: Random House. ISBN 978-0-375-421844 (0-375-41284-0)366pp <a href="2007" class="uri" title="wikilink">2007</a></li>
<li>Calladine, Chris R.; Drew, Horace R.; Luisi, Ben F. and Travers, Andrew A. <em>Understanding DNA</em>, Elsevier Academic Press, 2003. ISBN 978-0-12155089-9</li>
</ul>
<h2 id="dvd">DVD</h2>
<ul>
<li><em><a href="http://www.windfallfilms.com/html/productions/DNA.htm">DNA &lt;U+2014&gt; The Story of the Pioneers who Changed the World,</a></em> Windfall Films Production for <a href="http://www.channel4.com/science/microsites/D/dna_thestoryoflife/">Channel Four Television</a> &amp; <a href="http://www.pbs.org/wnet/dna/">PBS Thirteen-WNET</a> &lt;U+2014&gt; 2003, PAL <a href="http://www.ncbe.reading.ac.uk/DNA50/documentaries.html">1</a>, NTSC <a href="http://www.shoppbs.org/searchHandler/index.jsp?searchId=20744726972&amp;keywords=dna&amp;view=all">PBS Shop</a></li>
<li><em><a href="http://www.dnai.org/feature/dnai_dvd.html">DNA interactive</a></em> PAL <a href="http://www.ncbe.reading.ac.uk/DNA50/interactivepal.html">2</a>, NTSC <a href="http://www.ncbe.reading.ac.uk/DNA50/interactiventsc.html">3</a>, <a href="http://www.scienceinschool.org/2006/issue1/dnainteractive/">4</a></li>
<li><em><a href="http://www.carolina.com/biotech/DNA_secret.asp">DNA: The Secret of Life</a></em> Carolina Biological</li>
<li><em><a href="http://shop.wgbh.org/webapp/wcs/stores/servlet/ProductDisplay?productId=51808&amp;storeId=11051&amp;catalogId=10051&amp;langId=-1">DNA &lt;U+2014&gt; Secret of Photo 51</a></em> Rosalind Franklin &lt;U+2014&gt; NOVA documentary (NTSC &lt;U+2014&gt; Region 1?)</li>
<li><em><a href="http://shop.wgbh.org/webapp/wcs/stores/servlet/ProductDisplay?productId=18308&amp;storeId=11051&amp;catalogId=10051&amp;langId=-1">Cracking the Code of Life</a></em> NOVA documentary (NTSC &lt;U+2014&gt; All Regions)</li>
</ul>
<h2 id="external-links">External links</h2>


<ul>
<li><a href="http://orpheus.ucsd.edu/speccoll/testing/html/mss0660a.html#abstract">5</a> Crick's personal papers at Mandeville Special Collections Library, Geisel Library, University of California, San Diego</li>
<li><a href="http://www.dnai.org/">DNA Interactive</a> (requires <a href="Adobe_Flash" title="wikilink">Adobe Flash</a>)</li>
<li><a href="http://www.dnaftb.org/dnaftb/">DNA from the beginning</a></li>
<li><a href="http://www.ncbe.reading.ac.uk/DNA50/">Double Helix 1953 &lt;U+2013&gt; 2003</a> National Centre for Biotechnology Education</li>
<li><a href="http://www.nature.com/nature/dna50/archive.html">Double helix: 50 years of DNA</a>, <em><a href="Nature_(journal)" title="wikilink">Nature</a></em></li>
<li><a href="http://mason.gmu.edu/~emoody/rfranklin.html">Rosalind Franklin's contributions to the study of DNA</a></li>
<li><a href="http://www.genome.gov/10506367">U.S. National DNA Day</a> &lt;U+2014&gt; watch videos and participate in real-time chat with top scientists</li>
<li><a href="http://www.genome.gov/10506718">Genetic Education Modules for Teachers</a> &lt;U+2014&gt; <em>DNA from the Beginning</em> Study Guide</li>
<li><a href="http://www.bbc.co.uk/bbcfour/audiointerviews/profilepages/crickwatson1.shtml">Listen to Francis Crick and James Watson talking on the BBC in 1962, 1972, and 1974</a></li>
<li></li>
<li><a href="http://www.fidelitysystems.com/Unlinked_DNA.html">DNA under electron microscope</a></li>
<li></li>
<li><a href="http://dnawiz.com/">DNA Articles</a> &lt;U+2014&gt; articles and information collected from various sources</li>
<li></li>
<li><a href="http://biostudio.com/c_%20education%20mac.htm">DNA coiling to form chromosomes</a></li>
<li><a href="http://pipe.scs.fsu.edu/displar.html">DISPLAR: DNA binding site prediction on protein</a></li>
<li><a href="http://www.dnalc.org/">Dolan DNA Learning Center</a></li>
<li><a href="Robert_Olby" title="wikilink">Olby, R.</a> (2003) <a href="http://chem-faculty.ucsd.edu/joseph/CHEM13/DNA1.pdf">&quot;Quiet debut for the double helix&quot;</a> <em>Nature</em> <strong>421</strong> (January 23): 402 &lt;U+2013&gt; 405.</li>
<li><a href="http://www.blackwellpublishing.com/trun/artwork/Animations/cloningexp/cloningexp.html">Basic animated guide to DNA cloning</a></li>
</ul>





<p><a href="ar:&lt;U+062D&gt;&lt;U+0645&gt;&lt;U+0636&gt;_&lt;U+0646&gt;&lt;U+0648&gt;&lt;U+0648&gt;&lt;U+064A&gt;_&lt;U+0631&gt;&lt;U+064A&gt;&lt;U+0628&gt;&lt;U+064A&gt;_&lt;U+0645&gt;&lt;U+0646&gt;&lt;U+0642&gt;&lt;U+0648&gt;&lt;U+0635&gt;_&lt;U+0627&gt;&lt;U+0644&gt;&lt;U+0623&gt;&lt;U+0643&gt;&lt;U+0633&gt;&lt;U+062C&gt;&lt;U+064A&gt;&lt;U+0646&gt;" title="wikilink">ar:&lt;U+062D&gt;&lt;U+0645&gt;&lt;U+0636&gt; &lt;U+0646&gt;&lt;U+0648&gt;&lt;U+0648&gt;&lt;U+064A&gt; &lt;U+0631&gt;&lt;U+064A&gt;&lt;U+0628&gt;&lt;U+064A&gt; &lt;U+0645&gt;&lt;U+0646&gt;&lt;U+0642&gt;&lt;U+0648&gt;&lt;U+0635&gt; &lt;U+0627&gt;&lt;U+0644&gt;&lt;U+0623&gt;&lt;U+0643&gt;&lt;U+0633&gt;&lt;U+062C&gt;&lt;U+064A&gt;&lt;U+0646&gt;</a> <a href="az:Dezoksiribonuklein_tur&lt;U+015F&gt;usu" title="wikilink">az:Dezoksiribonuklein tur&lt;U+015F&gt;usu</a> <a href="zh-min-nan:DNA" class="uri" title="wikilink">zh-min-nan:DNA</a> <a href="bs:Dezoksiribonukleinska_kiselina" title="wikilink">bs:Dezoksiribonukleinska kiselina</a> <a href="bg:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;" class="uri" title="wikilink">bg:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;</a> <a href="ca:&lt;U+00C0&gt;cid_desoxiribonucleic" title="wikilink">ca:&lt;U+00C0&gt;cid desoxiribonucleic</a> <a href="cs:DNA" class="uri" title="wikilink">cs:DNA</a> <a href="cy:DNA" class="uri" title="wikilink">cy:DNA</a> <a href="da:Dna" class="uri" title="wikilink">da:Dna</a> <a href="pdc:DNA" class="uri" title="wikilink">pdc:DNA</a> <a href="de:Desoxyribonukleins&lt;U+00E4&gt;ure" class="uri" title="wikilink">de:Desoxyribonukleins&lt;U+00E4&gt;ure</a> <a href="et:Desoks&lt;U+00FC&gt;ribonukleiinhape" class="uri" title="wikilink">et:Desoks&lt;U+00FC&gt;ribonukleiinhape</a> <a href="el:DNA" class="uri" title="wikilink">el:DNA</a> <a href="es:ADN" class="uri" title="wikilink">es:ADN</a> <a href="eo:DNA" class="uri" title="wikilink">eo:DNA</a> <a href="eu:ADN" class="uri" title="wikilink">eu:ADN</a> <a href="fa:&lt;U+062F&gt;&lt;U+06CC&gt;&lt;U+200C&gt;&lt;U+0627&gt;&lt;U+0646&gt;&lt;U+200C&gt;&lt;U+0627&gt;&lt;U+06CC&gt;" class="uri" title="wikilink">fa:&lt;U+062F&gt;&lt;U+06CC&gt;&lt;U+200C&gt;&lt;U+0627&gt;&lt;U+0646&gt;&lt;U+200C&gt;&lt;U+0627&gt;&lt;U+06CC&gt;</a> <a href="fr:Acide_d&lt;U+00E9&gt;soxyribonucl&lt;U+00E9&gt;ique" title="wikilink">fr:Acide d&lt;U+00E9&gt;soxyribonucl&lt;U+00E9&gt;ique</a> <a href="ga:ADN" class="uri" title="wikilink">ga:ADN</a> <a href="gl:ADN" class="uri" title="wikilink">gl:ADN</a> <a href="ko:DNA" class="uri" title="wikilink">ko:DNA</a> <a href="hr:Deoksiribonukleinska_kiselina" title="wikilink">hr:Deoksiribonukleinska kiselina</a> <a href="id:Asam_deoksiribonukleat" title="wikilink">id:Asam deoksiribonukleat</a> <a href="it:DNA" class="uri" title="wikilink">it:DNA</a> <a href="he:DNA" class="uri" title="wikilink">he:DNA</a> <a href="ka:&lt;U+10D3&gt;&lt;U+10D4&gt;&lt;U+10D6&gt;&lt;U+10DD&gt;&lt;U+10E5&gt;&lt;U+10E1&gt;&lt;U+10D8&gt;&lt;U+10E0&gt;&lt;U+10D8&gt;&lt;U+10D1&gt;&lt;U+10DD&gt;&lt;U+10DC&gt;&lt;U+10E3&gt;&lt;U+10D9&gt;&lt;U+10DA&gt;&lt;U+10D4&gt;&lt;U+10D8&gt;&lt;U+10DC&gt;&lt;U+10D8&gt;&lt;U+10E1&gt;_&lt;U+10DB&gt;&lt;U+10DF&gt;&lt;U+10D0&gt;&lt;U+10D5&gt;&lt;U+10D0&gt;" title="wikilink">ka:&lt;U+10D3&gt;&lt;U+10D4&gt;&lt;U+10D6&gt;&lt;U+10DD&gt;&lt;U+10E5&gt;&lt;U+10E1&gt;&lt;U+10D8&gt;&lt;U+10E0&gt;&lt;U+10D8&gt;&lt;U+10D1&gt;&lt;U+10DD&gt;&lt;U+10DC&gt;&lt;U+10E3&gt;&lt;U+10D9&gt;&lt;U+10DA&gt;&lt;U+10D4&gt;&lt;U+10D8&gt;&lt;U+10DC&gt;&lt;U+10D8&gt;&lt;U+10E1&gt; &lt;U+10DB&gt;&lt;U+10DF&gt;&lt;U+10D0&gt;&lt;U+10D5&gt;&lt;U+10D0&gt;</a> <a href="ht:ADN" class="uri" title="wikilink">ht:ADN</a> <a href="la:Acidum_deoxyribonucleinicum" title="wikilink">la:Acidum deoxyribonucleinicum</a> <a href="lv:Dezoksiribonukle&lt;U+012B&gt;nsk&lt;U+0101&gt;be" class="uri" title="wikilink">lv:Dezoksiribonukle&lt;U+012B&gt;nsk&lt;U+0101&gt;be</a> <a href="lt:Deoksiribonukleor&lt;U+016B&gt;g&lt;U+0161&gt;tis" class="uri" title="wikilink">lt:Deoksiribonukleor&lt;U+016B&gt;g&lt;U+0161&gt;tis</a> <a href="hu:DNS_(biol&lt;U+00F3&gt;gia)" title="wikilink">hu:DNS (biol&lt;U+00F3&gt;gia)</a> <a href="mk:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;" class="uri" title="wikilink">mk:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;</a> <a href="ms:DNA" class="uri" title="wikilink">ms:DNA</a> <a href="nl:DNA" class="uri" title="wikilink">nl:DNA</a> <a href="ja:&lt;U+30C7&gt;&lt;U+30AA&gt;&lt;U+30AD&gt;&lt;U+30B7&gt;&lt;U+30EA&gt;&lt;U+30DC&gt;&lt;U+6838&gt;&lt;U+9178&gt;" class="uri" title="wikilink">ja:&lt;U+30C7&gt;&lt;U+30AA&gt;&lt;U+30AD&gt;&lt;U+30B7&gt;&lt;U+30EA&gt;&lt;U+30DC&gt;&lt;U+6838&gt;&lt;U+9178&gt;</a> <a href="no:DNA" class="uri" title="wikilink">no:DNA</a> <a href="nn:Deoksyribonukleinsyre" class="uri" title="wikilink">nn:Deoksyribonukleinsyre</a> <a href="pam:DNA" class="uri" title="wikilink">pam:DNA</a> <a href="pl:Kwas_deoksyrybonukleinowy" title="wikilink">pl:Kwas deoksyrybonukleinowy</a> <a href="pt:DNA" class="uri" title="wikilink">pt:DNA</a> <a href="ro:ADN" class="uri" title="wikilink">ro:ADN</a> <a href="ru:&lt;U+0414&gt;&lt;U+0435&gt;&lt;U+0437&gt;&lt;U+043E&gt;&lt;U+043A&gt;&lt;U+0441&gt;&lt;U+0438&gt;&lt;U+0440&gt;&lt;U+0438&gt;&lt;U+0431&gt;&lt;U+043E&gt;&lt;U+043D&gt;&lt;U+0443&gt;&lt;U+043A&gt;&lt;U+043B&gt;&lt;U+0435&gt;&lt;U+0438&gt;&lt;U+043D&gt;&lt;U+043E&gt;&lt;U+0432&gt;&lt;U+0430&gt;&lt;U+044F&gt;_&lt;U+043A&gt;&lt;U+0438&gt;&lt;U+0441&gt;&lt;U+043B&gt;&lt;U+043E&gt;&lt;U+0442&gt;&lt;U+0430&gt;" title="wikilink">ru:&lt;U+0414&gt;&lt;U+0435&gt;&lt;U+0437&gt;&lt;U+043E&gt;&lt;U+043A&gt;&lt;U+0441&gt;&lt;U+0438&gt;&lt;U+0440&gt;&lt;U+0438&gt;&lt;U+0431&gt;&lt;U+043E&gt;&lt;U+043D&gt;&lt;U+0443&gt;&lt;U+043A&gt;&lt;U+043B&gt;&lt;U+0435&gt;&lt;U+0438&gt;&lt;U+043D&gt;&lt;U+043E&gt;&lt;U+0432&gt;&lt;U+0430&gt;&lt;U+044F&gt; &lt;U+043A&gt;&lt;U+0438&gt;&lt;U+0441&gt;&lt;U+043B&gt;&lt;U+043E&gt;&lt;U+0442&gt;&lt;U+0430&gt;</a> <a href="sq:ADN" class="uri" title="wikilink">sq:ADN</a> <a href="simple:DNA" class="uri" title="wikilink">simple:DNA</a> <a href="sk:Deoxyribonukleov&lt;U+00E1&gt;_kyselina" title="wikilink">sk:Deoxyribonukleov&lt;U+00E1&gt; kyselina</a> <a href="sl:Deoksiribonukleinska_kislina" title="wikilink">sl:Deoksiribonukleinska kislina</a> <a href="sr:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;" class="uri" title="wikilink">sr:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;</a> <a href="sh:DNK" class="uri" title="wikilink">sh:DNK</a> <a href="su:DNA" class="uri" title="wikilink">su:DNA</a> <a href="fi:DNA" class="uri" title="wikilink">fi:DNA</a> <a href="sv:DNA" class="uri" title="wikilink">sv:DNA</a> <a href="tl:DNA" class="uri" title="wikilink">tl:DNA</a> <a href="ta:&lt;U+0B86&gt;&lt;U+0B95&gt;&lt;U+0BCD&gt;&lt;U+0B9A&gt;&lt;U+0BBF&gt;&lt;U+0B9C&gt;&lt;U+0BA9&gt;&lt;U+0BB1&gt;&lt;U+0BCD&gt;&lt;U+0BB1&gt;_&lt;U+0BB0&gt;&lt;U+0BC8&gt;&lt;U+0BAA&gt;&lt;U+0BCB&gt;_&lt;U+0B95&gt;&lt;U+0BB0&gt;&lt;U+0BC1&gt;_&lt;U+0B85&gt;&lt;U+0BAE&gt;&lt;U+0BBF&gt;&lt;U+0BB2&gt;&lt;U+0BAE&gt;&lt;U+0BCD&gt;" title="wikilink">ta:&lt;U+0B86&gt;&lt;U+0B95&gt;&lt;U+0BCD&gt;&lt;U+0B9A&gt;&lt;U+0BBF&gt;&lt;U+0B9C&gt;&lt;U+0BA9&gt;&lt;U+0BB1&gt;&lt;U+0BCD&gt;&lt;U+0BB1&gt; &lt;U+0BB0&gt;&lt;U+0BC8&gt;&lt;U+0BAA&gt;&lt;U+0BCB&gt; &lt;U+0B95&gt;&lt;U+0BB0&gt;&lt;U+0BC1&gt; &lt;U+0B85&gt;&lt;U+0BAE&gt;&lt;U+0BBF&gt;&lt;U+0BB2&gt;&lt;U+0BAE&gt;&lt;U+0BCD&gt;</a> <a href="th:&lt;U+0E14&gt;&lt;U+0E35&gt;&lt;U+0E40&gt;&lt;U+0E2D&gt;&lt;U+0E47&gt;&lt;U+0E19&gt;&lt;U+0E40&gt;&lt;U+0E2D&gt;" class="uri" title="wikilink">th:&lt;U+0E14&gt;&lt;U+0E35&gt;&lt;U+0E40&gt;&lt;U+0E2D&gt;&lt;U+0E47&gt;&lt;U+0E19&gt;&lt;U+0E40&gt;&lt;U+0E2D&gt;</a> <a href="vi:DNA" class="uri" title="wikilink">vi:DNA</a> <a href="tr:DNA" class="uri" title="wikilink">tr:DNA</a> <a href="uk:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;" class="uri" title="wikilink">uk:&lt;U+0414&gt;&lt;U+041D&gt;&lt;U+041A&gt;</a> <a href="ur:&lt;U+0688&gt;&lt;U+06CC&gt;_&lt;U+0627&gt;&lt;U+06CC&gt;&lt;U+0646&gt;_&lt;U+0627&gt;&lt;U+06D2&gt;" title="wikilink">ur:&lt;U+0688&gt;&lt;U+06CC&gt; &lt;U+0627&gt;&lt;U+06CC&gt;&lt;U+0646&gt; &lt;U+0627&gt;&lt;U+06D2&gt;</a> <a href="yi:&lt;U+05D3&gt;&lt;U+05D9&gt;_&lt;U+05E2&gt;&lt;U+05DF&gt;_&lt;U+05E2&gt;&lt;U+05D9&gt;" title="wikilink">yi:&lt;U+05D3&gt;&lt;U+05D9&gt; &lt;U+05E2&gt;&lt;U+05DF&gt; &lt;U+05E2&gt;&lt;U+05D9&gt;</a> <a href="zh:&lt;U+8131&gt;&lt;U+6C27&gt;&lt;U+6838&gt;&lt;U+7CD6&gt;&lt;U+6838&gt;&lt;U+9178&gt;" class="uri" title="wikilink">zh:&lt;U+8131&gt;&lt;U+6C27&gt;&lt;U+6838&gt;&lt;U+7CD6&gt;&lt;U+6838&gt;&lt;U+9178&gt;</a></p>
<p><a href="Category:DNA" title="wikilink"> </a> <a href="Category:Genetics" class="uri" title="wikilink">Category:Genetics</a></p>
<section class="footnotes">
<hr />
<ol>
<li id="fn1"><a href="#fnref1" class="footnote-back">↩</a></li>
<li id="fn2">Butler, John M. (2001) <em>Forensic DNA Typing</em> &quot;Elsevier&quot;. pp. 14 &lt;U+2013&gt; 15. ISBN 978-0-12-147951-0.<a href="#fnref2" class="footnote-back">↩</a></li>
<li id="fn3"><a href="#fnref3" class="footnote-back">↩</a></li>
<li id="fn4"><a href="#fnref4" class="footnote-back">↩</a></li>
<li id="fn5"><a href="#fnref5" class="footnote-back">↩</a></li>
<li id="fn6">Berg J., Tymoczko J. and Stryer L. (2002) <em>Biochemistry.</em> W. H. Freeman and Company ISBN 0-7167-4955-6<a href="#fnref6" class="footnote-back">↩</a></li>
<li id="fn7"><a href="http://www.chem.qmul.ac.uk/iupac/misc/naabb.html">Abbreviations and Symbols for Nucleic Acids, Polynucleotides and their Constituents</a> IUPAC-IUB Commission on Biochemical Nomenclature (CBN) Accessed 03 Jan 2006<a href="#fnref7" class="footnote-back">↩</a></li>
<li id="fn8"><a href="#fnref8" class="footnote-back">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11"><a href="#fnref11" class="footnote-back">↩</a></li>
<li id="fn12"><a href="#fnref12" class="footnote-back">↩</a></li>
<li id="fn13">Created from <a href="http://www.rcsb.org/pdb/cgi/explore.cgi?pdbId=1D65">PDB 1D65</a><a href="#fnref13" class="footnote-back">↩</a></li>
<li id="fn14"><a href="#fnref14" class="footnote-back">↩</a></li>
<li id="fn15"><a href="#fnref15" class="footnote-back">↩</a></li>
<li id="fn16"><a href="#fnref16" class="footnote-back">↩</a></li>
<li id="fn17"><a href="#fnref17" class="footnote-back">↩</a></li>
<li id="fn18"></li>
<li id="fn19"><a href="#fnref19" class="footnote-back">↩</a></li>
<li id="fn20"><a href="#fnref20" class="footnote-back">↩</a></li>
<li id="fn21"><a href="#fnref21" class="footnote-back">↩</a></li>
<li id="fn22"><a href="#fnref22" class="footnote-back">↩</a></li>
<li id="fn23"><a href="#fnref23" class="footnote-back">↩</a></li>
<li id="fn24"><a href="#fnref24" class="footnote-back">↩</a></li>
<li id="fn25"><a href="#fnref25" class="footnote-back">↩</a></li>
<li id="fn26"><a href="#fnref26" class="footnote-back">↩</a></li>
<li id="fn27"><a href="#fnref27" class="footnote-back">↩</a></li>
<li id="fn28"><a href="#fnref28" class="footnote-back">↩</a></li>
<li id="fn29"><a href="#fnref29" class="footnote-back">↩</a></li>
<li id="fn30"><a href="#fnref30" class="footnote-back">↩</a></li>
<li id="fn31"><a href="#fnref31" class="footnote-back">↩</a></li>
<li id="fn32"><a href="#fnref32" class="footnote-back">↩</a></li>
<li id="fn33"><a href="#fnref33" class="footnote-back">↩</a></li>
<li id="fn34"><a href="#fnref34" class="footnote-back">↩</a></li>
<li id="fn35"><a href="#fnref35" class="footnote-back">↩</a></li>
<li id="fn36"><a href="#fnref36" class="footnote-back">↩</a></li>
<li id="fn37"></li>
<li id="fn38"><a href="#fnref38" class="footnote-back">↩</a></li>
<li id="fn39"><a href="#fnref39" class="footnote-back">↩</a></li>
<li id="fn40"><a href="#fnref40" class="footnote-back">↩</a></li>
<li id="fn41"><a href="#fnref41" class="footnote-back">↩</a></li>
<li id="fn42"><a href="#fnref42" class="footnote-back">↩</a></li>
<li id="fn43">Created from <a href="http://ndbserver.rutgers.edu/atlas/xray/structures/U/ud0017/ud0017.html">NDB UD0017</a><a href="#fnref43" class="footnote-back">↩</a></li>
<li id="fn44"><a href="#fnref44" class="footnote-back">↩</a></li>
<li id="fn45"><a href="#fnref45" class="footnote-back">↩</a></li>
<li id="fn46"><a href="#fnref46" class="footnote-back">↩</a></li>
<li id="fn47"><a href="#fnref47" class="footnote-back">↩</a></li>
<li id="fn48"><a href="#fnref48" class="footnote-back">↩</a></li>
<li id="fn49"><a href="#fnref49" class="footnote-back">↩</a></li>
<li id="fn50"></li>
<li id="fn51"><a href="#fnref51" class="footnote-back">↩</a></li>
<li id="fn52"><a href="#fnref52" class="footnote-back">↩</a></li>
<li id="fn53"><a href="#fnref53" class="footnote-back">↩</a></li>
<li id="fn54"><a href="#fnref54" class="footnote-back">↩</a></li>
<li id="fn55"><a href="#fnref55" class="footnote-back">↩</a></li>
<li id="fn56">,<a href="#fnref56" class="footnote-back">↩</a></li>
<li id="fn57"><a href="#fnref57" class="footnote-back">↩</a></li>
<li id="fn58"><a href="#fnref58" class="footnote-back">↩</a></li>
<li id="fn59"><a href="#fnref59" class="footnote-back">↩</a></li>
<li id="fn60"><a href="#fnref60" class="footnote-back">↩</a></li>
<li id="fn61"><a href="#fnref61" class="footnote-back">↩</a></li>
<li id="fn62"><a href="#fnref62" class="footnote-back">↩</a></li>
<li id="fn63"><a href="#fnref63" class="footnote-back">↩</a></li>
<li id="fn64"><a href="#fnref64" class="footnote-back">↩</a></li>
<li id="fn65"><a href="#fnref65" class="footnote-back">↩</a></li>
<li id="fn66"><a href="#fnref66" class="footnote-back">↩</a></li>
<li id="fn67"><a href="#fnref67" class="footnote-back">↩</a></li>
<li id="fn68"><a href="#fnref68" class="footnote-back">↩</a></li>
<li id="fn69"></li>
<li id="fn70"><a href="#fnref70" class="footnote-back">↩</a></li>
<li id="fn71"><a href="#fnref71" class="footnote-back">↩</a></li>
<li id="fn72"><a href="#fnref72" class="footnote-back">↩</a></li>
<li id="fn73"><a href="#fnref73" class="footnote-back">↩</a></li>
<li id="fn74"><a href="#fnref74" class="footnote-back">↩</a></li>
<li id="fn75"><a href="#fnref75" class="footnote-back">↩</a></li>
<li id="fn76"><a href="#fnref76" class="footnote-back">↩</a></li>
<li id="fn77"><a href="#fnref77" class="footnote-back">↩</a></li>
<li id="fn78"><a href="#fnref78" class="footnote-back">↩</a></li>
<li id="fn79"><a href="#fnref79" class="footnote-back">↩</a></li>
<li id="fn80"><a href="#fnref80" class="footnote-back">↩</a></li>
<li id="fn81"><a href="#fnref81" class="footnote-back">↩</a></li>
<li id="fn82"><a href="#fnref82" class="footnote-back">↩</a></li>
<li id="fn83"><a href="#fnref83" class="footnote-back">↩</a></li>
<li id="fn84"><a href="#fnref84" class="footnote-back">↩</a></li>
<li id="fn85">Created from <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1RVA">PDB 1RVA</a><a href="#fnref85" class="footnote-back">↩</a></li>
<li id="fn86"><a href="#fnref86" class="footnote-back">↩</a></li>
<li id="fn87"><a href="#fnref87" class="footnote-back">↩</a></li>
<li id="fn88"></li>
<li id="fn89"></li>
<li id="fn90"><a href="#fnref90" class="footnote-back">↩</a></li>
<li id="fn91"></li>
<li id="fn92"><a href="#fnref92" class="footnote-back">↩</a></li>
<li id="fn93"><a href="#fnref93" class="footnote-back">↩</a></li>
<li id="fn94"><a href="#fnref94" class="footnote-back">↩</a></li>
<li id="fn95"><a href="#fnref95" class="footnote-back">↩</a></li>
<li id="fn96"><a href="#fnref96" class="footnote-back">↩</a></li>
<li id="fn97"></li>
<li id="fn98"></li>
<li id="fn99"><a href="#fnref99" class="footnote-back">↩</a></li>
<li id="fn100">Created from <a href="http://www.rcsb.org/pdb/explore/explore.do?structureId=1M6G">PDB 1M6G</a><a href="#fnref100" class="footnote-back">↩</a></li>
<li id="fn101"><a href="#fnref101" class="footnote-back">↩</a></li>
<li id="fn102"><a href="#fnref102" class="footnote-back">↩</a></li>
<li id="fn103"><a href="#fnref103" class="footnote-back">↩</a></li>
<li id="fn104"><a href="#fnref104" class="footnote-back">↩</a></li>
<li id="fn105"><a href="#fnref105" class="footnote-back">↩</a></li>
<li id="fn106"><a href="#fnref106" class="footnote-back">↩</a></li>
<li id="fn107"><a href="#fnref107" class="footnote-back">↩</a></li>
<li id="fn108"><a href="#fnref108" class="footnote-back">↩</a></li>
<li id="fn109"><a href="#fnref109" class="footnote-back">↩</a></li>
<li id="fn110"><a href="#fnref110" class="footnote-back">↩</a></li>
<li id="fn111"><a href="#fnref111" class="footnote-back">↩</a></li>
<li id="fn112"><a href="#fnref112" class="footnote-back">↩</a></li>
<li id="fn113"><a href="#fnref113" class="footnote-back">↩</a></li>
<li id="fn114"><a href="#fnref114" class="footnote-back">↩</a></li>
<li id="fn115"><a href="#fnref115" class="footnote-back">↩</a></li>
<li id="fn116"><a href="#fnref116" class="footnote-back">↩</a></li>
<li id="fn117"><a href="#fnref117" class="footnote-back">↩</a></li>
<li id="fn118"><a href="#fnref118" class="footnote-back">↩</a></li>
<li id="fn119"><a href="#fnref119" class="footnote-back">↩</a></li>
<li id="fn120"><a href="http://www.forensic.gov.uk/forensic_t/inside/news/list_casefiles.php?case=1">Colin Pitchfork &lt;U+2014&gt; first murder conviction on DNA evidence also clears the prime suspect</a> Forensic Science Service Accessed 23 Dec 2006<a href="#fnref120" class="footnote-back">↩</a></li>
<li id="fn121"><a href="#fnref121" class="footnote-back">↩</a></li>
<li id="fn122">Baldi, Pierre. Brunak, Soren. <em>Bioinformatics: The Machine Learning Approach</em> MIT Press (2001) ISBN 978-0-262-02506-5<a href="#fnref122" class="footnote-back">↩</a></li>
<li id="fn123">Gusfield, Dan. <em>Algorithms on Strings, Trees, and Sequences: Computer Science and Computational Biology</em>. Cambridge University Press, 15 January <a href="1997" class="uri" title="wikilink">1997</a>. ISBN 978-0-521-58519-4.<a href="#fnref123" class="footnote-back">↩</a></li>
<li id="fn124"><a href="#fnref124" class="footnote-back">↩</a></li>
<li id="fn125"><a href="#fnref125" class="footnote-back">↩</a></li>
<li id="fn126"><a href="#fnref126" class="footnote-back">↩</a></li>
<li id="fn127"><a href="#fnref127" class="footnote-back">↩</a></li>
<li id="fn128">Ashish Gehani, Thomas LaBean and John Reif. <a href="http://citeseer.ist.psu.edu/gehani99dnabased.html">DNA-Based Cryptography</a>. Proceedings of the 5th DIMACS Workshop on DNA Based Computers, Cambridge, MA, USA, 14 &lt;U+2013&gt; 15 June 1999.<a href="#fnref128" class="footnote-back">↩</a></li>
<li id="fn129"><a href="#fnref129" class="footnote-back">↩</a></li>
<li id="fn130"><em>Lost Tribes of Israel</em>, <a href="NOVA_(TV_series)" title="wikilink">NOVA</a>, PBS airdate: 22 February 2000. Transcript available from <a href="http://www.pbs.org/wgbh/nova/transcripts/2706israel.html">PBS.org,</a> (last accessed on 4 March 2006)<a href="#fnref130" class="footnote-back">↩</a></li>
<li id="fn131">Kleiman, Yaakov. <a href="http://www.aish.com/societywork/sciencenature/the_cohanim_-_dna_connection.asp">&quot;The Cohanim/DNA Connection: The fascinating story of how DNA studies confirm an ancient biblical tradition&quot;.</a> <em>aish.com</em> (January 13, 2000). Accessed 4 March 2006.<a href="#fnref131" class="footnote-back">↩</a></li>
<li id="fn132">Bhattacharya, Shaoni. <a href="http://www.newscientist.com/article.ns?id=dn4908">&quot;Killer convicted thanks to relative's DNA&quot;.</a> <em>newscientist.com</em> (20 April 2004). Accessed 22 Dec 06<a href="#fnref132" class="footnote-back">↩</a></li>
<li id="fn133"><a href="#fnref133" class="footnote-back">↩</a></li>
<li id="fn134"><a href="#fnref134" class="footnote-back">↩</a></li>
<li id="fn135"><a href="#fnref135" class="footnote-back">↩</a></li>
<li id="fn136"><a href="#fnref136" class="footnote-back">↩</a></li>
<li id="fn137"><a href="#fnref137" class="footnote-back">↩</a></li>
<li id="fn138">Watson J.D. and Crick F.H.C. <a href="http://www.nature.com/nature/dna50/watsoncrick.pdf">&quot;A Structure for Deoxyribose Nucleic Acid&quot;.</a> (PDF) <em>Nature</em> 171, 737 &lt;U+2013&gt; 738 (1953). Accessed 13 Feb 2007.<a href="#fnref138" class="footnote-back">↩</a></li>
<li id="fn139"></li>
<li id="fn140"></li>
<li id="fn141">Nature Archives <a href="http://www.nature.com/nature/dna50/archive.html">Double Helix of DNA: 50 Years</a><a href="#fnref141" class="footnote-back">↩</a></li>
<li id="fn142">Molecular Configuration in Sodium Thymonucleate. Franklin R. and Gosling R.G.Nature 171, 740 &lt;U+2013&gt; 741 (1953)<a href="http://www.nature.com/nature/dna50/franklingosling.pdf">Nature Archives Full Text (PDF)</a><a href="#fnref142" class="footnote-back">↩</a></li>
<li id="fn143">Molecular Structure of Deoxypentose Nucleic Acids. Wilkins M.H.F., A.R. Stokes A.R. &amp; Wilson, H.R. Nature 171, 738 &lt;U+2013&gt; 740 (1953)<a href="http://www.nature.com/nature/dna50/wilkins.pdf">Nature Archives (PDF)</a><a href="#fnref143" class="footnote-back">↩</a></li>
<li id="fn144">Evidence for 2-Chain Helix in Crystalline Structure of Sodium Deoxyribonucleate. Franklin R. and Gosling R.G. Nature 172, 156 &lt;U+2013&gt; 157 (1953)<a href="http://www.nature.com/nature/dna50/franklingosling2.pdf">Nature Archives,full text (PDF)</a><a href="#fnref144" class="footnote-back">↩</a></li>
<li id="fn145"><a href="http://nobelprize.org/nobel_prizes/medicine/laureates/1962/">The Nobel Prize in Physiology or Medicine 1962</a> Nobelprize .org Accessed 22 Dec 06<a href="#fnref145" class="footnote-back">↩</a></li>
<li id="fn146">Crick, F.H.C. <a href="http://genome.wellcome.ac.uk/assets/wtx030893.pdf">On degenerate templates and the adaptor hypothesis (PDF).</a> genome.wellcome.ac.uk (Lecture, 1955). Accessed 22 Dec 2006<a href="#fnref146" class="footnote-back">↩</a></li>
<li id="fn147"><a href="#fnref147" class="footnote-back">↩</a></li>
<li id="fn148"><a href="http://nobelprize.org/nobel_prizes/medicine/laureates/1968/">The Nobel Prize in Physiology or Medicine 1968</a> Nobelprize.org Accessed 22 Dec 06<a href="#fnref148" class="footnote-back">↩</a></li>
</ol>
</section>
