<p><strong>DNA barcoding</strong> is a <a href="Taxonomy" title="wikilink">taxonomic</a> method that uses a short genetic marker in an organism's DNA to identify it as belonging to a particular <a href="species" class="uri" title="wikilink">species</a>. It differs from <a href="molecular_phylogeny" title="wikilink">molecular phylogeny</a> in that the main goal is not to determine classification but to identify an unknown sample in terms of a known classification.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> Although barcodes are sometimes used in an effort to identify unknown species or assess whether species should be combined or separated,<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> the utility of DNA barcoding for these purposes is subject to debate.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a></p>
<p>Applications include, for example, identifying plant leaves even when flowers or fruit are not available, identifying insect larvae (which typically have fewer diagnostic characters than adults), identifying the diet of an animal based on stomach contents or faeces,<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a> and identifying products in commerce (for example, herbal supplements or wood).<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a></p>
<h2 id="choice-of-locus">Choice of Locus</h2>
<p>A desirable locus for DNA barcoding should be standardized (so that large databases of sequences for that locus can be developed),<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> present in most of the taxa of interest and sequencable without species-specific <a href="PCR_primer" title="wikilink">PCR primers</a>,<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a> short enough to be easily sequenced with current technology,<a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a> and provide a large variation between species yet a relatively small amount of variation within a species.<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a></p>
<p>Although several loci have been suggested, a common set of choices are:</p>
<ul>
<li>For animals and many other eukaryotes, the mitochondrial CO1 gene</li>
</ul>
<ul>
<li>For <a href="land_plants" title="wikilink">land plants</a>, the concatenation of the <a href="rbcL" class="uri" title="wikilink">rbcL</a> and <a href="matK" class="uri" title="wikilink">matK</a> chloroplast genes<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a></li>
</ul>
<h3 id="mitochondrial-dna">Mitochondrial DNA</h3>
<p>DNA barcoding is based on a relatively simple concept. Most <a href="eukaryote" class="uri" title="wikilink">eukaryote</a> cells contain <a href="mitochondria" class="uri" title="wikilink">mitochondria</a>, and mitochondrial DNA (mtDNA) has a relatively fast <a href="mutation" class="uri" title="wikilink">mutation</a> rate, which results in significant variation in mtDNA sequences between species and, in principle, a comparatively small variance within species. A 648-<a href="base_pair" title="wikilink">bp</a> region of the <a href="mitochondrial" class="uri" title="wikilink">mitochondrial</a> <a href="cytochrome_c_oxidase" title="wikilink">cytochrome c oxidase</a> subunit I (COI) gene was proposed as a potential 'barcode'.</p>
<p>However, because all <a href="mtDNA" class="uri" title="wikilink">mtDNA</a> genes are maternally inherited (direct evidence for recombination in mtDNA is available in some <a href="bivalve" title="wikilink">bivalves</a> such as <em>Mytilus</em><a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> but it is suspected that it may be more widespread<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a>), any occurrences of <a href="Hybrid_(biology)" title="wikilink">hybridization</a>,<a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a> male-killing microoroganisms,<a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a> cytoplasmic incompatibility-inducing symbionts (e.g., <em><a href="Wolbachia" class="uri" title="wikilink">Wolbachia</a></em><a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a>), <a href="horizontal_gene_transfer" title="wikilink">horizontal gene transfer</a> (such as via cellular symbionts<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a>), or other &quot;reticulate&quot; evolutionary phenomena in a lineage can lead to misleading results (i.e., it is possible for two different species to share mtDNA,<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a> or for one species to have more than one mtDNA sequence exhibited among different individuals).<a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a><a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a></p>
<p>As of 2009, databases of CO1 sequences included at least 620,000 specimens from over 58,000 species of animals, larger than databases available for any other gene.<a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a></p>
<h3 id="identifying-flowering-plants">Identifying flowering plants</h3>
<p>Kress <em>et al.</em> (2005<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a>) suggest that the use of the COI sequence &lt;U+201C&gt;is not appropriate for most species of plants because of a much slower rate of cytochrome c oxidase I gene evolution in higher plants than in animals&lt;U+201D&gt;. A series of experiments was then conducted to find a more suitable region of the <a href="genome" class="uri" title="wikilink">genome</a> for use in the DNA barcoding of <a href="flowering_plant" title="wikilink">flowering plants</a> (or the larger group of <a href="land_plant" title="wikilink">land plants</a>).<a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a> One 2005 proposal was the nuclear <a href="internal_transcribed_spacer" title="wikilink">internal transcribed spacer</a> region and the plastid trnH-psbA intergenic spacer;<a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a> other researchers advocated other regions such as <a href="matK" class="uri" title="wikilink">matK</a>.<a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a></p>
<p>In 2009, a collaboration of a large group of plant DNA barcode researchers proposed two chloroplast genes, <a href="rbcL" class="uri" title="wikilink">rbcL</a> and <a href="matK" class="uri" title="wikilink">matK</a>, taken together, as a barcode for plants.<a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a> Jesse Ausubel, a DNA barcode researcher not involved in that effort, suggested that standardizing on a sequence was the best way to produce a large database of plant sequences, and that time would tell whether this choice would be sufficiently good at distinguishing different plant species.<a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a></p>
<h2 id="vouchered-specimens">Vouchered specimens</h2>
<p>DNA sequence databases like GenBank contain many sequences that are not tied to vouchered specimens (for example, herbarium specimens, cultured cell lines, or sometimes images). This is problematic in the face of taxonomic issues such as whether several species should be split or combined, or whether past identifications were sound. Therefore, best practice for DNA barcoding is to sequence vouchered specimens.<a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a><a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a></p>
<h2 id="origin">Origin</h2>
<p>The use of <a href="nucleotide" class="uri" title="wikilink">nucleotide</a> sequence variations to investigate evolutionary relationships is not a new concept. <a href="Carl_Woese" title="wikilink">Carl Woese</a> used sequence differences in <a href="rRNA" title="wikilink">ribosomal RNA</a> (rRNA) to discover <a href="archaea" class="uri" title="wikilink">archaea</a>, which in turn led to the redrawing of the <a href="evolutionary_tree" title="wikilink">evolutionary tree</a>, and molecular markers (e.g., allozymes, rDNA, and mtDNAvage ) have been successfully used in molecular <a href="systematics" class="uri" title="wikilink">systematics</a> for decades. DNA barcoding provides a standardised method for this process via the use of a short DNA sequence from a particular region of the genome to provide a 'barcode' for identifying species. In 2003, <a href="Paul_D.N._Hebert" title="wikilink">Paul D.N. Hebert</a> from the <a href="University_of_Guelph" title="wikilink">University of Guelph</a>, <a href="Ontario" class="uri" title="wikilink">Ontario</a>, <a href="Canada" class="uri" title="wikilink">Canada</a>, proposed the compilation of a public library of DNA barcodes that would be linked to named <a href="specimen" title="wikilink">specimens</a>. This library would &lt;U+201C&gt;provide a new master key for identifying species, one whose power will rise with increased taxon coverage and with faster, cheaper sequencing&lt;U+201D&gt;.</p>
<h2 id="case-studies">Case studies</h2>
<h3 id="identification-of-birds">Identification of birds</h3>
<p>In an effort to find a correspondence between traditional species boundaries established by taxonomy and those inferred by DNA barcoding, Hebert and co-workers sequenced DNA barcodes of 260 of the 667 bird species that breed in <a href="North_America" title="wikilink">North America</a> (Hebert <em>et al.</em> 2004a<a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a>). They found that every single one of the 260 species had a different COI sequence. 130 species were represented by two or more specimens; in all of these species, COI sequences were either identical or were most similar to sequences of the same species. COI variations between species averaged 7.93%, whereas variation within species averaged 0.43%. In four cases there were deep intraspecific divergences, indicating possible new species. Three out of these four <a href="polytypic" class="uri" title="wikilink">polytypic</a> species are already split into two by some taxonomists. Hebert <em>et al.</em>'s (2004a<a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a>) results reinforce these views and strengthen the case for DNA barcoding. Hebert <em>et al.</em> also proposed a standard sequence threshold to define new species, this threshold, the so-called &quot;barcoding gap&quot;, was defined as 10 times the mean intraspecific variation for the group under study.</p>
<h3 id="delimiting-cryptic-species">Delimiting cryptic species</h3>
<p>The next major study into the efficacy of DNA barcoding was focused on the <a href="neotropical" class="uri" title="wikilink">neotropical</a> skipper butterfly, <em><a href="Astraptes" title="wikilink">Astraptes fulgerator</a></em> at the <a href="Area_Conservacion_de_Guanacaste" title="wikilink">Area Conservacion de Guanacaste</a> (ACG) in north-western <a href="Costa_Rica" title="wikilink">Costa Rica</a>. This species was already known as a <a href="cryptic_species_complex" title="wikilink">cryptic species complex</a>, due to subtle <a href="morphology_(biology)" title="wikilink">morphological</a> differences, as well as an unusually large variety of <a href="caterpillar" class="uri" title="wikilink">caterpillar</a> food plants. However, several years would have been required for taxonomists to completely delimit species. Hebert <em>et al.</em> (2004b<a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a>) sequenced the COI gene of 484 specimens from the ACG. This sample included &lt;U+201C&gt;at least 20 individuals reared from each species of food plant, extremes and intermediates of adult and caterpillar color variation, and representatives&lt;U+201D&gt; from the three major ecosystems where <em>Astraptes fulgerator</em> is found. Hebert <em>et al.</em> (2004b<a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a>) concluded that <em>Astraptes fulgerator</em> consists of 10 different species in north-western Costa Rica. These results, however, were subsequently challenged by Brower (2006<a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a>), who pointed out numerous serious flaws in the analysis, and concluded that the original data could support no more than the possibility of three to seven cryptic <a href="taxa" class="uri" title="wikilink">taxa</a> rather than ten cryptic species. This highlights that the results of DNA barcoding analyses can be dependent upon the choice of analytical methods used by the investigators, so the process of delimiting cryptic species using DNA barcodes can be as subjective as any other form of taxonomy.</p>
<p>A more recent example used DNA barcoding for the identification of cryptic species included in the ongoing long-term database of tropical caterpillar life generated by Dan Janzen and Winnie Hallwachs in Costa Rica at the ACG.<a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a> In 2006 Smith <em>et al.</em><a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a> examined whether a COI DNA barcode could function as a tool for identification and discovery for the 20 morphospecies of <em><a href="Belvosia" class="uri" title="wikilink">Belvosia</a></em> <a href="http://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&amp;search_value=650659">2</a> <a href="parasitoid" class="uri" title="wikilink">parasitoid</a> flies (<a href="Tachinidae" class="uri" title="wikilink">Tachinidae</a>) that have been reared from <a href="caterpillar" title="wikilink">caterpillars</a> in ACG. Barcoding not only discriminated among all 17 highly host-specific morphospecies of ACG <em>Belvosia</em>, but it also suggested that the species count could be as high as 32 by indicating that each of the three generalist species might actually be arrays of highly host-specific cryptic species.</p>
<p>In 2007 Smith <em>et al.</em> expanded on these results by barcoding 2,134 flies belonging to what appeared to be the 16 most generalist of the ACG tachinid morphospecies.<a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a> They encountered 73 mitochondrial lineages separated by an average of 4% sequence divergence and, as these lineages are supported by collateral ecological information, and, where tested, by independent nuclear markers (28S and ITS1), the authors therefore viewed these lineages as provisional species. Each of the 16 initially apparent generalist species were categorized into one of four patterns: (i) a single generalist species, (ii) a pair of morphologically cryptic generalist species, (iii) a complex of specialist species plus a generalist, or (iv) a complex of specialists with no remaining generalist. In sum, there remained 9 generalist species classified among the 73 mitochondrial lineages analyzed.</p>
<p>However, also in 2007, Whitworth <em>et al.</em> reported that flies in the related family <a href="Calliphoridae" class="uri" title="wikilink">Calliphoridae</a> could not be discriminated by barcoding.<a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a> They investigated the performance of barcoding in the fly genus <em><a href="Protocalliphora" class="uri" title="wikilink">Protocalliphora</a></em>, known to be infected with the <a href="endosymbiotic" class="uri" title="wikilink">endosymbiotic</a> bacteria <em><a href="Wolbachia" class="uri" title="wikilink">Wolbachia</a></em>. Assignment of unknown individuals to species was impossible for 60% of the species, and if the technique had been applied, as in the previous study, to identify new species, it would have underestimated the species number in the genus by 75%. They attributed the failure of barcoding to the non-monophyly of many of the species at the mitochondrial level; in one case, individuals from four different species had identical barcodes. The authors went on to state: </p>
<p>Marine biologists have also considered the value of the technique in identifying cryptic and polymorphic species and have suggested that the technique may be helpful when associations with voucher specimens are maintained,<a href="#fn38" class="footnote-ref" id="fnref38"><sup>38</sup></a> though cases of &quot;shared barcodes&quot; (e.g., non-unique) have been documented in <a href="cichlid" class="uri" title="wikilink">cichlid</a> fishes and <a href="cowries" class="uri" title="wikilink">cowries</a><a href="#fn39" class="footnote-ref" id="fnref39"><sup>39</sup></a></p>
<h3 id="cataloguing-ancient-life">Cataloguing ancient life</h3>
<p>Lambert <em>et al.</em> (2005<a href="#fn40" class="footnote-ref" id="fnref40"><sup>40</sup></a>) examined the possibility of using DNA barcoding to assess the past diversity of the Earth's <a href="biota_(ecology)" title="wikilink">biota</a>. The COI gene of a group of extinct <a href="ratite" class="uri" title="wikilink">ratite</a> birds, the <a href="moa" class="uri" title="wikilink">moa</a>, were sequenced using 26 <a href="subfossil" class="uri" title="wikilink">subfossil</a> moa bones. As with Hebert's results, each species sequenced had a unique barcode and intraspecific COI sequence variance ranged from 0 to 1.24%. To determine new species, a standard sequence threshold of 2.7% COI sequence difference was set. This value is 10 times the average intraspecies difference of North American birds, which is inconsistent with Hebert's recommendation that the threshold value be based on the group under study. Using this value, the group detected six moa species. In addition, a further standard sequence threshold of 1.24% was also used. This value resulted in 10 moa species which corresponded with the previously known species with one exception. This exception suggested a possible complex of species which was previously unidentified. Given the slow rate of growth and reproduction of moa, it is probable that the interspecies variation is rather low. On the other hand, there is no set value of molecular difference at which populations can be assumed to have irrevocably started to undergo <a href="speciation" class="uri" title="wikilink">speciation</a>. It is safe to say, however, that the 2.7% COI sequence difference initially used was far too high.</p>
<h3 id="the-moorea-biocode-project">The Moorea Biocode Project</h3>
<p>The <a href="http://www.mooreabiocode.org/%7CMoorea">Biocode Project</a> is a barcoding initiative to create the first comprehensive inventory of all non-microbial life in a complex tropical ecosystem, the island of Moorea in Tahiti. Supported by a grant from the <a href="Gordon_and_Betty_Moore_Foundation" title="wikilink">Gordon and Betty Moore Foundation</a>, the Moorea Biocode Project is a 3-year project that brings together researchers from the <a href="Smithsonian_Institution" title="wikilink">Smithsonian Institution</a>, <a href="UC_Berkeley" title="wikilink">UC Berkeley</a>, France&lt;U+2019&gt;s <a href="CNRS" title="wikilink">National Center for Scientific Research</a> (CNRS), and other partners. The outcome of the project is a library of genetic markers and physical identifiers for every species of plant, animal and fungi on the island that will be provided as a publicly available database resource for ecologists and evolutionary biologists around the world.</p>
<p>The software back-end to the Moore Biocode Project is <a href="Geneious" title="wikilink">Geneious Pro</a> and two custom-developed plugins from the <a href="New_Zealand" title="wikilink">New Zealand</a>-based company, <a href="Biomatters" class="uri" title="wikilink">Biomatters</a>. The <a href="http://software.mooreabiocode.org/index.php?title=Main_Page%7CGeneious">Biocode LIMS and Genbank Submission</a> plugins have been made freely available to the public<a href="#fn41" class="footnote-ref" id="fnref41"><sup>41</sup></a> and users of the free Geneious Basic software will be able to access and view the Biocode database upon completion of the project, while a commercial copy of Geneious Pro is required for researchers involved int data creation and analysis.</p>
<h2 id="criticisms">Criticisms</h2>
<p>DNA barcoding has met with spirited reaction from scientists, especially <a href="systematics" title="wikilink">systematists</a>, ranging from enthusiastic endorsement to vociferous opposition.<a href="#fn42" class="footnote-ref" id="fnref42"><sup>42</sup></a> For example, many stress the fact that DNA barcoding does not provide reliable information above the species level, while others indicate that it is inapplicable at the species level, but may still have merit for higher-level groups.<a href="#fn43" class="footnote-ref" id="fnref43"><sup>43</sup></a> Others resent what they see as a gross oversimplification of the science of taxonomy. And, more practically, some suggest that recently diverged species might not be distinguishable on the basis of their COI sequences.<a href="#fn44" class="footnote-ref" id="fnref44"><sup>44</sup></a> Due to various phenomena, Funk &amp; Omland (2003<a href="#fn45" class="footnote-ref" id="fnref45"><sup>45</sup></a>) found that some 23% of animal species are <a href="polyphyletic" class="uri" title="wikilink">polyphyletic</a> if their mtDNA data are accurate, indicating that using an mtDNA barcode to assign a species name to an animal will be ambiguous or erroneous some 23% of the time (see also Meyer &amp; Paulay, 2005<a href="#fn46" class="footnote-ref" id="fnref46"><sup>46</sup></a>). Studies with insects suggest an equal or even greater error rate, due to the frequent lack of correlation between the mitochondrial genome and the nuclear genome or the lack of a barcoding gap (e.g., Hurst and Jiggins, 2005,<a href="#fn47" class="footnote-ref" id="fnref47"><sup>47</sup></a> Whitworth <em>et al.</em>, 2007,<a href="#fn48" class="footnote-ref" id="fnref48"><sup>48</sup></a> Wiemers &amp; Fiedler, 2007<a href="#fn49" class="footnote-ref" id="fnref49"><sup>49</sup></a>). Problems with mtDNA arising from male-killing microoroganisms and cytoplasmic incompatibility-inducing symbionts (e.g., <em><a href="Wolbachia" class="uri" title="wikilink">Wolbachia</a></em>)<a href="#fn50" class="footnote-ref" id="fnref50"><sup>50</sup></a> are also particularly common among insects. Given that insects represent over 75% of all known organisms,<a href="#fn51" class="footnote-ref" id="fnref51"><sup>51</sup></a> this suggests that while mtDNA barcoding may work for <a href="vertebrate" title="wikilink">vertebrates</a>, it may not be effective for the majority of known organisms.</p>
<p>Moritz and Cicero (2004<a href="#fn52" class="footnote-ref" id="fnref52"><sup>52</sup></a>) have questioned the efficacy of DNA barcoding by suggesting that other avian data is inconsistent with Hebert <em>et al.</em>'s interpretation, namely, Johnson and Cicero's (2004<a href="#fn53" class="footnote-ref" id="fnref53"><sup>53</sup></a>) finding that 74% of sister species comparisons fall below the 2.7% threshold suggested by Hebert <em>et al.</em> These criticisms are somewhat misleading considering that, of the 39 species comparisons reported by Johnson and Cicero, only 8 actually use COI data to arrive at their conclusions. Johnson and Cicero (2004<a href="#fn54" class="footnote-ref" id="fnref54"><sup>54</sup></a>) have also claimed to have detected bird species with identical DNA barcodes, however, these 'barcodes' refer to an unpublished 723-bp sequence of ND6 which has never been suggested as a likely candidate for DNA barcoding.</p>
<p>The DNA barcoding debate resembles the <a href="phenetics" class="uri" title="wikilink">phenetics</a> debate of decades gone by. It remains to be seen whether what is now touted as a revolution in taxonomy will eventually go the same way as phenetic approaches, of which was claimed exactly the same decades ago, but which were all but rejected when they failed to live up to overblown expectations.<a href="#fn55" class="footnote-ref" id="fnref55"><sup>55</sup></a> Controversy surrounding DNA barcoding stems not so much from the method itself, but rather from extravagant claims that it will supersede or radically transform traditional taxonomy. Other critics fear a &quot;<a href="big_science" title="wikilink">big science</a>&quot; initiative like barcoding will make funding even more scarce for already underfunded disciplines like <a href="taxonomy" class="uri" title="wikilink">taxonomy</a>, but barcoders respond that they compete for funding not with fields like taxonomy, but instead with other big science fields, such as <a href="medicine" class="uri" title="wikilink">medicine</a> and <a href="genomics" class="uri" title="wikilink">genomics</a>.<a href="#fn56" class="footnote-ref" id="fnref56"><sup>56</sup></a> Barcoders also maintain that they are being dragged into <a href="species_problem" title="wikilink">long-standing debates over the definition of a species</a> and that barcoding is less controversial when viewed primarily as a method of identification, not classification.<a href="#fn57" class="footnote-ref" id="fnref57"><sup>57</sup></a><a href="#fn58" class="footnote-ref" id="fnref58"><sup>58</sup></a></p>
<p>The current trend appears to be that DNA barcoding needs to be used alongside traditional <a href="taxonomy" title="wikilink">taxonomic</a> tools and alternative forms of molecular systematics so that problem cases can be identified and errors detected. Non-cryptic species can generally be resolved by either traditional or molecular taxonomy without ambiguity. However, more difficult cases will only yield to a combination of approaches. And finally, as most of the global <a href="biodiversity" class="uri" title="wikilink">biodiversity</a> remains unknown, molecular barcoding can only hint at the existence of new taxa, but not delimit or describe them (DeSalle, 2006;<a href="#fn59" class="footnote-ref" id="fnref59"><sup>59</sup></a> Rubinoff, 2006<a href="#fn60" class="footnote-ref" id="fnref60"><sup>60</sup></a><a href="#fn61" class="footnote-ref" id="fnref61"><sup>61</sup></a>).</p>
<h2 id="dna-barcoding-software">DNA Barcoding Software</h2>
<p>Software for DNA barcoding requires integration of a field information management system (FIMS), laboratory information management system (LIMS), sequence analysis tools, workflow tracking to connect field data and laboratory data, database submission tools and pipeline automation for scaling up to eco-system scale projects. <a href="Geneious" title="wikilink">Geneious Pro</a> can be used for the sequence analysis components, and the two plugins made freely available through the Moorea Biocode Project, the <a href="http://software.mooreabiocode.org/index.php?title=Main_Page%7CGeneious">Biocode LIMS and Genbank Submission</a> plugins handle integration with the FIMS, the LIMS, workflow tracking and database submission.</p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="DNA_taxonomy" title="wikilink">DNA taxonomy</a></li>
<li><a href="Consortium_for_the_Barcode_of_Life" title="wikilink">Consortium for the Barcode of Life</a></li>
<li><a href="Identification_(biology)" title="wikilink">Identification (biology)</a></li>
</ul>
<h2 id="references">References</h2>

<h2 id="external-links">External links</h2>
<ul>
<li><a href="http://www.boldsystems.org/">Barcode of Life Data Systems</a></li>
<li><a href="http://www.ibol.org/">International Barcode of Life</a></li>
<li><a href="http://www.barcodeoflife.org">Consortium for the Barcode of Life</a></li>
<li><a href="http://www.fishbol.org">Fish Barcode of Life Initiative (FISH-BOL)</a></li>
<li><a href="http://barcoding.si.edu/AllBirds.htm">All Birds Barcoding Initiative (ABBI)</a></li>
<li><a href="http://www.polarbarcoding.org">Polar Flora and Fauna Barcoding website</a> (Latest outpost in the Canadian Arctic in the field)</li>
<li><a href="http://phe.rockefeller.edu/barcode/blog/">The Barcode of Life Blog</a></li>
<li><a href="http://connect.barcodeoflife.net/">DNA Barcoding Community Network</a></li>
<li><a href="http://www.barcoding.si.edu/PDF/Guidelines%20for%20non-CO1%20selection%20-%204%20June.pdf">Guidelines for non COI gene selection</a></li>
</ul>


<p><a href="ar:&lt;U+0634&gt;&lt;U+0641&gt;&lt;U+0631&gt;&lt;U+0629&gt;_&lt;U+0627&gt;&lt;U+0644&gt;&lt;U+062F&gt;&lt;U+0646&gt;&lt;U+0627&gt;_&lt;U+0627&gt;&lt;U+0644&gt;&lt;U+062E&gt;&lt;U+064A&gt;&lt;U+0637&gt;&lt;U+064A&gt;&lt;U+0629&gt;" title="wikilink">ar:&lt;U+0634&gt;&lt;U+0641&gt;&lt;U+0631&gt;&lt;U+0629&gt; &lt;U+0627&gt;&lt;U+0644&gt;&lt;U+062F&gt;&lt;U+0646&gt;&lt;U+0627&gt; &lt;U+0627&gt;&lt;U+0644&gt;&lt;U+062E&gt;&lt;U+064A&gt;&lt;U+0637&gt;&lt;U+064A&gt;&lt;U+0629&gt;</a> <a href="ca:DNA_barcoding" title="wikilink">ca:DNA barcoding</a> <a href="de:DNA_barcoding" title="wikilink">de:DNA barcoding</a> <a href="fr:Barcoding_mol&lt;U+00E9&gt;culaire" title="wikilink">fr:Barcoding mol&lt;U+00E9&gt;culaire</a> <a href="id:Kode_batang_DNA" title="wikilink">id:Kode batang DNA</a> <a href="it:DNA_barcoding" title="wikilink">it:DNA barcoding</a> <a href="nl:DNA-barcoding" class="uri" title="wikilink">nl:DNA-barcoding</a> <a href="ja:DNA&lt;U+30D0&gt;&lt;U+30FC&gt;&lt;U+30B3&gt;&lt;U+30FC&gt;&lt;U+30C7&gt;&lt;U+30A3&gt;&lt;U+30F3&gt;&lt;U+30B0&gt;" class="uri" title="wikilink">ja:DNA&lt;U+30D0&gt;&lt;U+30FC&gt;&lt;U+30B3&gt;&lt;U+30FC&gt;&lt;U+30C7&gt;&lt;U+30A3&gt;&lt;U+30F3&gt;&lt;U+30B0&gt;</a> <a href="pl:Barkod" class="uri" title="wikilink">pl:Barkod</a></p>
<p><a href="Category:Taxonomy" class="uri" title="wikilink">Category:Taxonomy</a> <a href="Category:Molecular_genetics" title="wikilink">Category:Molecular genetics</a> <a href="Category:Bioinformatics" class="uri" title="wikilink">Category:Bioinformatics</a> <a href="Category:Authentication_methods" title="wikilink">Category:Authentication methods</a> <a href="Category:Biometrics" class="uri" title="wikilink">Category:Biometrics</a></p>
<section class="footnotes">
<hr />
<ol>
<li id="fn1"></li>
<li id="fn2">Koch, H. 2010. Combining morphology and DNA barcoding resolves the taxonomy of Western Malagasy <em>Liotrigona</em> Moure, 1961. <em><a href="African_Invertebrates" title="wikilink">African Invertebrates</a></em> <strong>51</strong> (2): 413-421.<a href="http://www.africaninvertebrates.org.za/Koch_2010_51_2_474.aspx">1</a><a href="http://www.tb1.ethz.ch/PublicationsEO/PDFpapers/Koch_AFRICAN_INVERTEBRATES_2010_51_413-421.pdf">PDF fulltext</a><a href="#fnref2" class="footnote-back">↩</a></li>
<li id="fn3"><a href="#fnref3" class="footnote-back">↩</a></li>
<li id="fn4"><a href="#fnref4" class="footnote-back">↩</a></li>
<li id="fn5"></li>
<li id="fn6"></li>
<li id="fn7"></li>
<li id="fn8"><a href="#fnref8" class="footnote-back">↩</a></li>
<li id="fn9"><a href="#fnref9" class="footnote-back">↩</a></li>
<li id="fn10"><a href="#fnref10" class="footnote-back">↩</a></li>
<li id="fn11"><a href="#fnref11" class="footnote-back">↩</a></li>
<li id="fn12"><a href="#fnref12" class="footnote-back">↩</a></li>
<li id="fn13"><a href="#fnref13" class="footnote-back">↩</a></li>
<li id="fn14"><a href="#fnref14" class="footnote-back">↩</a></li>
<li id="fn15"></li>
<li id="fn16"><a href="#fnref16" class="footnote-back">↩</a></li>
<li id="fn17"><a href="#fnref17" class="footnote-back">↩</a></li>
<li id="fn18"><a href="#fnref18" class="footnote-back">↩</a></li>
<li id="fn19"><a href="#fnref19" class="footnote-back">↩</a></li>
<li id="fn20"><a href="#fnref20" class="footnote-back">↩</a></li>
<li id="fn21"> <a href="http://www.pnas.org/cgi/content/full/0503123102/DC1">Supporting Information</a><a href="#fnref21" class="footnote-back">↩</a></li>
<li id="fn22"></li>
<li id="fn23"></li>
<li id="fn24"></li>
<li id="fn25"></li>
<li id="fn26"></li>
<li id="fn27"></li>
<li id="fn28"><a href="#fnref28" class="footnote-back">↩</a></li>
<li id="fn29"> <a href="http://biology.plosjournals.org/archive/1545-7885/2/10/supinfo/10.1371_journal.pbio.0020312.sg001.pdf">Supporting Information</a><a href="#fnref29" class="footnote-back">↩</a></li>
<li id="fn30"></li>
<li id="fn31"> <a href="http://www.pnas.org/cgi/content/full/0406166101/DC1">Supporting Information</a><a href="#fnref31" class="footnote-back">↩</a></li>
<li id="fn32"></li>
<li id="fn33"><a href="#fnref33" class="footnote-back">↩</a></li>
<li id="fn34"><a href="#fnref34" class="footnote-back">↩</a></li>
<li id="fn35"><a href="#fnref35" class="footnote-back">↩</a></li>
<li id="fn36"><a href="#fnref36" class="footnote-back">↩</a></li>
<li id="fn37"></li>
<li id="fn38"><a href="#fnref38" class="footnote-back">↩</a></li>
<li id="fn39"></li>
<li id="fn40"><a href="#fnref40" class="footnote-back">↩</a></li>
<li id="fn41"><a href="http://www.bio-itworld.com/2010/11/30/biomatters-moorea-LIMS.html" class="uri">http://www.bio-itworld.com/2010/11/30/biomatters-moorea-LIMS.html</a><a href="#fnref41" class="footnote-back">↩</a></li>
<li id="fn42"><a href="#fnref42" class="footnote-back">↩</a></li>
<li id="fn43"></li>
<li id="fn44">Kevin, C.R. Kerr, Mark Y. Stoeckle, Carla J. Dove, Lee A. Weigt, Charles M. Francis &amp; Paul D. N. Hebert. 2006. Comprehensive DNA barcode coverage of North American birds. Molecular Ecology Notes. (OnlineEarly Articles). <a href="doi:10.1111/j.1471-8286.2006.01670.x" class="uri">doi:10.1111/j.1471-8286.2006.01670.x</a> <a href="http://www.blackwell-synergy.com/doi/full/10.1111/j.1471-8286.2006.01670.x">Full text</a><a href="#fnref44" class="footnote-back">↩</a></li>
<li id="fn45"><a href="#fnref45" class="footnote-back">↩</a></li>
<li id="fn46"><a href="#fnref46" class="footnote-back">↩</a></li>
<li id="fn47"></li>
<li id="fn48"></li>
<li id="fn49"><a href="#fnref49" class="footnote-back">↩</a></li>
<li id="fn50"></li>
<li id="fn51"><a href="http://www.environment.gov.au/biodiversity/abrs/publications/other/species-numbers/02-exec-summary.html#allspecies">3</a><a href="#fnref51" class="footnote-back">↩</a></li>
<li id="fn52"><a href="#fnref52" class="footnote-back">↩</a></li>
<li id="fn53"><a href="#fnref53" class="footnote-back">↩</a></li>
<li id="fn54"></li>
<li id="fn55"><a href="#fnref55" class="footnote-back">↩</a></li>
<li id="fn56"><a href="#fnref56" class="footnote-back">↩</a></li>
<li id="fn57"></li>
<li id="fn58"></li>
<li id="fn59"><a href="#fnref59" class="footnote-back">↩</a></li>
<li id="fn60"><a href="#fnref60" class="footnote-back">↩</a></li>
<li id="fn61"><a href="#fnref61" class="footnote-back">↩</a></li>
</ol>
</section>
