<p><img src="brokechromo.jpg" title="fig:DNA damage resulting in multiple broken chromosomes" alt="DNA damage resulting in multiple broken chromosomes" /> <strong>DNA repair</strong> refers to a collection of processes by which a <a href="cell_(biology)" title="wikilink">cell</a> identifies and corrects damage to the <a href="DNA" class="uri" title="wikilink">DNA</a> molecules that encode its <a href="genome" class="uri" title="wikilink">genome</a>. In human cells, both normal <a href="metabolism" title="wikilink">metabolic</a> activities and environmental factors such as <a href="UV" class="uri" title="wikilink">UV</a> light can cause DNA damage, resulting in as many as 1 million individual <a href="molecular_lesion" title="wikilink">molecular lesions</a> per cell per day.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a> Many of these lesions cause structural damage to the DNA molecule and can alter or eliminate the cell's ability to <a href="Transcription_(genetics)" title="wikilink">transcribe</a> the <a href="gene" class="uri" title="wikilink">gene</a> that the affected DNA encodes. Other lesions induce potentially harmful <a href="mutation" title="wikilink">mutations</a> in the cell's genome, which affect the survival of its daughter cells after it undergoes <a href="mitosis" class="uri" title="wikilink">mitosis</a>. Consequently, the DNA repair process must be constantly active so it can respond rapidly to any damage in the DNA structure.</p>
<p>The rate of DNA repair is dependent on many factors, including the cell type, the age of the cell, and the extracellular environment. A cell that has accumulated a large amount of DNA damage, or one that no longer effectively repairs damage incurred by its DNA, can enter one of three possible states:</p>
<ol>
<li>an irreversible state of dormancy, known as <a href="senescence" class="uri" title="wikilink">senescence</a></li>
<li>cell suicide, also known as <a href="apoptosis" class="uri" title="wikilink">apoptosis</a> or <a href="programmed_cell_death" title="wikilink">programmed cell death</a></li>
<li>unregulated cell division, which can lead to the formation of a <a href="tumor" class="uri" title="wikilink">tumor</a> that is <a href="cancer" title="wikilink">cancerous</a></li>
</ol>
<p>The DNA repair ability of a cell is vital to the integrity of its genome and thus to its normal functioning and that of the organism. Many genes that were initially shown to influence <a href="lifespan" class="uri" title="wikilink">lifespan</a> have turned out to be involved in DNA damage repair and protection.<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a> Failure to correct molecular lesions in cells that form <a href="gamete" title="wikilink">gametes</a> can introduce mutations into the genomes of the offspring and thus influence the rate of <a href="evolution" class="uri" title="wikilink">evolution</a>.</p>
<h2 id="dna-damage">DNA damage</h2>
<p>DNA damage, due to environmental factors and normal <a href="metabolism" title="wikilink">metabolic</a> processes inside the cell, occurs at a rate of 1,000 to 1,000,000 molecular lesions per cell per day.<a href="#fn3" class="footnote-ref" id="fnref3"><sup>3</sup></a> While this constitutes only 0.000165% of the human genome's approximately 6 billion bases (3 billion base pairs), unrepaired lesions in critical genes (such as <a href="tumor_suppressor_gene" title="wikilink">tumor suppressor genes</a>) can impede a cell's ability to carry out its function and appreciably increase the likelihood of <a href="tumor" class="uri" title="wikilink">tumor</a> formation.</p>
<p>The vast majority of DNA damage affects the <a href="primary_structure" title="wikilink">primary structure</a> of the double helix; that is, the bases themselves are chemically modified. These modifications can in turn disrupt the molecules' regular helical structure by introducing non-native chemical bonds or bulky adducts that do not fit in the standard double helix. Unlike <a href="protein" title="wikilink">proteins</a> and <a href="RNA" class="uri" title="wikilink">RNA</a>, DNA usually lacks <a href="tertiary_structure" title="wikilink">tertiary structure</a> and therefore damage or disturbance does not occur at that level. DNA is, however, <a href="supercoil" title="wikilink">supercoiled</a> and wound around &quot;packaging&quot; proteins called <a href="histone" title="wikilink">histones</a>, and both superstructures are vulnerable to the effects of DNA damage.</p>
<h3 id="sources-of-damage">Sources of damage</h3>
<p>DNA damage can be subdivided into two main types:</p>
<ol>
<li><a href="endogenous" class="uri" title="wikilink">endogenous</a> damage such as attack by <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a> produced from normal metabolic byproducts (spontaneous mutation), especially the process of <a href="oxidative_deamination" title="wikilink">oxidative deamination</a>;</li>
<li>exogenous damage caused by external agents such as
<ol>
<li>ultraviolet [UV 200-300<a href="nanometre" title="wikilink">nm</a>] <a href="radiation" class="uri" title="wikilink">radiation</a> from the sun</li>
<li>other radiation frequencies, including <a href="x-ray" title="wikilink">x-rays</a> and <a href="gamma_ray" title="wikilink">gamma rays</a></li>
<li><a href="hydrolysis" class="uri" title="wikilink">hydrolysis</a> or thermal disruption</li>
<li>certain <a href="plant" class="uri" title="wikilink">plant</a> <a href="toxin" title="wikilink">toxins</a></li>
<li>human-made <a href="mutagen" title="wikilink">mutagenic chemicals</a>, especially <a href="aromatic" class="uri" title="wikilink">aromatic</a> compounds that act as DNA <a href="intercalation_(chemistry)" title="wikilink">intercalating agents</a></li>
<li>cancer <a href="chemotherapy" class="uri" title="wikilink">chemotherapy</a> and <a href="radiotherapy" class="uri" title="wikilink">radiotherapy</a></li>
</ol></li>
</ol>
<p>The replication of damaged DNA before cell division can lead to the incorporation of wrong bases opposite damaged ones. Daughter cells that inherit these wrong bases carry mutations from which the original DNA sequence is unrecoverable (except in the rare case of a <a href="back_mutation" title="wikilink">back mutation</a>, for example, through <a href="gene_conversion" title="wikilink">gene conversion</a>).</p>
<h3 id="types-of-damage">Types of damage</h3>
<p>There are four main types of damage to DNA due to endogenous cellular processes:</p>
<ol>
<li><em><a href="oxidation" class="uri" title="wikilink">oxidation</a></em> of bases [e.g. 8-oxo-7,8-dihydroguanine (8-oxoG)] and generation of DNA strand interruptions from reactive oxygen species,</li>
<li><em><a href="alkylation" class="uri" title="wikilink">alkylation</a></em> of bases (usually <a href="methylation" class="uri" title="wikilink">methylation</a>), such as formation of 7-methylguanine</li>
<li><em><a href="hydrolysis" class="uri" title="wikilink">hydrolysis</a></em> of bases, such as deamination, depurination and depyrimidination.</li>
<li><em>mismatch</em> of bases, due to <a href="DNA_replication" title="wikilink">DNA replication</a> in which the wrong DNA base is stitched into place in a newly forming DNA strand.</li>
</ol>
<p>Damage caused by exogenous agents comes in many forms. Some examples are:</p>
<ol>
<li><em><a href="UV_light" title="wikilink">UV light</a></em> causes crosslinking between adjacent cytosine and thymine bases creating <em><a href="pyrimidine_dimers" title="wikilink">pyrimidine dimers</a></em></li>
<li><em><a href="Ionizing_radiation" title="wikilink">Ionizing radiation</a></em> such as that created by radioactive decay or in <em><a href="cosmic_rays" title="wikilink">cosmic rays</a></em> causes breaks in DNA strands.</li>
<li><em>Thermal disruption</em> at elevated temperature increases the rate of <a href="depurination" class="uri" title="wikilink">depurination</a> (loss of <a href="purine" class="uri" title="wikilink">purine</a> bases from the DNA backbone) and single strand breaks. For example, hydrolytic depurination is seen in the <a href="thermophilic_bacteria" title="wikilink">thermophilic bacteria</a>, which grow in <a href="hot_springs" title="wikilink">hot springs</a> at 85-250<sup>o</sup>C.<a href="#fn4" class="footnote-ref" id="fnref4"><sup>4</sup></a> The rate of depurination (300 <a href="purine" class="uri" title="wikilink">purine</a> residues per genome per generation) is too high in these species to be repaired by normal repair machinery, hence a possibility of an <a href="adaptive" class="uri" title="wikilink">adaptive</a> response cannot be ruled out.</li>
<li><em>Industrial chemicals</em> such as vinyl chloride and hydrogen peroxide, and environmental chemicals such as polycyclic hydrocarbons found in smoke, soot and tar create a huge diversity of DNA adducts- ethenobases, oxidized bases, alkylated phosphotriesters and <a href="Crosslinking_of_DNA" title="wikilink">Crosslinking of DNA</a> just to name a few.</li>
</ol>
<h3 id="nuclear-versus-mitochondrial-dna-damage">Nuclear versus mitochondrial DNA damage</h3>
<p>In human cells, and <a href="eukaryotic" class="uri" title="wikilink">eukaryotic</a> cells in general, DNA is found in two cellular locations - inside the <a href="Cell_nucleus" title="wikilink">nucleus</a> and inside the <a href="mitochondria" class="uri" title="wikilink">mitochondria</a>. Nuclear DNA (nDNA) exists as <a href="chromatin" class="uri" title="wikilink">chromatin</a> during non-replicative stages of the <a href="cell_cycle" title="wikilink">cell cycle</a> and is condensed into aggregate structures known as <a href="chromosomes" class="uri" title="wikilink">chromosomes</a> during <a href="cell_division" title="wikilink">cell division</a>. In either state the DNA is highly compacted and wound up around bead-like proteins called <a href="histones" class="uri" title="wikilink">histones</a>. Whenever a cell needs to express the genetic information encoded in its nDNA the required chromosomal region is unravelled, genes located therein are expressed, and then the region is condensed back to its resting conformation. Mitochondrial DNA (mtDNA) is located inside mitochondria <a href="organelles" class="uri" title="wikilink">organelles</a>, exists in multiple copies, and is also tightly associated with a number of proteins to form a complex known as the nucleoid. Inside mitochondria, <a href="reactive_oxygen_species" title="wikilink">reactive oxygen species</a> (ROS), or <a href="radical_(chemistry)" title="wikilink">free radicals</a>, byproducts of the constant production of <a href="adenosine_triphosphate" title="wikilink">adenosine triphosphate</a> (ATP) via <a href="oxidative_phosphorylation" title="wikilink">oxidative phosphorylation</a>, create a highly oxidative environment that is known to damage mtDNA. A critical enzyme in counteracting the toxicity of these species is <a href="superoxide_dismutase" title="wikilink">superoxide dismutase</a>, which is present in both the mitochondria and <a href="cytoplasm" class="uri" title="wikilink">cytoplasm</a> of eukaryotic cells.</p>
<h3 id="senescence-and-apoptosis">Senescence and apoptosis</h3>
<p>Senescence, an irreversible state in which the cell no longer divides (<a href="mitosis" class="uri" title="wikilink">mitosis</a>), is a protective response to the shortening of the chromosome ends (<a href="telomeres" class="uri" title="wikilink">telomeres</a>). The telomeres are long regions of repetitive <a href="noncoding_DNA" title="wikilink">noncoding DNA</a> that cap chromosomes and undergo partial degradation each time a cell undergoes division (see <a href="Hayflick_limit" title="wikilink">Hayflick limit</a>).<a href="#fn5" class="footnote-ref" id="fnref5"><sup>5</sup></a> In contrast, <a href="quiescence" class="uri" title="wikilink">quiescence</a> is a reversible state of cellular dormancy that is unrelated to genome damage (see <a href="cell_cycle" title="wikilink">cell cycle</a>). Senescence in cells may serve as a functional alternative to apoptosis in cases where the physical presence of a cell for spatial reasons is required by the organism,<a href="#fn6" class="footnote-ref" id="fnref6"><sup>6</sup></a> which serves as a &quot;last resort&quot; mechanism to prevent a cell with damaged DNA from replicating inappropriately in the absence of pro-growth <a href="cellular_signaling" title="wikilink">cellular signaling</a>. Unregulated cell division can lead to the formation of a tumor (see <a href="cancer" class="uri" title="wikilink">cancer</a>), which is potentially lethal to an organism. Therefore the induction of senescence and apoptosis is considered to be part of a strategy of protection against cancer.</p>
<h2 id="dna-repair-mechanisms">DNA repair mechanisms</h2>
<figure>
<img src="Ssvsds.jpg" title="Single strand and double strand DNA damage" alt="Single strand and double strand DNA damage" /><figcaption>Single strand and double strand DNA damage</figcaption>
</figure>
<p>Cells cannot function if DNA damage corrupts the integrity and accessibility of essential information in the <a href="genome" class="uri" title="wikilink">genome</a> (but cells remain superficially functional when so-called &quot;non-essential&quot; genes are missing or damaged). Depending on the type of damage inflicted on the DNA's double helical structure, a variety of repair strategies have evolved to restore lost information. If possible, cells use the unmodified complementary strand of the DNA or the sister <a href="chromatid" class="uri" title="wikilink">chromatid</a> as a template to losslessly recover the original information. Without access to a template, cells use an error-prone recovery mechanism known as translesion synthesis as a last resort.</p>
<p>Damage to DNA alters the spatial configuration of the helix and such alterations can be detected by the cell. Once damage is localized, specific DNA repair molecules are summoned to, and bind at or near the site of damage, inducing other molecules to bind and form a complex that enables the actual repair to take place. The types of molecules involved and the mechanism of repair that is mobilized depend on the type of damage that has occurred and the phase of the <a href="cell_cycle" title="wikilink">cell cycle</a> that the cell is in.</p>
<h3 id="direct-reversal">Direct reversal</h3>
<p>Cells are known to eliminate three types of damage to their DNA by chemically reversing it. These mechanisms do not require a template, since the types of damage they counteract can only occur in one of the four bases. Such direct reversal mechanisms are specific to the type of damage incurred. The formation of <a href="thymine_dimer" title="wikilink">thymine dimers</a> (a common type of <a href="cyclobutyl_dimer" title="wikilink">cyclobutyl dimer</a>) upon irradiation with UV light results in an abnormal covalent bond between adjacent thymidine bases. The photoreactivation process in bacteria directly reverses this damage by the action of the enzyme <a href="photolyase" class="uri" title="wikilink">photolyase</a>, which uses energy absorbed from UV light to promote catalysis. Another type of damage, methylation of guanine bases, is directly reversed by the protein methyl guanine methyl transferase (MGMT),the bacterial equivalent of which is called as <a href="ogt" class="uri" title="wikilink">ogt</a>. This is an expensive process because each MGMT molecule can only be used once; that is, the reaction is <a href="stoichiometric" class="uri" title="wikilink">stoichiometric</a> rather than <a href="catalytic" class="uri" title="wikilink">catalytic</a>.<a href="#fn7" class="footnote-ref" id="fnref7"><sup>7</sup></a> A generalized response to methylating agents in bacteria is known as the <a href="adaptive_response" title="wikilink">adaptive response</a> and confers a level of resistance to alkylating agents upon sustained exposure.<a href="#fn8" class="footnote-ref" id="fnref8"><sup>8</sup></a>The third type of DNA damage reversed by cells is certain methylation of the bases cytosine and adenine.</p>
<h3 id="single-strand-damage">Single strand damage</h3>
<p><img src="Uracil_base_glycosidase.jpg" title="fig:Structure of the base-excision repair enzyme uracil-DNA glycosylase. The uracil residue is shown in yellow." alt="Structure of the base-excision repair enzyme uracil-DNA glycosylase. The uracil residue is shown in yellow." width="250" /> When only one of the two strands of a double helix has a defect, the other strand can be used as a template to guide the correction of the damaged strand. In order to repair damage to one of the two paired molecules of DNA, there exist a number of excision repair mechanisms that remove the damaged nucleotide and replace it with an undamaged nucleotide complementary to that found in the undamaged DNA strand.<a href="#fn9" class="footnote-ref" id="fnref9"><sup>9</sup></a></p>
<ol>
<li><a href="Base_excision_repair" title="wikilink">Base excision repair</a> (BER), which repairs damage due to a single nucleotide caused by oxidation, alkylation, hydrolysis, or deamination;</li>
<li><a href="Nucleotide_excision_repair" title="wikilink">Nucleotide excision repair</a> (NER), which repairs damage affecting longer strands of 2-30 bases. This process recognizes bulky, helix-distorting changes such as <a href="thymine_dimer" title="wikilink">thymine dimers</a> as well as single-strand breaks (repaired with enzymes such <a href="UvrABC_endonuclease" title="wikilink">UvrABC endonuclease</a>). A specialized form of NER known as <a href="Transcription-Coupled_Repair" title="wikilink">Transcription-Coupled Repair</a> (TCR) deploys high-priority NER repair enzymes to genes that are being actively <a href="transcription_(genetics)" title="wikilink">transcribed</a>;</li>
<li><a href="Mismatch_repair" title="wikilink">Mismatch repair</a> (MMR), which corrects errors of <a href="DNA_replication" title="wikilink">DNA replication</a> and <a href="genetic_recombination" title="wikilink">recombination</a> that result in mispaired nucleotides following DNA replication.<br />
</li>
</ol>
<h3 id="double-strand-breaks">Double-strand breaks</h3>
<p>Double-strand breaks (DSBs), in which both strands in the double helix are severed, are particularly hazardous to the cell because they can lead to genome rearrangements. Two mechanisms exist to repair DSBs: <a href="non-homologous_end_joining" title="wikilink">non-homologous end joining</a> (NHEJ) and <a href="genetic_recombination" title="wikilink">recombinational repair</a> (also known as template-assisted repair or homologous recombination repair).<a href="#fn10" class="footnote-ref" id="fnref10"><sup>10</sup></a> <img src="DNA_Repair.jpg" title="fig:DNA ligase, shown above repairing chromosomal damage, is an enzyme that joins broken nucleotides together by catalyzing the formation of an internucleotide ester bond between the phosphate backbone and the deoxyribose nucleotides." alt="DNA ligase, shown above repairing chromosomal damage, is an enzyme that joins broken nucleotides together by catalyzing the formation of an internucleotide ester bond between the phosphate backbone and the deoxyribose nucleotides." width="230" /> In NHEJ, <a href="DNA_Ligase_IV" title="wikilink">DNA Ligase IV</a>, a specialized <a href="DNA_Ligase" title="wikilink">DNA Ligase</a> that forms a complex with the cofactor <a href="XRCC4" class="uri" title="wikilink">XRCC4</a>, directly joins the two ends.<a href="#fn11" class="footnote-ref" id="fnref11"><sup>11</sup></a> To guide accurate repair, NHEJ relies on short homologous sequences called microhomologies present on the single-stranded tails of the DNA ends to be joined. If these overhangs are compatible, repair is usually accurate.<a href="#fn12" class="footnote-ref" id="fnref12"><sup>12</sup></a><a href="#fn13" class="footnote-ref" id="fnref13"><sup>13</sup></a><a href="#fn14" class="footnote-ref" id="fnref14"><sup>14</sup></a><a href="#fn15" class="footnote-ref" id="fnref15"><sup>15</sup></a> NHEJ can also introduce mutations during repair. Loss of damaged nucleotides at the break site can lead to deletions, and joining of nonmatching termini forms translocations. NHEJ is especially important before the cell has replicated its DNA, since there is no template available for repair by homologous recombination. There are &quot;backup&quot; NHEJ pathways in higher <a href="eukaryote" title="wikilink">eukaryotes</a>.<a href="#fn16" class="footnote-ref" id="fnref16"><sup>16</sup></a> Besides its role as a genome caretaker, NHEJ is required for joining hairpin-capped double-strand breaks induced during <a href="V(D)J_recombination" title="wikilink">V(D)J recombination</a>, the process that generates diversity in <a href="B-cell_receptor" title="wikilink">B-cell</a> and <a href="T-cell_receptor" title="wikilink">T-cell receptors</a> in the <a href="vertebrate" class="uri" title="wikilink">vertebrate</a> <a href="immune_system" title="wikilink">immune system</a>.<a href="#fn17" class="footnote-ref" id="fnref17"><sup>17</sup></a></p>
<p>Recombinational repair requires the presence of an identical or nearly identical sequence to be used as a template for repair of the break. The enzymatic machinery responsible for this repair process is nearly identical to the machinery responsible for <a href="chromosomal_crossover" title="wikilink">chromosomal crossover</a> during meiosis. This pathway allows a damaged chromosome to be repaired using a sister <a href="chromatid" class="uri" title="wikilink">chromatid</a> or a <a href="homologous_chromosome" title="wikilink">homologous chromosome</a> as a template. DSBs caused by the replication machinery attempting to synthesize across a single-strand break or unrepaired lesion cause collapse of the <a href="replication_fork" title="wikilink">replication fork</a> and are typically repaired by recombination.</p>
<p><a href="Topoisomerase" title="wikilink">Topoisomerases</a> introduce both single- and double-strand breaks in the course of changing the DNA's state of <a href="supercoil" title="wikilink">supercoiling</a>, which is especially common in regions near an open replication fork. Such breaks are not considered DNA damage because they are a natural intermediate in the topoisomerase biochemical mechanism and are immediately repaired by the enzymes that created them.</p>
<p>A team of French researchers bombarded <a href="Deinococcus_radiodurans" title="wikilink">Deinococcus radiodurans</a> to study the mechanism of double-strand break DNA repair in that organism. At least two copies of the genome, with random DNA breaks, can form DNA fragments through <a href="Annealing_(biology)" title="wikilink">annealing</a>. Partially overlapping fragments are then used for synthesis of <a href="Homology_(biology)#Homology_of_sequences_in_genetics" title="wikilink">homologous</a> regions through a moving <a href="D-loop" class="uri" title="wikilink">D-loop</a> that can continue extension until they find <a href="complementary" class="uri" title="wikilink">complementary</a> partner strands. In the final step there is <a href="Chromosomal_crossover" title="wikilink">crossover</a> by means of <a href="RecA" class="uri" title="wikilink">RecA</a>-dependent <a href="Chromosomal_crossover" title="wikilink">homologous recombination</a>.<a href="#fn18" class="footnote-ref" id="fnref18"><sup>18</sup></a></p>
<h3 id="translesion-synthesis">Translesion synthesis</h3>
<p>Translesion synthesis allows the <a href="DNA_replication" title="wikilink">DNA replication</a> machinery to replicate past damaged DNA. This involves the use of specialized translesion <a href="DNA_polymerase" title="wikilink">DNA polymerases</a> that can insert bases at the site of damage. Some mechanisms of translesion synthesis introduce mutations, but others do not. For example, Pol &lt;U+03B7&gt; mediates error-free bypass of lesions induced by <a href="ultraviolet" title="wikilink">UV irradiation</a>, whereas Pol &lt;U+03B6&gt; introduces mutations at these sites. From the cell's perspective, the potential for introducing mutations during translesion synthesis is less dangerous than continuing the cell cycle with an incompletely replicated chromosome.</p>
<h2 id="global-response-to-dna-damage">Global response to DNA damage</h2>
<p>Cells exposed to <a href="ionizing_radiation" title="wikilink">ionizing radiation</a>, <a href="ultraviolet_light" title="wikilink">ultraviolet light</a> or chemicals are prone to acquire multiple sites of bulky DNA lesions and double strand breaks. Moreover, DNA damaging agents can damage other <a href="biomolecule" title="wikilink">biomolecules</a> such as <a href="protein" title="wikilink">proteins</a>, <a href="carbohydrate" title="wikilink">carbohydrates</a>, <a href="lipid" title="wikilink">lipids</a> and <a href="RNA" class="uri" title="wikilink">RNA</a>. The accumulation of damage, specifically double strand breaks or adducts stalling the <a href="replication_fork" title="wikilink">replication forks</a>, are among known stimulation signals for a global response to DNA damage.<a href="#fn19" class="footnote-ref" id="fnref19"><sup>19</sup></a> The global response to damage is an act directed toward the cells' own preservation and triggers multiple pathways of macromolecular repair, lesion bypass, tolerance or <a href="apoptosis" class="uri" title="wikilink">apoptosis</a>. The common features of global response are induction of multiple <a href="gene" title="wikilink">genes</a>, <a href="cell_cycle" title="wikilink">cell cycle</a> arrest, and inhibition of <a href="cell_division" title="wikilink">cell division</a>.</p>
<h3 id="dna-damage-checkpoints">DNA damage checkpoints</h3>
<p>After DNA damage, <a href="cell_cycle" title="wikilink">cell cycle</a> <a href="cell_cycle_checkpoint" title="wikilink">checkpoints</a> are activated. Checkpoint activation pauses the cell cycle and gives the cell time to repair the damage before continuing to divide. DNA damage checkpoints occur at the <a href="G1_phase" title="wikilink">G1</a>/<a href="S_phase" title="wikilink">S</a> and <a href="G2_phase" title="wikilink">G2</a>/<a href="mitosis" title="wikilink">M</a> boundaries. An intra-<a href="S_phase" title="wikilink">S</a> checkpoint also exists. Checkpoint activation is controlled by two master <a href="kinase" title="wikilink">kinases</a>, <a href="ataxia_telangiectasia_mutated" title="wikilink">ATM</a> and <a href="Ataxia_Telangiectasia_and_Rad3_related" title="wikilink">ATR</a>. ATM responds to DNA double-strand breaks and disruptions in chromatin structure,<a href="#fn20" class="footnote-ref" id="fnref20"><sup>20</sup></a> whereas ATR primarily responds to stalled <a href="replication_fork" title="wikilink">replication forks</a>. These kinases <a href="phosphorylation" title="wikilink">phosphorylate</a> downstream targets in a <a href="signal_transduction" title="wikilink">signal transduction</a> cascade, eventually leading to cell cycle arrest.</p>
<h3 id="the-prokaryotic-sos-response">The prokaryotic SOS response</h3>
<p>The <a href="SOS_response" title="wikilink">SOS response</a> is the term used to describe changes in <a href="gene_expression" title="wikilink">gene expression</a> in <a href="Escherichia_coli" title="wikilink">Escherichia coli</a> and other bacteria in response to DNA damage. The <a href="prokaryotic" class="uri" title="wikilink">prokaryotic</a> SOS system is regulated by two key proteins: <a href="LexA" class="uri" title="wikilink">LexA</a> and <a href="RecA" class="uri" title="wikilink">RecA</a>. The LexA <a href="homodimer" class="uri" title="wikilink">homodimer</a> is a <a href="Transcription_(genetics)" title="wikilink">transcriptional</a> <a href="repressor" class="uri" title="wikilink">repressor</a> that binds to <a href="operator" class="uri" title="wikilink">operator</a> sequences commonly referred to as SOS boxes. It is known that LexA regulates transcription of approximately 48 genes including the lexA and recA genes.<a href="#fn21" class="footnote-ref" id="fnref21"><sup>21</sup></a> The most common cellular signals activating the SOS response are regions of single stranded DNA (ssDNA), arising from stalled <a href="replication_fork" title="wikilink">replication forks</a> or double strand breaks, which are processed by <a href="DNA_helicase" title="wikilink">DNA helicase</a> to separate the two DNA strands.<ref name="textbook"> In the initiation step, RecA protein binds to ssDNA in an <a href="ATP_hydrolysis" title="wikilink">ATP hydrolysis</a> driven reaction creating RecA&lt;U+2013&gt;ssDNA filaments. RecA&lt;U+2013&gt;ssDNA filaments activate LexA auto<a href="protease" class="uri" title="wikilink">protease</a> activity which ultimately leads to cleavage of LexA dimmer and subsequent LexA degradation. The loss of LexA repressor induces transcription of the SOS genes and allows for further signal induction, inhibition of cell division and an increase in levels of proteins responsible for damage processing.</p>
<p>SOS boxes are 20-nucleotide long sequences near promoters with <a href="palindromic" class="uri" title="wikilink">palindromic</a> structure and a high degree of sequence conservation. This distinction in promoter sequences causes differential binding of LexA to different promoters and allows for timing of the SOS response. Logically, the lesion repair genes are induced at the beginning of SOS response. The error prone translession polymerases, for example: UmuCD&lt;U+2019&gt;2 (also called DNA polymerase V), are induced later on as a last resort.<a href="#fn22" class="footnote-ref" id="fnref22"><sup>22</sup></a> Once the DNA damage is repaired or bypassed using polymerases or through recombination, the amount of single-stranded DNA in cells is decreased, lowering the amounts of RecA filaments decreases cleavage activity of LexA homodimer which subsequently binds to the SOS boxes near promoters and restores normal gene expression.</p>
<h3 id="eukaryotic-transcriptional-responses-to-dna-damage">Eukaryotic transcriptional responses to DNA damage</h3>
<p><a href="Eukaryotic" class="uri" title="wikilink">Eukaryotic</a> cells exposed to DNA damaging agents also activate important defensive pathways by inducing multiple proteins involved in DNA repair, <a href="cell_cycle_checkpoint" title="wikilink">cell cycle checkpoint</a> control, protein trafficking and degradation. Such genome wide transcriptional response is very complex and tightly regulated, thus allowing coordinated global response to damage. Exposure of <a href="yeast" class="uri" title="wikilink">yeast</a> <a href="Saccharomyces_cerevisiae" title="wikilink">Saccharomyces cerevisiae</a> to DNA damaging agents results in overlapping but distinct transcriptional profiles. Similarities to environmental <a href="shock_response" title="wikilink">shock response</a> indicates that a general global stress response pathway exist at the level of transcriptional activation. In contrast, different human cell types respond to damage differently indicating an absence of a common global response. The probable explanation for this difference between yeast and human cells may be in the <a href="heterogeneity" class="uri" title="wikilink">heterogeneity</a> of <a href="mammalian" class="uri" title="wikilink">mammalian</a> cells. In an animal different types of cells are distributed amongst different organs which have evolved different sensitivities to DNA damage.<a href="#fn23" class="footnote-ref" id="fnref23"><sup>23</sup></a></p>
<p>In general global response to DNA damage involves expression of multiple genes responsible for postreplication repair, homologous recombination, nucleotide excision repair, DNA damage checkpoint, global transcriptional activation, genes controlling mRNA decay and many others. The vast amount of damage to a cell leaves it with an important decision; undergo apoptosis and die, or survive at the cost of living with a modified genome. An increase in tolerance to damage can lead to an increased rate of survival which will allow a greater accumulation of mutations. Yeast Rev1 and human polymerase &lt;U+03B7&gt; are members of [Y family translesion DNA <a href="polymerase" title="wikilink">polymerases</a> present during global response to DNA damage and are responsible for enhanced mutagenesis during a global response to DNA damage in eukaryotes.<a href="#fn24" class="footnote-ref" id="fnref24"><sup>24</sup></a></p>
<h2 id="dna-repair-and-aging">DNA repair and aging</h2>
<h3 id="pathological-effects-of-poor-dna-repair">Pathological effects of poor DNA repair</h3>
<figure>
<img src="Dnarepair1.jpg" title="DNA repair rate is an important determinant of cell pathology" alt="DNA repair rate is an important determinant of cell pathology" /><figcaption>DNA repair rate is an important determinant of cell pathology</figcaption>
</figure>
<p>Experimental animals with genetic deficiencies in DNA repair often show decreased lifespan and increased cancer incidence. For example, mice deficient in the dominant NHEJ pathway and in telomere maintenance mechanisms get <a href="lymphoma" class="uri" title="wikilink">lymphoma</a> and infections more often, and consequently have shorter lifespans than wild-type mice.<a href="#fn25" class="footnote-ref" id="fnref25"><sup>25</sup></a> Similarly, mice deficient in a key repair and transcription protein that unwinds DNA helices have premature onset of aging-related diseases and consequent shortening of lifespan.<a href="#fn26" class="footnote-ref" id="fnref26"><sup>26</sup></a> However, not every DNA repair deficiency creates exactly the predicted effects; mice deficient in the NER pathway exhibited shortened lifespan without correspondingly higher rates of mutation.<a href="#fn27" class="footnote-ref" id="fnref27"><sup>27</sup></a></p>
<p>If the rate of DNA damage exceeds the capacity of the cell to repair it, the accumulation of errors can overwhelm the cell and result in early senescence, apoptosis or cancer. Inherited diseases associated with faulty DNA repair functioning result in premature aging, increased sensitivity to carcinogens, and correspondingly increased cancer risk (see <a href="DNA_repair#Medicine_and_DNA_repair_modulation" title="wikilink">below</a>). On the other hand, organisms with enhanced DNA repair systems, such as <em><a href="Deinococcus_radiodurans" title="wikilink">Deinococcus radiodurans</a></em>, the most radiation-resistant known organism, exhibit remarkable resistance to the double strand break-inducing effects of <a href="radioactivity" class="uri" title="wikilink">radioactivity</a>, likely due to enhanced efficiency of DNA repair and especially NHEJ.<a href="#fn28" class="footnote-ref" id="fnref28"><sup>28</sup></a> It is noteworthy that some workers suggest that if a DNA damage event occurs during the self repair process then the combination of the two events will exert an effect greater than the sum of the individual events (if they occurred with a long time delay between them), this is the basis of the <em>second event theory</em> favoured by C. Busby (<a href="The_Low_Level_Radiation_Campaign" title="wikilink">The Low Level Radiation Campaign</a>).</p>
<h3 id="longevity-and-caloric-restriction">Longevity and caloric restriction</h3>
<p><img src="Dnadamage.jpg" title="fig:Most lifespan influencing genes affect the rate of DNA damage" alt="Most lifespan influencing genes affect the rate of DNA damage" /> A number of individual genes have been identified as influencing variations in lifespan within a population of organisms. The effects of these genes is strongly dependent on the environment, particularly on the organism's diet. <a href="Caloric_restriction" title="wikilink">Caloric restriction</a> reproducibly results in extended lifespan in a variety of organisms, likely via <a href="nutrient_sensing" title="wikilink">nutrient sensing</a> pathways and decreased <a href="metabolic_rate" title="wikilink">metabolic rate</a>. The molecular mechanisms by which such restriction results in lengthened lifespan are as yet unclear (see<a href="#fn29" class="footnote-ref" id="fnref29"><sup>29</sup></a> for some discussion); however, the behavior of many genes known to be involved in DNA repair is altered under conditions of caloric restriction.</p>
<p>For example, increasing the <a href="gene_dosage" title="wikilink">gene dosage</a> of the gene SIR-2, which regulates DNA packaging in the <a href="nematode_worm" title="wikilink">nematode worm</a> <em>Caenorhabditis elegans</em>, can significantly extend lifespan.<a href="#fn30" class="footnote-ref" id="fnref30"><sup>30</sup></a> The mammalian homolog of SIR-2 is known to induce downstream DNA repair factors involved in NHEJ, an activity that is especially promoted under conditions of caloric restriction.<a href="#fn31" class="footnote-ref" id="fnref31"><sup>31</sup></a> Caloric restriction has been closely linked to the rate of base excision repair in the nuclear DNA of rodents,<a href="#fn32" class="footnote-ref" id="fnref32"><sup>32</sup></a> although similar effects have not been observed in mitochondrial DNA.<a href="#fn33" class="footnote-ref" id="fnref33"><sup>33</sup></a></p>
<p>Interestingly, the <em>C. elegans</em> gene AGE-1, an upstream effector of DNA repair pathways, confers dramatically extended lifespan under free-feeding conditions but leads to a decrease in reproductive fitness under conditions of caloric restriction.<a href="#fn34" class="footnote-ref" id="fnref34"><sup>34</sup></a> This observation supports the <a href="pleiotropy" class="uri" title="wikilink">pleiotropy</a> theory of the <a href="senescence#theories_of_aging" title="wikilink">biological origins of aging</a>, which suggests that genes conferring a large survival advantage early in life will be selected for even if they carry a corresponding disadvantage late in life.</p>
<h2 id="medicine-and-dna-repair-modulation">Medicine and DNA repair modulation</h2>
<h3 id="hereditary-dna-repair-disorders">Hereditary DNA repair disorders</h3>
<p>Defects in the NER mechanism are responsible for several genetic disorders, including:</p>
<ul>
<li><a href="xeroderma_pigmentosum" title="wikilink">xeroderma pigmentosum</a>: hypersensitivity to sunlight/UV, resulting in increased skin cancer incidence and premature aging</li>
<li><a href="Cockayne_syndrome" title="wikilink">Cockayne syndrome</a>: hypersensitivity to UV and chemical agents</li>
<li><a href="trichothiodystrophy" class="uri" title="wikilink">trichothiodystrophy</a>: sensitive skin, brittle hair and nails</li>
</ul>
<p>Mental retardation often accompanies the latter two disorders, suggesting increased vulnerability of developmental neurons.</p>
<p>Other DNA repair disorders include:</p>
<ul>
<li><a href="Werner&#39;s_syndrome" title="wikilink">Werner's syndrome</a>: premature aging and retarded growth</li>
<li><a href="Bloom_syndrome" title="wikilink">Bloom's syndrome</a>: sunlight hypersensitivity, high incidence of <a href="malignant" title="wikilink">malignancies</a> (especially <a href="leukemia" title="wikilink">leukemias</a>).</li>
<li><a href="ataxia_telangiectasia" title="wikilink">ataxia telangiectasia</a>: sensitivity to ionizing radiation and some chemical agents</li>
</ul>
<p>All of the above diseases are often called &quot;segmental <a href="progeria" title="wikilink">progerias</a>&quot; (&quot;<a href="accelerated_aging_disease" title="wikilink">accelerated aging diseases</a>&quot;) because their victims appear elderly and suffer from aging-related diseases at an abnormally young age.</p>
<p>Other diseases associated with reduced DNA repair function include <a href="Fanconi&#39;s_anemia" title="wikilink">Fanconi's anemia</a>, hereditary <a href="breast_cancer" title="wikilink">breast cancer</a> and hereditary <a href="colon_cancer" title="wikilink">colon cancer</a>.</p>
<h3 id="dna-repair-and-cancer">DNA repair and cancer</h3>
<p>Inherited mutations that affect DNA repair genes are strongly associated with high cancer risks in humans. <a href="Hereditary_nonpolyposis_colorectal_cancer" title="wikilink">Hereditary nonpolyposis colorectal cancer</a> (HNPCC) is strongly associated with specific mutations in the DNA mismatch repair pathway. <a href="BRCA1" class="uri" title="wikilink">BRCA1</a> and <a href="BRCA2" class="uri" title="wikilink">BRCA2</a>, two famous mutations conferring a hugely increased risk of breast cancer on carriers, are both associated with a large number of DNA repair pathways, especially NHEJ and homologous recombination.</p>
<p>Cancer therapy procedures such as <a href="chemotherapy" class="uri" title="wikilink">chemotherapy</a> and <a href="radiotherapy" class="uri" title="wikilink">radiotherapy</a> work by overwhelming the capacity of the cell to repair DNA damage, resulting in cell death. Cells that are most rapidly dividing - most typically cancer cells - are preferentially affected. The side effect is that other non-cancerous but rapidly dividing cells such as <a href="stem_cell" title="wikilink">stem cells</a> in the bone marrow are also affected. Modern cancer treatments attempt to localize the DNA damage to cells and tissues only associated with cancer, either by physical means (concentrating the therapeutic agent in the region of the tumor) or by biochemical means (exploiting a feature unique to cancer cells in the body).</p>
<h2 id="dna-repair-and-evolution">DNA repair and evolution</h2>
<p>The basic processes of DNA repair are highly <a href="conservation_(genetics)" title="wikilink">conserved</a> among both <a href="prokaryotes" class="uri" title="wikilink">prokaryotes</a> and <a href="eukaryotes" class="uri" title="wikilink">eukaryotes</a> and even among <a href="bacteriophage" class="uri" title="wikilink">bacteriophage</a> (<a href="virus" title="wikilink">viruses</a> that infect <a href="bacteria" class="uri" title="wikilink">bacteria</a>); however, more complex organisms with more complex genomes have correspondingly more complex repair mechanisms.<a href="#fn35" class="footnote-ref" id="fnref35"><sup>35</sup></a> The ability of a large number of protein <a href="structural_motif" title="wikilink">structural motifs</a> to catalyze relevant chemical reactions has played a significant role in the elaboration of repair mechanisms during evolution. For an extremely detailed review of hypotheses relating to the evolution of DNA repair, see.<a href="#fn36" class="footnote-ref" id="fnref36"><sup>36</sup></a></p>
<p>The <a href="fossil_record" title="wikilink">fossil record</a> indicates that single celled life began to proliferate on the planet at some point during the <a href="Precambrian" class="uri" title="wikilink">Precambrian</a> period, although exactly when recognizably modern life first emerged is unclear. <a href="Nucleic_acid" title="wikilink">Nucleic acids</a> became the sole and universal means of encoding genetic information, requiring DNA repair mechanisms that in their basic form have been inherited by all extant life forms from their common ancestor. The emergence of Earth's oxygen-rich atmosphere (known as the &quot;<a href="oxygen_catastrophe" title="wikilink">oxygen catastrophe</a>&quot;) due to <a href="photosynthesis" title="wikilink">photosynthetic</a> organisms, as well as the presence of potentially damaging <a href="free_radical" title="wikilink">free radicals</a> in the cell due to <a href="oxidative_phosphorylation" title="wikilink">oxidative phosphorylation</a>, necessitated the evolution of DNA repair mechanisms that act specifically to counter the types of damage induced by <a href="oxidative_stress" title="wikilink">oxidative stress</a>.</p>
<h3 id="rate-of-evolutionary-change">Rate of evolutionary change</h3>
<p>On some occasions, DNA damage is not repaired, or is repaired by an error-prone mechanism which results in a change from the original sequence. When this occurs, <a href="mutation" title="wikilink">mutations</a> may propagate into the genomes of the cell's progeny. Should such an event occur in a <a href="germ_line" title="wikilink">germ line</a> cell that will eventually produce a <a href="gamete" class="uri" title="wikilink">gamete</a>, the mutation has the potential to be passed on to the organism's offspring. The rate of <a href="evolution" class="uri" title="wikilink">evolution</a> in a particular species (or, more narrowly, in a particular gene) is a function of the rate of mutation. Consequently, the rate and accuracy of DNA repair mechanims have an influence over the process of evolutionary change.<a href="#fn37" class="footnote-ref" id="fnref37"><sup>37</sup></a></p>
<h2 id="see-also">See also</h2>
<ul>
<li><a href="Accelerated_aging_disease" title="wikilink">Accelerated aging disease</a></li>
<li><a href="Aging_DNA" title="wikilink">Aging DNA</a></li>
<li><a href="Cell_cycle" title="wikilink">Cell cycle</a></li>
<li><a href="DNA_replication" title="wikilink">DNA replication</a></li>
<li><a href="Gene_therapy" title="wikilink">Gene therapy</a></li>
<li><a href="Life_extension" title="wikilink">Life extension</a></li>
<li><a href="Human_mitochondrial_genetics" title="wikilink">Human mitochondrial genetics</a></li>
<li><a href="Progeria" class="uri" title="wikilink">Progeria</a></li>
<li><a href="Senescence" class="uri" title="wikilink">Senescence</a></li>
</ul>
<h2 id="references">References</h2>
<div class="references-small" style="column-count:2;-moz-column-count:2;">
<references />
</div>
<h2 id="external-links">External links</h2>

<ul>
<li><a href="http://www.cgal.icnet.uk/DNA_Repair_Genes.html">A comprehensive list of Human DNA Repair Genes</a></li>
<li><a href="http://www.biochem.ucl.ac.uk/bsm/xtal/teach/repair/tibs3.html">3D structures of some DNA repair enzymes</a></li>
<li><a href="http://www.scielo.br/scielo.php?pid=S0100-84551997000400032&amp;script=sci_arttext&amp;tlng=en">Human DNA repair diseases</a></li>
<li><a href="http://tango01.cit.nih.gov/sig/home.taf?_function=main&amp;SIGInfo_SIGID=32">DNA repair special interest group</a></li>
<li><a href="http://users.rcn.com/jkimball.ma.ultranet/BiologyPages/D/DNArepair.html">DNA Repair</a></li>
<li><a href="http://www.benbest.com/lifeext/aging.html#dna">DNA Damage and DNA Repair</a></li>
<li><a href="http://www.benbest.com/lifeext/aging.html#progeria">Segmental Progeria</a></li>
</ul>

<p><a href="de:DNA-Reparatur" class="uri" title="wikilink">de:DNA-Reparatur</a> <a href="es:Reparaci&lt;U+00F3&gt;n_del_ADN" title="wikilink">es:Reparaci&lt;U+00F3&gt;n del ADN</a> <a href="fr:R&lt;U+00E9&gt;paration_de_l&#39;ADN" title="wikilink">fr:R&lt;U+00E9&gt;paration de l'ADN</a> <a href="it:Riparazione_del_DNA" title="wikilink">it:Riparazione del DNA</a> <a href="hu:DNS-jav&lt;U+00ED&gt;t&lt;U+00E1&gt;s" class="uri" title="wikilink">hu:DNS-jav&lt;U+00ED&gt;t&lt;U+00E1&gt;s</a> <a href="ja:DNA&lt;U+4FEE&gt;&lt;U+5FA9&gt;" class="uri" title="wikilink">ja:DNA&lt;U+4FEE&gt;&lt;U+5FA9&gt;</a> <a href="pt:Reparo_de_ADN" title="wikilink">pt:Reparo de ADN</a> <a href="zh:DNA&lt;U+4FEE&gt;&lt;U+5FA9&gt;" class="uri" title="wikilink">zh:DNA&lt;U+4FEE&gt;&lt;U+5FA9&gt;</a></p>
<p><a href="Category:Aging" class="uri" title="wikilink">Category:Aging</a> <a href="Category:Cellular_processes" title="wikilink">Category:Cellular processes</a> <a href="Category:Gerontology" class="uri" title="wikilink">Category:Gerontology</a> <a href="Category:Molecular_genetics" title="wikilink">Category:Molecular genetics</a> <a href="Category:Mutation" class="uri" title="wikilink">Category:Mutation</a> <a href="Category:DNA_repair" title="wikilink"> </a></p>
<section class="footnotes">
<hr />
<ol>
<li id="fn1">Lodish H, Berk A, Matsudaira P, Kaiser CA, Krieger M, Scott MP, Zipursky SL, Darnell J. (2004). Molecular Biology of the Cell, p963. WH Freeman: New York, NY. 5th ed.<a href="#fnref1" class="footnote-back">↩</a></li>
<li id="fn2">Browner WS, Kahn AJ, Ziv E, Reiner AP, Oshima J, Cawthon RM, Hsueh WC, Cummings SR. (2004). The genetics of human longevity. <em>Am J Med</em> 117(11):851-60.<a href="#fnref2" class="footnote-back">↩</a></li>
<li id="fn3"></li>
<li id="fn4">Toshihiro Ohta, Shin-ichi Tokishita, Kayo Mochizuki, Jun Kawase, Masahide Sakahira and Hideo Yamagata, UV Sensitivity and Mutagenesis of the Extremely Thermophilic Eubacterium Thermus thermophilus HB27, Genes and Environment Vol. 28 (2006) , No. 2 p.56-61.<a href="#fnref4" class="footnote-back">↩</a></li>
<li id="fn5">Braig M, Schmitt CA. (2006) . Oncogene-induced senescence: putting the brakes on tumor development. <em>Cancer Res</em> 66: 2881-2884.<a href="#fnref5" class="footnote-back">↩</a></li>
<li id="fn6">Lynch MD. (2006). How does cellular senescence prevent cancer? <em>DNA Cell Biol</em> 25(2):69-78.<a href="#fnref6" class="footnote-back">↩</a></li>
<li id="fn7">Watson JD, Baker TA, Bell SP, Gann A, Levine M, Losick R. (2004). Molecular Biology of the Gene, ch. 9 and 10. Peason Benjamin Cummings; CSHL Press. 5th ed.<a href="#fnref7" class="footnote-back">↩</a></li>
<li id="fn8">Volkert MR. (1988). Adaptive response of Escherichia coli to alkylation damage. <em>Environ Mol Mutagen</em> 11(2):241-55.<a href="#fnref8" class="footnote-back">↩</a></li>
<li id="fn9"></li>
<li id="fn10"></li>
<li id="fn11">Wilson, T. E., Grawunder, U., and Lieber, M. R. Yeast DNA ligase IV mediates non-homologous DNA end joining. (1997) Nature 388, 495&lt;U+2013&gt;498. PMID 9242411<a href="#fnref11" class="footnote-back">↩</a></li>
<li id="fn12">Moore JK, Haber JE. Cell cycle and genetic requirements of two pathways of nonhomologous end-joining repair of double-strand breaks in Saccharomyces cerevisiae. Mol Cell Biol. 1996 May;16(5):2164-73. PMID 8628283<a href="#fnref12" class="footnote-back">↩</a></li>
<li id="fn13">Boulton SJ, Jackson SP. Saccharomyces cerevisiae Ku70 potentiates illegitimate DNA double-strand break repair and serves as a barrier to error-prone DNA repair pathways. EMBO J. 1996 Sep 16;15(18):5093-103. PMID 8890183<a href="#fnref13" class="footnote-back">↩</a></li>
<li id="fn14">Wilson, T. E., and Lieber, M. R. Efficient processing of DNA ends during yeast nonhomologous end joining. Evidence for a DNA polymerase beta (Pol4)-dependent pathway. (1999) J. Biol. Chem. 274, 23599&lt;U+2013&gt;23609. PMID 10438542<a href="#fnref14" class="footnote-back">↩</a></li>
<li id="fn15">Budman J, Chu G. Processing of DNA for nonhomologous end-joining by cell-free extract. EMBO J. 2005 Feb 23;24(4):849-60. PMID: 15692565<a href="#fnref15" class="footnote-back">↩</a></li>
<li id="fn16">Wang H, Perrault AR, Takeda Y, Qin W, Wang H, Iliakis G. (2003). Biochemical evidence for Ku-independent backup pathways of NHEJ. <em>Nucleic Acids Res</em> 31(18):5377-88.<a href="#fnref16" class="footnote-back">↩</a></li>
<li id="fn17">Jung D, Alt FW. Unraveling V(D)J recombination; insights into gene regulation. Cell. 2004 Jan 23;116(2):299-311. Review. PMID: 14744439<a href="#fnref17" class="footnote-back">↩</a></li>
<li id="fn18"><a href="#fnref18" class="footnote-back">↩</a></li>
<li id="fn19">Friedberg EC, Walker GC, Siede W, Wood RD, Schultz RA, Ellenberger T. (2006). DNA Repair and Mutagenesis, part 3. ASM Press. 2nd ed.<a href="#fnref19" class="footnote-back">↩</a></li>
<li id="fn20">Bakkenist CJ, Kastan MB. DNA damage activates ATM through intermolecular autophosphorylation and dimer dissociation. Nature. 2003 Jan 30;421(6922):499-506.<a href="#fnref20" class="footnote-back">↩</a></li>
<li id="fn21">Janion C. (2001). Some aspects of the SOS response system-a critical survey. Acta Biochim Pol. 48(3):599-610<a href="#fnref21" class="footnote-back">↩</a></li>
<li id="fn22">Schlacher K, Pham P, Cox MM, and Goodman MF. (2006). Roles of DNA Polymerase V and RecA Protein in SOS Damage-Induced Mutation. Chem. Rev 106(2) pp 406 - 419<a href="#fnref22" class="footnote-back">↩</a></li>
<li id="fn23">Fry RC, Begley TJ, Samson LD. (2004). Genome-wide responses to DNA-damaging agents. Annu Rev Microbiol. 59 pp 357 - 77<a href="#fnref23" class="footnote-back">↩</a></li>
<li id="fn24"></li>
<li id="fn25">Espejel S, Martin M, Klatt P, Martin-Caballero J, Flores JM, Blasco MA. (2004). Shorter telomeres, accelerated ageing and increased lymphoma in DNA-PKcs-deficient mice. <em>EMBO Rep</em> 5(5):503-9.<a href="#fnref25" class="footnote-back">↩</a></li>
<li id="fn26">de Boer J, Andressoo JO, de Wit J, Huijmans J, Beems RB, van Steeg H, Weeda G, van der Horst GT, van Leeuwen W, Themmen AP, Meradji M, Hoeijmakers JH. (2002). Premature aging in mice deficient in DNA repair and transcription. <em>Science</em> 296(5571):1276-9.<a href="#fnref26" class="footnote-back">↩</a></li>
<li id="fn27">Dolle ME, Busuttil RA, Garcia AM, Wijnhoven S, van Drunen E, Niedernhofer LJ, van der Horst G, Hoeijmakers JH, van Steeg H, Vijg J. (2006). Increased genomic instability is not a prerequisite for shortened lifespan in DNA repair deficient mice. <em>Mutat Res</em> 596(1-2):22-35.<a href="#fnref27" class="footnote-back">↩</a></li>
<li id="fn28">Kobayashi Y, Narumi I, Satoh K, Funayama T, Kikuchi M, Kitayama S, Watanabe H. (2004). Radiation response mechanisms of the extremely radioresistant bacterium Deinococcus radiodurans.<em>Biol Sci Space</em> 18(3):134-5.<a href="#fnref28" class="footnote-back">↩</a></li>
<li id="fn29">Spindler SR. (2005). Rapid and reversible induction of the longevity, anticancer and genomic effects of caloric restriction. <em>Mech Ageing Dev</em> 126(9):960-6.<a href="#fnref29" class="footnote-back">↩</a></li>
<li id="fn30">Tissenbaum HA, Guarente L. (2001). Increased dosage of a sir-2 gene extends lifespan in Caenorhabditis elegans. <em>Nature</em> 410(6825):227-30.<a href="#fnref30" class="footnote-back">↩</a></li>
<li id="fn31">Cohen HY, Miller C, Bitterman KJ, Wall NR, Hekking B, Kessler B, Howitz KT, Gorospe M, de Cabo R, Sinclair DA. (2004). Calorie restriction promotes mammalian cell survival by inducing the SIRT1 deacetylase. <em>Science</em> 305(5682):390-2.<a href="#fnref31" class="footnote-back">↩</a></li>
<li id="fn32">Cabelof DC, Yanamadala S, Raffoul JJ, Guo Z, Soofi A, Heydari AR. (2003). Caloric restriction promotes genomic stability by induction of base excision repair and reversal of its age-related decline. <em>DNA Repair (Amst.)</em> 2(3):295-307.<a href="#fnref32" class="footnote-back">↩</a></li>
<li id="fn33">Stuart JA, Karahalil B, Hogue BA, Souza-Pinto NC, Bohr VA. (2004). Mitochondrial and nuclear DNA base excision repair are affected differently by caloric restriction. <em>FASEB J</em> 18(3):595-7.<a href="#fnref33" class="footnote-back">↩</a></li>
<li id="fn34">Walker DW, McColl G, Jenkins NL, Harris J, Lithgow GJ. (2000). Evolution of lifespan in C. elegans. <em>Nature</em> 405(6784):296-7.<a href="#fnref34" class="footnote-back">↩</a></li>
<li id="fn35">Cromie GA, Connelly JC, Leach DR (2001). Recombination at double-strand breaks and DNA ends: conserved mechanisms from phage to humans. <em>Mol Cell.</em> 8(6):1163-74.<a href="#fnref35" class="footnote-back">↩</a></li>
<li id="fn36">O'Brien PJ. (2006). Catalytic promiscuity and the divergent evolution of DNA repair enzymes. <em>Chem Rev</em> 106(2):720-52.<a href="#fnref36" class="footnote-back">↩</a></li>
<li id="fn37">Maresca B, Schwartz JH (2006). Sudden origins: a general mechanism of evolution based on stress protein concentration and rapid environmental change. <em>Anat Rec B New Anat.</em> Jan;289(1):38-46<a href="#fnref37" class="footnote-back">↩</a></li>
</ol>
</section>
